<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
"http://www.w3.org/TR/html4/loose.dtd">
<html><head><title>Spatial transcriptomic landscape unveils immunoglobin-associated senescence as a hallmark of aging</title>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
</head>
<body>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div><p>Resource</p>

</div><div>
</div><div>
</div><div>
</div><div><p>Spatial transcriptomic landscape unveils<br/>immunoglobin-associated senescence as a hallmark<br/>of aging</p>

</div><div>
</div><div><br/>Graphical abstract<p></p>

</div><div>
</div><div><p>Highlights</p>

</div><div>
</div><div><br/>d Spatial transcriptomics shows increased entropy in tissue<br/><p></p>
<p>structure in aging male mice<br/></p>
<p>d Senescence-sensitive spots are characteristic features of<br/>tissue aging<br/></p>
<p>d Immunoglobulin G accumulates in various tissues of mice<br/>and humans during aging<br/></p>
<p>d Immunoglobulin-associated senescence might be a target<br/>for geroprotective intervention</p>

</div><div><p>Ma et al., 2024, Cell 187, 7025&#8211;7044<br/>November 27, 2024&#170; 2024 Elsevier Inc. All rights are reserved, inc<br/>for text and data mining, AI training, and similar technologies.<br/>https://doi.org/10.1016/j.cell.2024.10.019</p>

</div><div><p>Authors<br/>Shuai Ma, Zhejun Ji, Bin Zhang, ...,<br/>Weiqi Zhang, Ying Gu, Guang-Hui Liu<br/></p>
<p>Correspondence<br/>qujing@ioz.ac.cn (J.Q.),<br/>zhangwq@big.ac.cn (W.Z.),<br/>guying@genomics.cn (Y.G.),<br/>ghliu@ioz.ac.cn (G.-H.L.)<br/></p>
<p>In brief<br/>Spatial transcriptomic analysis of<br/>multiple tissues inmalemice during aging<br/>identifies senescence-sensitive spots<br/>and immunoglobulin G accumulation as<br/>characteristic features of tissue aging and<br/>suggests immunoglobulin-associated<br/>senescence as a potential target for<br/>geroprotective intervention.</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>luding those<br/></p>
<p><b>ll</b></p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div><p><b>ll</b></p>

</div><div>
</div><div><p>Resource<br/>Spatial transcriptomic landscape unveils<br/>immunoglobin-associated senescence<br/>as a hallmark of aging<br/>Shuai Ma,1,5,6,8,24,25,26 Zhejun Ji,2,5,6,25 Bin Zhang,1,8,25 Lingling Geng,7,25,26 Yusheng Cai,1,5,6,25,26 Chao Nie,9,25,26<br/>Jiaming Li,3,4,8,25 Yuesheng Zuo,3,4,8,25 Yuzhe Sun,12,25 Gang Xu,11,25 Beibei Liu,3,4 Jiaqi Ai,7 Feifei Liu,1,5,6,10 Liyun Zhao,7<br/>Jiachen Zhang,7 Hui Zhang,1,8,10 Shuhui Sun,1,5,6,10,26 Haoyan Huang,7 Yiyuan Zhang,1,5,6 Yanxia Ye,2,5,6 Yanling Fan,3,4<br/>Fangshuo Zheng,13 Jinghao Hu,7 Baohu Zhang,2,8 Jingyi Li,1,5,6,8,26 Xin Feng,14,26 Feng Zhang,15,26 Yuan Zhuang,16<br/>Tianjie Li,17 Yang Yu,18,26 Zhaoshi Bao,19,26 Sipei Pan,20 Concepcion Rodriguez Esteban,21 Zhili Liu,9 Haohao Deng,12<br/></p>
<p>(Author list continued on next page)<br/></p>
<p>1Key Laboratory of Organ Regeneration and Reconstruction, State Key Laboratory of Membrane Biology, Institute of Zoology, Chinese</p>

</div><div><br/>Academy of Sciences, Beijing 100101, China<br/>2Key Laboratory of Organ Regeneration and Reconstruction, State Key Laboratory of Stem Cell and Reproductive Biology, Institute of<br/>Zoology, Chinese Academy of Sciences, Beijing 100101, China<br/>3China National Center for Bioinformation, Beijing, China<br/>4Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China<br/>5Institute for Stem Cell and Regeneration, CAS, Beijing 100101, China<br/>6Beijing Institute for Stem Cell and Regenerative Medicine, Beijing 100101, China<br/>7National Clinical Research Center for Geriatric Disorders, Aging Translational Medicine Center, International Center for Aging and Cancer,<br/>Xuanwu Hospital Capital Medical University, Beijing 100053, China<br/>8University of Chinese Academy of Sciences, Beijing 100049, China<br/>9BGI Research, Shenzhen 518083, China<br/>10Beijing Anzhen Hospital, Capital Medical University, Beijing Institute of Heart, Lung, and Blood Vessel Diseases, Beijing 100029, China<br/>11Liver Transplant Center, Organ Transplant Center, West China Hospital of Sichuan University, Chengdu 610000, China<br/>12BGI Research, Beijing 102601, China<br/>13The Fifth People&#8217;s Hospital of Chongqing, Chongqing 400062, China<br/>14CAS Key Laboratory of Molecular Imaging, Institute of Automation, Chinese Academy of Sciences, Beijing 100190, China<br/>15Joint Innovation Center for Engineering in Medicine, Quzhou Affiliated Hospital of Wenzhou Medical University, Quzhou 324000, China<br/>16Department of Blood Transfusion, First Medical Center of Chinese PLA General Hospital, Beijing 100853, China<br/>17Department of Obstetrics and Gynecology, Beijing Friendship Hospital, Capital Medical University, Beijing, China<br/>18Clinical Stem Cell Research Center, State Key Laboratory of Female Fertility Promotion, Center for Reproductive Medicine, Peking<br/><p></p>
<p>(Affiliations continued on next page)</p>

</div><div><p>SUMMARY</p>

</div><div>
</div><div><p>To systematically characterize the loss of tissue integrity and organ dysfunction resulting from aging, we pro-<br/>duced an in-depth spatial transcriptomic profile of nine tissues in male mice during aging. We showed that<br/>senescence-sensitive spots (SSSs) colocalized with elevated entropy in organizational structure and that<br/>the aggregation of immunoglobulin-expressing cells is a characteristic feature of the microenvironment sur-<br/>rounding SSSs. Immunoglobulin G (IgG) accumulated across the aged tissues in both male and female mice,<br/>and a similar phenomenon was observed in human tissues, suggesting the potential of the abnormal eleva-<br/>tion of immunoglobulins as an evolutionarily conserved feature in aging. Furthermore, we observed that IgG<br/>could induce a pro-senescent state in macrophages and microglia, thereby exacerbating tissue aging, and<br/>that targeted reduction of IgG mitigated aging across various tissues in male mice. This study provides a<br/>high-resolution spatial depiction of aging and indicates the pivotal role of immunoglobulin-associated senes-<br/>cence during the aging process.</p>

</div><div>
</div><div><p>INTRODUCTION<br/></p>
<p>Living organisms exhibit intricate organization, with billions of<br/>cells that coordinate as functional units across organs, systems,<br/>and the body as a whole.1&#8211;3 In aging tissues, molecular and</p>

</div><div><p>C<br/>All rights are reserved, including those</p>

</div><div><p>cellular changes disrupt spatial organization, resulting in uneven<br/>functional decline.4&#8211;7 Investigating these spatiotemporal dy-<br/>namics can reveal new mechanisms of aging and potential ther-<br/>apeutic targets.8&#8211;10 However, single-cell transcriptomics, which<br/>generates detailed gene expression profiles in both young and</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>ell 187, 7025&#8211;7044, November 27, 2024 &#170; 2024 Elsevier Inc. 7025<br/>for text and data mining, AI training, and similar technologies.</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p>Feng Wen,12 Moshi Song,1,5,6,8,26 Si Wang,7,24,26 Guodong Zhu,22,26 Jiayin Yang,11,26 Tao Jiang,19,26<br/>Weihong Song,20,26 Juan Carlos Izpisua Belmonte,21,26 Jing Qu,2,5,6,8,10,24,* Weiqi Zhang,3,4,8,24,* Ying Gu,9,12,23,* and<br/>Guang-Hui Liu1,5,6,7,8,24,27,*<br/>University Third Hospital, Beijing, China<br/>19Department of Neurosurgery, Beijing Tiantan Hospital, Capital Medical University, Beijing 100070, China<br/>20Oujiang Laboratory, Center for Geriatric Medicine and Institute of Aging, Key Laboratory of Alzheimer&#8217;s Disease of Zhejiang Province,<br/>Zhejiang Provincial Clinical Research for Mental Disorders, The First-affiliated Hospital, Wenzhou Medical University, Wenzhou, Zhejiang<br/>325035, China<br/>21Altos Labs San Diego Institute of Science, San Diego, CA, USA<br/>22Institute of Gerontology, Guangzhou Geriatric Hospital, Guangzhou Medical University, Guangzhou, China<br/>23BGI Research, Hangzhou 310030, China<br/>24Aging Biomarker Consortium (ABC), Beijing 100101, China<br/>25These authors contributed equally<br/>26Senior author<br/>27Lead contact<br/>*Correspondence: qujing@ioz.ac.cn (J.Q.), zhangwq@big.ac.cn (W.Z.), guying@genomics.cn (Y.G.), ghliu@ioz.ac.cn (G.-H.L.)<br/>https://doi.org/10.1016/j.cell.2024.10.019<br/></p>
<p><b>ll<br/></b>Resource</p>

</div><div><p>aged organisms,11&#8211;18 detaches cells from their native tissue<br/>context. Spatially resolved transcriptomics, which links cellular<br/>position to sequencing data, offers an in-depth view of gene<br/>expression within tissues.19&#8211;23 Thus, to dissect tissue-level aging<br/>mechanisms, it is essential to explore how aging affects cellular<br/>and molecular dynamics within the intact tissue&#8217;s cellular<br/>microenvironments.<br/></p>
<p>Accumulation of senescent cells is causally linked to func-<br/>tional decline in aging tissues.24&#8211;28 Senescent cells release<br/>pro-inflammatory molecules, such as cytokines and chemo-<br/>kines, collectively known as the senescence-associated secre-<br/>tory phenotype (SASP), that can harm neighboring non-senes-<br/>cent cells and contribute to age-related diseases.4,29&#8211;31<br/>However, senescent cells are heterogeneous in their geograph-<br/>ical distribution and molecular characteristics, both within and<br/>across organs.32,33 Consequently, analyzing the localization<br/>and environment of senescent cells and their interactions with<br/>neighboring cells holds significant potential for advancing our<br/>understanding of aging drivers and may also help identify poten-<br/>tial intervention targets.<br/></p>
<p>Here, we delineated the Gerontological Geography (GG) for<br/>male mice by profiling the expansive spatiotemporal transcrip-<br/>tomic landscape of tissue aging across consecutive life stages.<br/>Furthermore, we have spatially characterized senescence-sen-<br/>sitive spots (SSSs) and their microenvironments. Through these<br/>efforts, we discovered that immunoglobulin-expressing cells<br/>localize in the vicinity of the SSS-centered niche, facilitating tis-<br/>sue aging. Additionally, we confirmed the consistent accumula-<br/>tion of immunoglobulins across species and genders during ag-<br/>ing, evident in both mice and humans. Crucially, immunoglobulin<br/>G (IgG) aggregation emerges as a pivotal catalyst in tissue<br/>senescence, presenting a promising molecular target for aging<br/>interventions.<br/></p>
<p>RESULTS<br/></p>
<p>Spatial molecular cartography of cell types across<br/>multiple tissues<br/>To explore the spatial cellular and molecular alterations in aging<br/>across various organs, we examined nine key tissues&#8212;hippo-</p>

</div><div><p>7026 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>campus, spinal cord, heart, lung, liver, small intestine, spleen,<br/>lymph node, and testis&#8212;from C57BL/6J male mice at young<br/>(2 months) and old (25 months) stages (Figure 1A, see STAR<br/>Methods). We first conducted a comprehensive phenotypic<br/>comparison between young and aged tissue samples (Figures<br/>1B and S1A&#8211;S1I). In both SA-b-Gal and SPiDER-bGal staining<br/>analyses, we observed an increase in positive staining in aged<br/>male mice, presenting as tissue-specific topological patterns<br/>(Figures 1B and S1A). Concurrently, the numbers of CD45-pos-<br/>itive immune cells were elevated in multiple tissues, a feature of<br/>age-related chronic inflammation (Figure S1B). Accordingly, we<br/>detected enlarged inflammatory areas in aged livers and<br/>increased numbers of microglial cells, the primary immune cells<br/>of the central nervous system, in both the hippocampus and spi-<br/>nal cord, indicative of neuroinflammation34,35 (Figures S1C and<br/>S1D). In addition, levels of fibrosis were elevated in both the<br/>heart and intestine of aged mice36 (Figure S1E). Furthermore,<br/>a reduction in endothelial cells (CD31+) was observed in multiple<br/>aged tissues, alongside a decrease in newly formed neurons<br/>(DCX+) in the dentate gyrus (DG) of aged hippocampus, pointing<br/>to impaired microvascular density and neurogenesis<br/>(Figures S1F and S1G). We also detected structural changes<br/>in the organs of old mice when compared with young mice,<br/>such as an increased cardiomyocyte cross-sectional area in<br/>the heart and enlarged alveolar area in the lungs (Figures S1H<br/>and S1I).<br/>To achieve a more granular tissue resolution, we conducted<br/></p>
<p>spatial transcriptomics RNA sequencing (stRNA-seq) on nine<br/>tissues from both young and old male mice using Stereo-seq<br/>technology (Figures 1C, S1J, and S2A&#8211;S2E).21,37,38 After<br/>rigorous quality control, we obtained 1,535,191 high-resolution<br/>spots from 103 tissue samples, averaging 1,450 genes per<br/>spot with variations across tissue types (Figures S2A&#8211;S2C;<br/>Table S1). Annotation with marker genes led to the identification<br/>of 71 distinct spot types, corresponding to known cellular com-<br/>ponents in various tissues and organs (Figures 1C, 1D, and S2E;<br/>Table S1).<br/>We delineated the spatial attributes of key stem cells that sus-<br/></p>
<p>tain tissue homeostasis and aging, encompassing neural pro-<br/>genitor cells in the DG and intestinal stem cells in the small</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p><b>A<br/></b></p>
<p><b>C D<br/></b></p>
<p><b>B<br/></b></p>
<p>(legend on next page)<br/></p>
<p><b>ll<br/></b></p>
<p>Cell 187, 7025&#8211;7044, November 27, 2024 7027<br/></p>
<p>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>intestinal crypts (Figures 1C and S2E). Our analysis character-<br/>ized the distribution of core functional and stromal cells, such<br/>as glial cells, fibroblasts, smooth muscle cells, endothelial cells,<br/>and Leydig cells, within interstitial and matrix regions across tis-<br/>sues and organs (Figures 1C and S2E). Additionally, we mapped<br/>the spatial profiles of vital immune cells, notably in immune hubs<br/>like the spleen and lymph nodes, including macrophages, gran-<br/>ulocytes, B cells, plasmocytes, and T cells (Figures 1C and S2E;<br/>Table S1).<br/></p>
<p>Spatial organization disorders associated with aging in<br/>male mice tissues<br/>In aged tissues, as shown above, histological analysis often re-<br/>veals that thewell-orderedorganization is compromised. Toquan-<br/>tify this phenomenon, we introduced &#8216;&#8216;organizational structure en-<br/>tropy&#8217;&#8217; (OSE), a concept formeasuring thedegreeofdisorderusing<br/>spatial transcriptome datasets (Figure 2A, see STAR Methods).<br/>Across various aged tissues, we found that the OSE is increased,<br/>with the highest levels of spatial transcriptional perturbation in the<br/>hippocampus, spleen, lymph node, and liver (Figures 2B, 2C, S3A,<br/>and S3B). We also revealed a preferential enrichment of immune<br/>cells, including microglia, macrophages, and plasmocytes, in<br/>regions of aged tissues exhibiting high OSE (R-OSEhigh)<br/>(Figures S3C and S3D). Furthermore, cellular identity loss, a<br/>common occurrence in the aging process, is more prevalent in<br/>R-OSEhigh compared with R-OSElow (Figures 2D and 2E).<br/></p>
<p>In particular, we noted blurred boundaries between white<br/>and red pulp in aged mouse spleens in both spatial transcrip-<br/>tomic and histological analyses (Figures S3E&#8211;S3H). This struc-<br/>tural deterioration was reflected as disrupted white pulp struc-<br/>tures and the dispersal of lymphocytes in aged spleens<br/>(Figures S3E&#8211;S3H). In the aged lymph node, another critical<br/>lymphoid tissue, we detected a decrease in T cells and a concur-<br/>rent increase in macrophages infiltrating the paracortical area<br/>and, in the follicular zone, a decrease in B cells but an increase<br/>in plasmocytes, terminally differentiated B cells that produce an-<br/>tibodies (Figure S3I). Consistently, we detected diffusion of peri-<br/>central zone 3 hepatocytes surrounding the central vein and<br/>reduced periportal zone 1 hepatocytes, which were displaced<br/>by accumulated adipocyte-like characteristics in aged mice<br/>(Figures S3J&#8211;S3L).<br/></p>
<p>To enhance our understanding of aging-induced structural en-<br/>tropy changes with greater temporal resolution, we obtained<br/>supplementary tissue samples from male mice at the ages of<br/>4, 13, and 19 months. We performed high-precision spatial tran-<br/>scriptomic analyses on a range of tissues, including the hippo-<br/>campus, liver, spleen, and lymph node (Figures 2F, 2G, S4A,</p>

</div><div><p>Figure 1. Building a spatial transcriptomic atlas for young and aged m<br/>(A) Schematic diagram of tissue sampling, histological analysis, and spatial trans<br/>(B) SA-b-Gal analysis of multiple tissues from young (2-month-old) and old (25-mo<br/>Whitney test. Scale bars, 50 mm.<br/>(C) Spatial transcriptomic atlas of young mouse tissues. The central UMAP plot s<br/>diagrams indicating the respective tissues. Refer to Table S1 for the abbreviation k<br/>spinal cord), 1 mm (others). Scale bars in the outer plots, 100 mm (lymph node),<br/>(D) The proportion of spot types in each tissue from young samples. The dendrog<br/>proportion of different spot types in each tissue, with each dot representing one<br/>See also Figures S1 and S2.<br/></p>
<p>7028 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>and S4B; Table S1, see STARMethods). Data from these distinct<br/>life stages, in line with our primary findings, revealed an escala-<br/>tion in OSE and a reduction in cellular identity as age advances<br/>(Figures 2H&#8211;2J).<br/></p>
<p>Spatially resolved age-related gene expression<br/>alterations in male mice tissues<br/>Next, to delineate cell-type-dependent aging-related alterations<br/>at spatial resolution, we conducted an in-depth analysis of differ-<br/>entially expressed genes (DEGs) between young and aged tis-<br/>sues and across different spatial locations (Figures S4C and<br/>S4D; Table S2). This comprehensive approach highlighted a<br/>key finding: a substantial proportion of aging-related DEGs ex-<br/>hibited both spatial and cell-type-specific expression patterns<br/>(Figure S4D). For instance, aging-induced notable changes in<br/>specific spatial regions, including the neural progenitor cell<br/>area in the hippocampal DG, the middle region of gray matter<br/>(GM) in the spinal cord, ventricular cardiomyocytes in the heart,<br/>and zone 1 hepatocytes in liver. In lymphoid tissues, such as the<br/>spleen and lymph nodes, pronounced changeswere observed in<br/>T and B cells within the white pulp and B cells in the follicles,<br/>respectively (Figures S4C and S4D). Of note, aging-related<br/>DEGs also displayed spatial heterogeneity within the similar<br/>cell types. For example, in the heart, numbers of such DEGs<br/>were higher in ventricular cardiomyocytes than in those in the<br/>atrial region, whereas in the liver, we noted a higher number of<br/>DEGs in zone 1 hepatocytes than in zone 2 and 3 hepatocytes<br/>(Figures S4C and S4D).<br/>In general, aging DEGs were closely linked to chronic inflam-<br/></p>
<p>mation, mitochondrial dysfunction, and disrupted nutrient<br/>sensing (Figures S4E and S5A&#8211;S5F). Upregulated DEGs were<br/>mainly associated with inflammation in lymph nodes, spleens,<br/>small intestine, and lungs, particularly in plasmocytes of aged<br/>lymph node medulla and spleen red pulp (Figures S4E and<br/>S5A). In comparison, downregulated DEGs correlated with mito-<br/>chondrial function in the lymph node, hippocampus, and small<br/>intestine (Figures S4E and S5B). For example, we observed<br/>weakened expression of genes associated with mitochondrial<br/>function pathways, along with an overrepresentation of upregu-<br/>lated apoptosis-related genes in neural progenitor cells and<br/>mature neural cells within the hippocampus DG and CA1-3 re-<br/>gions (Figures 4E and S5C). Moreover, the weakened expression<br/>of genes was associated with mitochondrial function pathways<br/>and fatty acid metabolism in hepatocytes of the hepatic lobule<br/>region within the liver, highlighting how spatially specific<br/>dysfunction in energy dynamics and metabolic function impacts<br/>the liver during aging (Figures S4E, S5B, and S5F).</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>ale mice across multiple tissues<br/>criptomics sequencing in young and old mice.<br/>nth-old) mice (n = 5 per group). Data are presented as the mean &#177; SEM. Mann-<br/></p>
<p>hows the distribution of spots across various mouse tissues, with surrounding<br/>eys. Scale bars in inner plots, 200 mm (lymph node), 500 mm (hippocampus and<br/>200 mm (others).<br/>ram shows the positional similarity between spot types. The bar plot shows the<br/>sample.</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p><b>A B<br/></b></p>
<p><b>C<br/></b></p>
<p><b>D<br/></b></p>
<p><b>F<br/></b></p>
<p><b>H<br/></b></p>
<p><b>I J<br/></b></p>
<p><b>G<br/></b></p>
<p><b>E<br/></b></p>
<p>Figure 2. Exploration of structural deterioration and cellular identity loss in aged male mice<br/>(A) Schematic illustrating the organizational structure entropy (OSE) analysis.<br/>(B) Violin plots demonstrating the increased OSE score during aging. Wilcoxon rank-sum test.<br/>(C) The spatial mappings showing the OSE score in young (2-month-old) and old (25-month-old) mice tissues. Scale bars, 500 and 100 mm (zoom-in) for hip-<br/>pocampus and lymph node, 1 mm and 200 mm (zoom-in) for the spleen.<br/></p>
<p>(legend continued on next page)<br/></p>
<p><b>ll<br/></b></p>
<p>Cell 187, 7025&#8211;7044, November 27, 2024 7029<br/></p>
<p>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>Spatial mapping of SSSs and their niches in male mouse<br/>tissues<br/>Next, we shifted focus to pinpoint regions harboring senescence<br/>hotspots and characterize their adjacent microenvironments. To<br/>achieve this goal, we leveraged five well-established senes-<br/>cence gene sets extensively utilized in prior studies intersected<br/>with spot type-specific DEGs from each tissue.39&#8211;42 Based on<br/>our aging DEGs and aging-related gene set, we computed an ag-<br/>ing score for each spatial spot across various tissues (Figure 3A;<br/>Table S2, see STAR Methods). By ranking spots based on their<br/>aging scores, our approach enables the identification of regions<br/>in different tissues with heightened susceptibility to aging. We<br/>designated the top 5% of spots with the highest aging scores<br/>in each aged sample as SSSs (Figure 3B). We noted that the in-<br/>tensity of aging scores peaked at the center of the SSS and grad-<br/>ually decreased outward to peripheral cells in a distance-depen-<br/>dent manner (Figure 3C). Intriguingly, our analysis revealed that<br/>areas with increased OSE scores and decreased cell identity<br/>scores tended to cluster near the SSS (Figures 3D&#8211;3F and S6A).<br/></p>
<p>We then dissected the molecular gradient characteristics sur-<br/>rounding the SSS by ranking gene expression based on the<br/>proximity to the SSS (Figures 3G, 3H, and S6B; Table S2). In<br/>this way, we found within the SSS, humoral immune response<br/>and complement activation were elevated, and proximity to the<br/>SSS correlated with higher SASP scores (Figures 3G, 3H, and<br/>S6C). Consistently, TNF-signaling gradually decreases with<br/>increasing distance from the SSS (Figure S6D). By contrast,<br/>the expression levels of genes associated with ATP biosynthetic<br/>process and cell cycle were restored with increasing distance<br/>from the SSS (Figures 3G and 3H). We consistently observed<br/>this spatial pattern across multiple organs in aged mice<br/>(Figures S6A&#8211;S6D). These findings suggest that SSSs serve as<br/>epicenters for heightened inflammation that compromises sur-<br/>rounding cells in a distance-dependent manner.<br/></p>
<p>Next, we investigated the cellular identity of SSSs at a spatial<br/>resolution. In the hippocampus, we detected oligodendrocyte<br/>(OL) and CA2&amp;CA3 neurons as the most prevalent SSSs<br/>(Figures 3I and S6E&#8211;S6H). In the aged hippocampus, the SSS<br/>spatial pattern coincided with the distribution of SPiDER-bGal-<br/>positive cells, which were preferentially located within the<br/>neuronal regions (Figure S6I). In the spinal cord, almost half of<br/>SSSs were neuronal spots in the middle-GM region, whereas<br/>in the heart, myocardial cells were predominant (Figures 3I and<br/>S6G). Similarly, alveolar epithelial cells in the lung, zone 2 hepa-<br/>tocytes in the liver, and the androgen-producing Leydig cells in<br/>the testis were the main SSS components (Figures 3I and S6J).<br/></p>
<p>Our findings consistently documented that plasmocytes and<br/>terminally differentiated B cells make up the predominant SSS</p>

</div><div><p>(D) Heatmaps display cell identity scores in aged tissues and regions with high O<br/>(E) The spatial mappings showing the decreased cell identity scores during agin<br/>(F) Schematic diagram of tissue sampling and spatial transcriptomics sequencin<br/>(G) Spatial mappings showing spot types of 4-month-old mice. Scale bars, 200<br/>(zoom-in) for liver and spleen.<br/>(H) Boxplots showing the change of OSE during aging, with each point indicating<br/>(I) Heatmap showing the decreased cell identity scores during aging.<br/>(J) Spatial mappings showing the decreased cell identity scores during aging. Sc<br/>See also Figures S3 and S4.<br/></p>
<p>7030 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>fraction in the spleen and lymph node, constituting approxi-<br/>mately 50% and 30% of SSSs within each tissue, respectively<br/>(Figures 3I and S6G). Specifically, we found plasmocyte SSSs<br/>surrounding the white pulp of the spleen and the medulla region<br/>of lymph node (Figures 3I and S6E). Moreover, based on histo-<br/>logical staining in the lymph nodes and spleen, the majority of<br/>SPiDER-bGal-positive cells co-localized with CD138-positive<br/>cells, providing evidence for the strong correlation between plas-<br/>mocytes and the regions characterized as SSSs (Figures S6K<br/>and S6L), and we noted the accumulated plasmocytes in aged<br/>lymph node medullary regions, spleen&#8217;s red pulp, and intestinal<br/>villi (Figure S6M).<br/>Of note, spatial transcriptomic assessments at 4, 13, and<br/></p>
<p>19 months consistently validated increased OSE levels in prox-<br/>imity to SSSs and a trend toward reduced cellular identity within<br/>these areas (Figures 3J&#8211;3L and S7A&#8211;S7E). Additionally, this da-<br/>taset substantiates that plasma cells are essential cellular con-<br/>stituents of the microenvironment surrounding SSSs in the<br/>spleen and lymph node (Figure 3M).<br/></p>
<p>Immunoglobulin-expressing cells and SSSs establish a<br/>symbiotic system<br/>To identify culprit genes likely to functionally contribute to SSSs<br/>and their microenvironments, we analyzed the most frequently<br/>occurring DEGs and examined their spatial relationships with<br/>SSSs. Across tissues, the most highly upregulated genes were<br/>those encoding immunoglobulins, such as Igkc, Jchain (Igj),<br/>Ighg1, and Igha, and those encoding complement proteins,<br/>such as C4b and C1qa (Figures 4A and S7F). Of those, Igkc,<br/>which encodes the constant region of immunoglobulin light<br/>chains, and Jchain, which is synthesized by all plasmocytes<br/>secreting polymeric immunoglobulins, were most frequently<br/>increased in aged tissues, especially in the lymph nodes, spleen,<br/>and small intestine (Figures 4B and S7G). We also found that the<br/>expression of other immunoglobulin-related genes, such as<br/>Ighg2c, Ighg2b, and Ighg3, which encode the heavy chains for<br/>IgG, was elevated in aged tissues, particularly in the spleen,<br/>lymph node, and small intestine (Figure S7G). In general, the pro-<br/>portion of spots with high Igkc gene expression in combination<br/>with expression of either a heavy chain encoding gene (Ighigh<br/>spot for short) was also markedly elevated in aged tissues<br/>(Figures 4C and 4D). By western blot analysis, we confirmed<br/>the extraordinary accumulation of light and heavy IgG chains,<br/>as well as Igkc, across various tissues (Figures S8A and S8B).<br/>Furthermore, our data revealed that most Igkc-positive cells<br/>also expressed IgG in tissues such as hippocampus, small intes-<br/>tine, spleen, and lymph node (Figure S8C). In line with increased<br/>immunoglobulin infiltration in aged tissues, Igkc and IgG levels</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>SE scores.<br/>g. Scale bars, 200 mm.<br/>g of 4, 13, 19-month-old mice.<br/>and 100 mm (zoom-in) for hippocampus and lymph node, 1 mm and 200 mm<br/></p>
<p>a superpixel unit. Wilcoxon rank-sum test.<br/></p>
<p>ale bars, 200 mm.</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p><b>A B<br/></b></p>
<p><b>C<br/></b></p>
<p><b>F<br/></b></p>
<p><b>G<br/></b></p>
<p><b>H<br/></b></p>
<p><b>J<br/></b></p>
<p><b>M<br/></b></p>
<p><b>K L<br/></b></p>
<p><b>I<br/></b></p>
<p><b>D E<br/></b></p>
<p>(legend on next page)<br/></p>
<p><b>ll<br/></b></p>
<p>Cell 187, 7025&#8211;7044, November 27, 2024 7031<br/></p>
<p>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>were also elevated in the plasma of aged animals, similar to that<br/>of the classic senescence-associated secretory factor IL-6<br/>(Figure 4E).<br/></p>
<p>We next sought to dissect the spatial relationship between<br/>immunoglobulin-producing cells and SSSs. We categorized three<br/>types of Ighigh spots: those co-localizing with SSSs, those adja-<br/>cent to SSSs, and the relatively few that were at a distance from<br/>SSSs (Figure S8D). In the spleen and lymph node, Ighigh spots<br/>either directly colocalized with SSSs or were in their closest prox-<br/>imity (Figures 4F and S8D). When we conducted co-staining for<br/>CD138 and Igkc, we noted that, in aged lymph nodes and spleen,<br/>most Igkc-positive cells are CD138-positive plasmocytes and that<br/>the population of cells double-positive for Igkc and CD138<br/>dramatically increased in aged tissues (Figure S8E). When we<br/>co-stained with Igkc and SPiDER-bGal to quantify the distribution<br/>of Ighigh spot at varying proximities to SSSs, we found that<br/>SPiDER-bGal-positive cells were closely surroundedby Igkc-pos-<br/>itive cells in aged lymph node and spleen (Figure S8F).<br/></p>
<p>In the hippocampus, spinal cord, heart, lung, liver, intestine, and<br/>testis, unlike in the lymph node and spleen, the predominant SSS<br/>component was not plasmocytes (Figures 3I and S6G). However,<br/>we noted that a substantial number of Ighigh spots were consis-<br/>tently located in the peripheral zone closest to SSSs (Figures 4F<br/>and S8D). Moreover, Ighigh spots tended to aggregate with en-<br/>riched SSSs in these aged tissues (Figures 4F and S8D). To be<br/>noted, in these tissues, Ighigh spots are not exclusively plasmo-<br/>cytes; they are OLs in the hippocampus and spinal cord, and<br/>cardiomyocytes in the heart (Figure S8G). This supports the hy-<br/>pothesis that certain structural cellsmay gain immunoglobulin-ex-<br/>pressing capabilities in aged individuals. Consistently, multi-time<br/>point spatial transcriptomic data further confirm the ongoing<br/>activation of immunoglobulin-encoding genes, particularly immu-<br/>noglobulin heavy chain genes and Igkc, across various aging tis-<br/>sues and organs (hippocampus, liver, spleen, and lymph node)<br/>(Figures 4G&#8211;4J, S8H, and S8I).<br/></p>
<p>Immunoglobulin accumulation as a hallmark of aging<br/>To dynamically map immunoglobulin expression changes with<br/>age, we analyzed multiple tissues in male mice at five life stages:</p>

</div><div><p>Figure 3. Characterizing senescence-sensitive spot in diverse tissues<br/>(A) Schematic diagram showing the gene number of aging-related gene sets and<br/>(B) The schematic diagram showing the definition of SSS and SSS niche.<br/>(C) Boxplots showing the aging score among different distances from SSS in old<br/>(D) Boxplot showing the decreased entropy around the SSS in old mouse hippoc<br/>(E) Spatial mapping showing the increased entropy around the SSS in old mouse<br/>(F) Density plots showing the loss of cell identity surrounding the SSS. Kolmogor<br/>(G) Categorizing the genes changes surrounding the SSS into different gene expre<br/>the enrichment of GO terms and pathways (bottom) of genes with the increased<br/>(H) Fitting curves of expression levels of genes involved in the indicated pathway<br/>(I) Top, spatial mappings showing the density distribution of SSSs. Scale bars, 200<br/>500 and 200 mm (zoom-in) for the others. Bottom, boxplot showing the spot prop<br/>(J) Boxplots showing the decreased aging score along with the increased distan<br/>(K) Density plots showing the loss of cell identity surrounding the SSS in 19-mon<br/>(L) Boxplot (left) and spatial mapping (right) showing the increased entropy arou<br/>(zoom-in). Kruskal-Wallis test.<br/>(M) Left, spatial mappings showing the density areas of SSSs. Scale bars, 200 and<br/>(zoom-in) for liver. Right, boxplot showing the spot proportion of SSSs in 19-mo<br/>See also Figures S4, S5, S6, and S7.<br/></p>
<p>7032 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>2, 4, 13, 19, and 25 months. As expected, aging hallmarks such<br/>as SPiDER-bGal-positive cells, P21, 4-HNE, gH2AX-positive<br/>cells, and MMTV (a mouse homolog of human HERVK)-positive<br/>cells were found to be upregulated with advancing age across<br/>various tissues43&#8211;46 (Figures S9A&#8211;S9J). Additionally, we noted<br/>an expansion of aggresomes and fibrosis, along with elevated<br/>levels of inflammatory cytokines, while the lamin B1 signal dimin-<br/>ished with age (Figures S9C&#8211;S9E, S9H, and S9I). Parallel to<br/>classic aging biomarkers, we detected an age-dependent in-<br/>crease in IgG expression across tissues in male mice<br/>(Figures 5A and 5B). In addition, we conducted co-staining of<br/>Igkc and SPiDER-bGal and found that SPiDER-bGal-positive<br/>cells were closely surrounded by Igkc-positive cells in aged<br/>spleen and lymph node (Figure S9K). To verify if immunoglobulin<br/>accumulation with age is sex-independent, we stained samples<br/>from female mice at ages 4, 13, and 19 months and observed<br/>similar IgG and Igkc buildup in multiple organs during the aging<br/>process (Figure 5C). By contrast, established geroprotective<br/>strategies such as long-term exercise and exposure to youthful<br/>bodily fluids,17,47&#8211;49 mitigated the accumulation of IgG and se-<br/>nescent cells in aging tissues (Figures S9L&#8211;S9Q).<br/>More importantly, we demonstrated an age-related accumula-<br/></p>
<p>tion of IgG in human lymph node, liver, spleen, and brain<br/>(Figures 5D and 5E). Thus, we have identified a consistent<br/>pattern of intra-tissue IgG accumulation across multiple life<br/>stages, sexes, and species, supporting that our observations<br/>represent a widespread phenomenon in mammals and could<br/>serve as a marker of human aging.<br/></p>
<p>IgG acts as a senescence-promoting factor<br/>To investigate whether and how excessive IgG levels contribute<br/>to tissue aging, we next sought to identify cell types capable of<br/>recognizing and responding to immunoglobulins. Across various<br/>tissues, we observed an upregulation in the expression of genes<br/>encoding immunoglobulin receptors present in macrophages in<br/>tissues (Figure S10A). Furthermore, through immunofluores-<br/>cence staining, we observed thatmacrophages and IgG-positive<br/>cells were in close spatial proximity to each other in the spleen<br/>and lymph nodes of aged mice (Figure S10B). Additionally,</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>of male mice<br/>the definition of aging-sensitive genes.<br/></p>
<p>mouse (25-month-old). Kruskal-Wallis test.<br/>ampus. Kruskal-Wallis test.<br/>hippocampus. Scale bars, 500 and 100 mm (zoom-in).<br/>ov-Smirnov test.<br/>ssion modules. Fitting curves of expression levels (top) of several modules and<br/>distance from SSSs.<br/>s in the module 1 + module 2 and the module 3 + module 4.<br/>and 100 mm (zoom-in) for hippocampus, spinal cord, spleen, and lymph node,<br/>ortion of SSS in old samples, with each dot representing a sample.<br/>ce from SSS in 19-month-old mice. Kruskal-Wallis test.<br/>th-old mice. Kolmogorov-Smirnov test.<br/>nd the SSS in 19-month-old mice hippocampus. Scale bars, 500 and 100 mm<br/></p>
<p>100 mm (zoom-in) for hippocampus, spleen, and lymph node, 500 and 200 mm<br/>nth-old mice samples, with each dot representing a sample.</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p><b>A<br/></b></p>
<p><b>D<br/></b></p>
<p><b>E<br/></b></p>
<p><b>F<br/></b></p>
<p><b>H<br/></b></p>
<p><b>I J<br/></b></p>
<p><b>G<br/></b></p>
<p><b>CB<br/></b></p>
<p>(legend on next page)<br/></p>
<p><b>ll<br/></b></p>
<p>Cell 187, 7025&#8211;7044, November 27, 2024 7033<br/></p>
<p>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>relative to their younger counterparts, transcriptional profiles of<br/>macrophages in aged tissues revealed upregulation of genes<br/>associated with immunoglobulin-mediated immune responses<br/>(Figure S10C). Based on these observations, we hypothesize<br/>that age-associated intra-tissue accumulation of immunoglobu-<br/>lins will likely contribute toward transforming macrophages into<br/>an aged state.<br/></p>
<p>To substantiate our hypothesis, we isolated and treated<br/>three distinct macrophage populations&#8212;those derived from<br/>bone marrow, spleen, and peritoneal cavity&#8212;with IgG (Fig-<br/>ure 6A). Our data revealed that IgG induced senescence in<br/>these macrophages, as indicated by an increase in SA-<br/>b-Gal activity and P21 expression, along with upregulated<br/>MMTV-positive cells, aggresome formation, and decreased<br/>H3K9me3 and lamin B1 levels46,50&#8211;52 (Figures 6B&#8211;6G and<br/>S10D&#8211;S10G). Simultaneously, we observed a marked increase<br/>in activated nuclear factor kB (NF-kB)/P65 (p-P65), STAT1,<br/>and secretion of pro-inflammatory factors from IgG-treated<br/>macrophages (Figures 6H, S10H, and S10I). Furthermore,<br/>following IgG treatment, we observed an increase in the<br/>expression of inducible nitric oxide synthase (iNOS) and the<br/>production of nitric oxide (NO), both hallmarks of pro-inflam-<br/>matory macrophages (Figures 6C, S10G, and S10J). In line<br/>with these findings, RNA sequencing data corroborate the pro-<br/>nounced upregulation of genes linked to senescence and<br/>inflammation after IgG treatment (Figures S10K&#8211;S10N;<br/>Table S3). Knockdown of Fcgr4, a gene encoding an IgG Fc<br/>receptor, alleviated IgG-induced macrophage senescence<br/>(Figures 6I&#8211;6K and S10O), supporting a role of IgG receptor<br/>on the cell surface in mediating the pro-senescence signaling<br/>of IgG. Similarly, we observed that IgG treatment induced<br/>senescence in human macrophages (Figures 6L&#8211;6O and<br/>S10P&#8211;S10R). The efficacy of IgG in promoting macrophage<br/>senescence is contingent upon its active protein conformation,<br/>as heat-inactivated IgG fails to induce cellular aging (Fig-<br/>ure S10S). In summary, these data indicate that macrophages<br/>become primed toward a senescent and pro-inflammatory<br/>state when exposed to increased IgG levels.<br/></p>
<p>We next extended our investigation into microglia, a special-<br/>ized type of macrophage in the brain, and found that genes en-<br/>coding immunoglobulin receptors were upregulated in microglia<br/>during aging (Figure S10A). Similar to our observations in macro-<br/>phages, treatment with IgG of both mouse and human microglia<br/>induced the expression of a series of senescence-related</p>

</div><div><p>Figure 4. Immunoglobulin-expressing cells accumulate in multiple tiss<br/>(A) Heatmap showing the high-frequency upregulated aging DEGs across differe<br/>(B) Schematic diagram of the structures for immunoglobulins. Immunoglobulin e<br/>(C) Boxplots showing the proportion of Ighigh spots in young (2-month-old) and o<br/>(D) Spatial mappings of Igkc expression and Ighigh spots in young and oldmouse tis<br/>lymph node; 1 mm and 200 mm (zoom-in) for the others.<br/>(E) ELISA test to detect the Igkc, IgG, and IL-6 levels in the plasma from young a<br/>Mann-Whitney test.<br/>(F) Boxplots displaying the proportion of Ighigh spots in relation to their distance<br/>(G) Heatmap showing the expression of upregulated immunoglobulin-related gen<br/>(H) Boxplots displaying the proportion of Ighhigh and Igkchigh spots in relation to t<br/>(I) Spatial mappings of the Ighhigh and Igkchigh spots in different ages. Scale bars<br/>(J) Linear fitting of the increased proportion of Ighhigh and Igkchigh spots during m<br/>See also Figures S7 and S8.<br/></p>
<p>7034 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>markers and the secretion of inflammatory factors (Figures 6P&#8211;<br/>6T, S10K&#8211;S10N, and S10T&#8211;S10V).<br/>Next, we aimed to determine if elevated systemic IgG levels<br/></p>
<p>are sufficient to trigger aging in multiple tissues. To test this,<br/>we administered IgG intraperitoneally to 6-month-old mice and<br/>then evaluated various aging-associated biomarkers (Figure 7A).<br/>Notably, after 100 days of IgG treatment, we observed an accu-<br/>mulation of IgG in the spleen, lymph nodes, hippocampus, lung,<br/>and heart (Figures 7B and 7C). This IgG accumulation led to a<br/>marked senescence phenotype, characterized by increased<br/>levels of the aging marker P21 and the presence of pro-inflam-<br/>matory cytokines and indicators of fibrosis53 (Figures 7D&#8211;7G<br/>and S10X).<br/>Tissue IgG levels are regulated through FcRn (Fcgrt-encoded)<br/></p>
<p>receptor-mediated recycling, the depletion of which is known to<br/>reduce tissue IgG levels.54,55 We then designed an antisense<br/>oligonucleotide (ASO) targeting Fcgrt and administered it intra-<br/>peritoneally to 19-month-old mice (Figures 7H and S10W). After<br/>40 days of treatment, a significant downregulation of FcRn<br/>expression was observed in the spleen, lymph nodes, hippo-<br/>campus, lung, and heart, accompanied by decreased IgG levels<br/>within these tissues (Figures 7I&#8211;7L). Simultaneously, a reduction<br/>in senescence markers was noted across these tissues<br/>(Figures 7M&#8211;7P and S10Y). These results reinforce the idea<br/>that IgG accumulation plays a pivotal role in tissue aging and<br/>suggest that interventions reducing IgG expression could poten-<br/>tially ameliorate systemic aging.<br/></p>
<p>DISCUSSION<br/></p>
<p>Maintenance of organizational spatial compartmentalization is a<br/>hallmark of organismal health, while aging is associated with a<br/>decline in this capacity.56,57 However, our understanding of how<br/>spatial compartmentalization and structural organization are<br/>compromised in aging is incomplete. Here, we introduce GG, the<br/>first high-precision aging spatial transcriptomic atlas that spans<br/>multiple organs and lifetime stages in male mice. Leveraging GG,<br/>we document increased structural entropy in aged tissues, re-<br/>flected as reduced polarity and regionalization and loss of cell<br/>identity. We also introduce SSSs as a conceptual framework to<br/>investigate senescent cells and their interactions with the macro-<br/>and microenvironment. Our characterization of SSS cellular and<br/>molecular signatures revealed that immunoglobulin-producing<br/>cells in various tissues preferentially localize near SSSs, thereby</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>ues across the male mouse lifespan<br/>nt spot types.<br/>ncoding genes that are upregulated in aging are labeled in red.<br/>ld (25-month-old) mouse tissues. Wilcoxon rank-sum test.<br/>sues. Scale bars, 500 and 100 mm (zoom-in) for hippocampus, spinal cord, and<br/></p>
<p>nd old mice (n = 5&#8211;7 mice per group). Data are presented as the mean &#177; SEM.<br/></p>
<p>from SSS.<br/>es during aging.<br/>heir distance from SSS.<br/>, 200 mm.<br/>ouse aging. Each dot represents a sample. Pearson&#8217;s correlation.</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p><b>A<br/></b></p>
<p><b>B<br/></b></p>
<p><b>C<br/></b></p>
<p><b>D<br/></b></p>
<p><b>E<br/></b></p>
<p>(legend on next page)<br/></p>
<p><b>ll<br/></b></p>
<p>Cell 187, 7025&#8211;7044, November 27, 2024 7035<br/></p>
<p>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>forming a pro-inflammatory niche. Furthermore, IgG levels in-<br/>crease with age in tissues of both sexes in mice, and crucially, in<br/>human tissues as well. Functionally, elevated IgG levels induce<br/>macrophages andmicroglia to adopt a senescent and pro-inflam-<br/>matorystate,which in turnexacerbates tissueaging inmice. Taken<br/>together, our findingssuggest thatGGcouldprovidevaluablecon-<br/>tributions to both the aging and immunology fields, given that its<br/>rich spatiotemporal information provides a deeper understanding<br/>of the biological underpinnings of aging and enables us to identify<br/>immunoglobulin-driven immunosenescence as a key contributor<br/>to the aging process.<br/></p>
<p>Spatial transcriptomics methods enable the creation of<br/>detailed molecular maps with location information that is lost in<br/>scRNAseq.19,20,58 Previously, this technology has been utilized<br/>to create spatiotemporal maps of various life stages, including<br/>mouse brain, regenerating axolotl telencephalons, aging ovary,<br/>and brain.21,37,59&#8211;61 Here, we applied it to investigate systemic<br/>aging through the development of algorithms that quantify spatial<br/>structural entropy across nine distinct tissues and organs from<br/>both young and agedmalemice.We show that structural entropy<br/>increases with aging and is particularly pronounced in immune<br/>tissues such as the spleen and lymph nodes. These aging-asso-<br/>ciated changes resulted in disruptions in zone structures, dimin-<br/>ished cell identity, andalterations in cell proportions. Notably, ag-<br/>ing-related alterations in cell composition and gene expression<br/>were closely associated with chronic inflammation, consistent<br/>with and extending from previous studies supporting that inflam-<br/>mation is a prominent feature of systemic aging.4,11,13,17,47,62&#8211;64<br/></p>
<p>The accumulation of senescent cells contributes to tissue and<br/>organismal aging and is a well-known aging hallmark.56,65&#8211;71 Es-<br/>tablished methodologies frequently rely on the identification of<br/>senescent cells through specific biomarkers, such as SA-b-Gal<br/>staining.72 However, these markers alone cannot comprehen-<br/>sively characterize senescent cells and their surrounding envi-<br/>ronment. To delve deeper into the drivers of senescence, we<br/>introduce the concept of the SSS, emphasizing its context-,<br/>location-, and tissue-specific nature. Notably, our research has<br/>revealed a large amount of immunoglobulin-producing cells in<br/>the vicinity of SSSs, which constitute an essential part of the<br/>SSS microenvironment. Furthermore, our findings indicate that<br/>heightened IgG, a key humoral immune factor, may induce<br/>cellular senescence, enhance chronic inflammation, and accel-<br/>erate tissue aging, corroborated by evidence that IgG can pro-<br/>mote fibrosis in adipose tissue.55<br/></p>
<p>Thus, our study sheds light on the dual role of IgG in the<br/>context of aging: while essential for immune protection in and</p>

</div><div><p>Figure 5. Age-related accumulation of immunoglobulin G in multiple ti<br/>(A) Representative images and quantifications of western blot analysis for the pr<br/>male mice aged 2, 4, 13, 19, and 25 months (n = 3 mice per group).<br/>(B) Co-staining of IgG and Igkc in the male mouse hippocampus, spleen, and lymp<br/>15 mm (zoomed in).<br/>(C) Co-staining of IgG and Igkc in the female mouse spleen and lymph node from<br/>(D) Representative images and quantifications of western blot analysis for the pro<br/>(E) Immunofluorescence staining of IgG in the human brain, spleen, and lymph no<br/>lymph node and spleen, 40 and 10 mm (zoomed in). For brain, 50 and 20 mm (zoo<br/>The correlation coefficients and p values were calculated using Pearson&#8217;s correlat<br/>and data are presented as the mean &#177; SEM.<br/>See also Figure S9.<br/></p>
<p>7036 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>of itself, its excessive accumulation may lead to cellular senes-<br/>cence and chronic inflammation, both of which are characteristic<br/>features of aging. Given our discovery that IgG levels rise in<br/>human tissues as aging progresses, we propose that immuno-<br/>globulin-driven processes may represent a critical mechanism<br/>underlying aging, which we term &#8216;&#8216;immunoglobin-associated<br/>senescence phenotype (IASP).&#8217;&#8217; Our results suggest that inter-<br/>ventions targeting IgG accumulation, such as the clearance of<br/>antibody-producing cells,73 ASO-mediated FcRn ablation, or<br/>neutralizing IgG&#8217;s pro-senescent effects on effector cells, may<br/>constitute geroprotective therapeutics. Simultaneously, our re-<br/>sults raise a caution that therapeutics relying on IgG antibodies,<br/>if administered excessively, might induce aging-related<br/>defects.74&#8211;77<br/>In conclusion, our study delivers a high-precision spatiotem-<br/></p>
<p>poral transcriptomic profile of murine aging, spanning diverse<br/>tissues across various chronological stages. Notably, we have<br/>identified the accrual of immunoglobulins as a seminal biomarker<br/>indicative of aging, underscoring its critical role in the broader<br/>context of senescence and age-related conditions.<br/></p>
<p>Limitations of the study<br/>Confronted with the logistical hurdles of analyzing samples from<br/>a range of tissues and five stages, our study prioritized male<br/>mice. Future efforts should target a thorough spatial transcrip-<br/>tomic analysis in female mammals. With our rich dataset, we<br/>foresee the discovery of more aging-related targets and path-<br/>ways. Yet, constrained by article length, we highlighted immuno-<br/>globulin-induced senescence. Further investigation should<br/>explore the mechanisms of IgG in cellular aging and develop<br/>strategies to reduce its levels or counteract its effects, possibly<br/>leading to the development of immunoglobulin-targeted thera-<br/>pies to combat age-related health decline.<br/></p>
<p>RESOURCE AVAILABILITY<br/></p>
<p>Lead contact<br/>Further information and requests for resources and reagents should be<br/>directed to Dr. Guang-Hui Liu (ghliu@ioz.ac.cn).<br/></p>
<p>Materials availability<br/>This study did not generate new unique reagents.<br/></p>
<p>Data and code availability<br/>d The accession number for the spatial transcriptomic data reported in<br/></p>
<p>this paper is in STOmicsDB (https://db.cngb.org/stomics/) under the<br/>accession number STT0000039. The accession number for the raw</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>ssues in mice and humans<br/>otein levels of IgG heavy chain (IgG-H) and IgG light chain (IgG-L) in tissues of<br/></p>
<p>h node from 5 different age groups (n = 4&#8211;5mice per group). Scale bars, 40 and<br/></p>
<p>3 different age groups (n = 5 mice per group). Scale bars, 40 mm.<br/>tein levels of IgG-H in liver and lymph node tissues from 8&#8211;9 human individuals.<br/>des from young and old age groups (n = 5 individuals per group). Scale bars for<br/>med in).<br/>ion analysis in (A) and (D). TheMann-Whitney test was utilized in (B), (C), and (E),</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p><b>A<br/></b></p>
<p><b>C<br/></b></p>
<p><b>B D<br/></b></p>
<p><b>E<br/></b></p>
<p><b>H<br/></b></p>
<p><b>L<br/></b></p>
<p><b>O<br/></b></p>
<p><b>R TS<br/></b></p>
<p><b>P<br/>Q<br/></b></p>
<p><b>M N<br/></b></p>
<p><b>I J K<br/></b></p>
<p><b>F G<br/></b></p>
<p>Figure 6. Immunoglobulin G triggers senescence in macrophages and microglia<br/>(A) Schematic diagram of isolation and treatment of macrophages isolated from mouse bone marrow (bone-marrow-derived macrophage, BMDM), spleen<br/>(spleen derived macrophage, SPDM), and peritoneal cavity (peritoneal cavity-derived macrophage, PCDM).<br/>(B) SA-b-Gal staining of BMDM treated with vehicle (Veh) or IgG. n = 4 biological replicates. Scale bars, 50 mm.<br/></p>
<p>(legend continued on next page)<br/></p>
<p><b>ll<br/></b></p>
<p>Cell 187, 7025&#8211;7044, November 27, 2024 7037<br/></p>
<p>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p>(C)<br/>leve<br/>(D)<br/>(E)<br/>(F)<br/>IgG<br/>(G)<br/>trea<br/>(H)<br/>Pro<br/>(I) R<br/>(J)<br/>leve<br/>(K)<br/>inte<br/>(L)<br/>(M)<br/>(N)<br/>in V<br/>(O)<br/>Veh<br/>(P)<br/>(Q)<br/>are<br/>(R)<br/>(S)<br/>Pro<br/>(T)<br/>The<br/>See<br/></p>
<p><b>ll<br/></b></p>
<p>703<br/></p>
<p>Resource</p>

</div><div><p>bulk RNA-seq data reported in this paper is in GSA (Genome Sequence<br/>Archive) under the accession number CRA012965.<br/></p>
<p>d Custom code supporting this study is available at https://github.com/<br/>zhang-bin-98/A-spatial-transcriptome-landscape-of-mouse-aging.<br/></p>
<p>d Any additional information required to reanalyze the data reported in this<br/>paper is available from the lead contact upon request.</p>

</div><div>
</div><div><p>ACKNOWLEDGMENTS<br/></p>
<p>We would like to express our sincere gratitude to the Aging Biomarker Con-<br/>sortium for their perceptive feedback and suggestions. We are thankful to J.<br/>Lei, Y. Huang, D. Huang, M. Jiang, D. Lv, and G. Sun for their contributions<br/>to the sample collection; J. Jia and C. Xie for their assistance with animal ex-<br/>periments; S. Li and J. Hao for their help with imaging scanning; H. Zheng,<br/>X. Tian, and Q. Ma for their assistance with stereo-seq experiments; and Pro-<br/>fessor X. Qiu for providing advice. We are grateful to the China National<br/>GeneBank for providing us with experiments and a data analysis platform.<br/>Additionally, we also wish to extend our appreciation to L. Tian, L. Bai, J. Lu,<br/>Y. Yang, S. Qiao, Q. Chu, X. Li, J. Chen, R. Bai, and X. Jing for their<br/>administrative support. This work was supported by the National<br/>Natural Science Foundation of China (82125011, 82488301, 82322025,<br/>81921006, and 82122024), the National Key Research and Development<br/>Program of China (2020YFA0804000, 2022YFA1103700, 2020YFA0112200,<br/>2021YFF1201000, 2021YFA1101401, 2022YFA1103800, 2022YFC3602503,<br/>2022YFA1104701, 2023YFC3504301, 2020YFA0113400, and the STI2030-<br/>Major Projects-2021ZD0202400), the National Natural Science Founda-<br/>tion of China (92149301, 92168201, 82330044, 32341001, 92049304,<br/>92049116, 32121001, 82192863, 82361148131, 82271600, 32200610,<br/>82371570, 82201727, 82301758, 32300980, 82071588, 82230043,<br/>82361148130, and 8231101626), the CAS Project for Young Scientists in Basic<br/>Research (YSBR-076, YSBR-012, and YSBR-036), the CAS Youth Interdisci-<br/>plinary Team, the Program of the Beijing Natural Science Foundation<br/>(Z230011, Z240018, JQ24044, and 5242024), Shenzhen Science and Technol-<br/>ogy Program (grant no. RCJC20221008092804002), the Informatization Plan</p>

</div><div><p>Representative western blot images and quantifications of P21 and iNOS prot<br/>ls are illustrated as a heatmap on the right, with a color gradient from white t<br/>Co-staining of MMTV and F4/80 in BMDM treated with Veh or IgG. n = 5 biol<br/>Co-staining of aggresome and F4/80 in BMDM treated with Veh or IgG. n = 5<br/>Co-staining of H3K9me3 and F4/80 in BMDM treated with Veh or IgG (left). Th<br/>-treated groups (R250 cells were calculated per group). Scale bars, 20 mm.<br/>Co-staining of lamin B1 and F4/80 in BMDM treated with Veh or IgG (left). The r<br/>ted groups (R250 cells were calculated per group). Scale bars, 20 mm.<br/>Representative western blot images and quantifications of p-P65 and STAT1<br/>tein levels are illustrated as a heatmap on the right, with a color gradient from<br/>T-qPCR analysis of Fcgr4 expression level in BMDM treated with control or<br/>Representative western blot images and quantifications of FcgRIV protein lev<br/>ls are illustrated as a heatmap on the right, with a color gradient from white t<br/>Immunofluorescence staining of lamin B1 of siRNA-transfected macrophages<br/>nsity values of lamin B1 in each group (si-NC Veh, si-NC IgG, or si-Fcgr4 IgG<br/>SA-b-Gal staining of human macrophage treated with Veh or IgG. n = 5 biolo<br/>Co-staining of HERVK-Env and CD68 in human macrophage treated with Ve<br/>Co-staining of H3K9me3 and CD68 in humanmacrophage treatedwith Veh or I<br/>eh- or IgG-treated groups (R250 cells were calculated per group). Scale bar<br/>Co-staining of lamin B1 and CD68 in humanmacrophage treated with Veh or Ig<br/>- or IgG-treated groups (R250 cells were calculated per group). Scale bars,<br/>SA-b-Gal staining of microglia treated with Veh or IgG. n = 4 biological replic<br/>Representative western blot images and the quantification of P21 protein leve<br/>illustrated as a heatmap on the right, with a color gradient from white to red<br/>Co-staining of aggresome and IBA-1 in microglia treated with Veh or IgG. n =<br/>Representative western blot images and quantifications of p-P65 and STAT1<br/>tein levels are illustrated as a heatmap on the right, with a color gradient from<br/>Co-staining of P21 and IBA-1 in human microglia treated with Veh or IgG. n =<br/>Mann-Whitney test was utilized in (B), (D)&#8211;(G), (I), (K)&#8211;(P), (R), and (T). Data a<br/>also Figure S10.<br/></p>
<p>8 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>of CAS (CAS-WX2022SDC-XK14), the New Cornerstone Science Foundation<br/>through the XPLORER PRIZE (2021-1045), the Beijing Municipal Public Wel-<br/>fare Development and Reform Pilot Project for Medical Research Institutes<br/>(JYY2023-13), the Youth Innovation Promotion Association of CAS<br/>(E1CAZW0401, 2022083, 2023092, and 2020085), the Young Elite Scientists<br/>Sponsorship Program by CAST (2021QNRC001), the Initiative Scientific<br/>Research Program, the Institute of Zoology, the CAS (2023IOZ0102 and<br/>2023IOZ0202), the Beijing Nova Program (20240484493), the Excellent Young<br/>Talents Program of Capital Medical University (12300927), the Key Laboratory<br/>of Alzheimer&#8217;s Disease of Zhejiang Province (ZJAD-2024001), the Project for<br/>Technology Development of Beijing-Affiliated Medical Research Institutes<br/>(11000023T000002036310), and the Excellent Young Talents Training Pro-<br/>gram for the Construction of Beijing Municipal University Teacher Team<br/>(BPHR202203105).</p>

</div><div>
</div><div><p>AUTHOR CONTRIBUTIONS<br/></p>
<p>G.-H.L., W.Z., J.Q., and Y.G. developed the conceptual framework for this<br/>project and supervised the full execution of the experiments. S.M., B.Z., Jiam-<br/>ing Li, and Y. Zuo performed bioinformatics analysis of spatial transcriptomics<br/>and bulk RNA-seq data. Z.J. conducted the animal experiments and conduct-<br/>ed spatial transcriptomics isolation of the mouse&#8217;s tissues. L.G. carried out the<br/>in vitro cell validation experiments. Y.C. and Z.J. performed the mouse in vivo<br/>IgG injection and ASO intervention in mice and the related staining experi-<br/>ments. C.N., Y.S., G.X., Z.L., H.D., and F.W. carried out the spatial transcrip-<br/>tomics sample processing and sequencing experiments. Z.J., Y.C., B.L., F.L.,<br/>L.Z., H.Z., S.S., Y. Zhang, and H.H. performed tissue section and staining ex-<br/>periments. J.A., B.Z., J.Z., and J.H. conducted cell culture. Y. Ye., Y.F., F.Z., Y.<br/>Zhang, and S.S. carried out tissue dissociation from different mouse tissues.<br/>F.Z. facilitated female mouse sample collection. Y. Zhuang, T.L., Y. Yu, Z.B.,<br/>S.P., T.J., and W.S. performed human sample collection. Jingyi Li, C.R.E.,<br/>M.S., S.W., G.Z., J.Y., F.X., and J.C.I.B. helped with supervision of the project.<br/>G.-H.L., W.Z., J.Q., S.M., Z.J., B.Z., L.G., Y.C., Jiaming Li, and Y. Zuo contrib-<br/>uted to writing, reviewing, and editing the manuscript.</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>ein levels in BMDM treated with Veh or IgG. n = 3 biological replicates. Protein<br/>o red denoting expression levels from low to high.<br/>ogical replicates. Scale bars, 20 mm.<br/>biological replicates. Scale bars, 20 mm.<br/>e right panel displays the fluorescence intensity values of H3K9me3 in Veh- or<br/></p>
<p>ight panel displays the fluorescence intensity values of lamin B1 in Veh- or IgG-<br/></p>
<p>protein levels in BMDM treated with Veh or IgG. n = 3 biological replicates.<br/>white to red denoting expression levels from low to high.<br/></p>
<p>siRNA. n = 4 biological replicates.<br/>el in BMDM treated with control or siRNA. n = 3 biological replicates. Protein<br/>o blue denoting expression levels from high to low.<br/>(BMDM) treated with Veh or IgG (left). The right panel displays the fluorescence<br/>treated) (R250 cells were calculated per group). Scale bars, 20 mm.<br/></p>
<p>gical replicates. Scale bars, 50 mm.<br/>h or IgG. n = 5 biological replicates. Scale bars, 20 mm.<br/>gG (left). The right panel displays the fluorescence intensity values of H3K9me3<br/>s, 20 mm.<br/>G (left). The right panel displays the fluorescence intensity values of lamin B1 in<br/>20 mm.<br/>ates. Scale bars, 50 mm.<br/>ls in microglia treated with Veh or IgG. n = 3 biological replicates. Protein levels<br/>denoting expression levels from low to high.<br/>5 biological replicates. Scale bars, 20 mm.<br/>protein levels in microglia treated with Veh or IgG. n = 3 biological replicates.<br/>white to red denoting expression levels from low to high.<br/>5 biological replicates. Scale bars, 20 mm.<br/>re presented as the mean &#177; SEM. The arrows indicate positive cells.</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p><b>A B<br/></b></p>
<p><b>C D<br/></b></p>
<p><b>E F G<br/></b></p>
<p><b>H I<br/></b></p>
<p><b>J K<br/></b></p>
<p><b>L M<br/></b></p>
<p><b>N O P<br/></b></p>
<p>(legend on next page)<br/></p>
<p><b>ll<br/></b></p>
<p>Cell 187, 7025&#8211;7044, November 27, 2024 7039<br/></p>
<p>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>DECLARATION OF INTERESTS<br/></p>
<p>Y.G., C.N., Y.S., Z.L., H.D., and F.W. are employees of BGI. J.C.I.B. and C.R.E.<br/>are employees of Altos Labs.<br/></p>
<p>STAR+METHODS<br/></p>
<p>Detailed methods are provided in the online version of this paper and include<br/>the following:<br/></p>
<p>d KEY RESOURCES TABLE<br/>d EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS</p>

</div><div><p>Fig<br/>(A)<br/>(B)<br/>(C)<br/>(D)<br/>and<br/>(E)<br/>(F) I<br/>40<br/>(G)<br/>(H)<br/>(I) Im<br/>(J) S<br/>(K)<br/>(L)<br/>(M)<br/>(zoo<br/>(N)<br/>(O)<br/>and<br/>(P)<br/>sple<br/>The<br/>See<br/></p>
<p>704</p>

</div><div><p>B Ethical statement<br/>B ASO treatment of old mice<br/>B Experimental mouse models<br/>B Human samples<br/></p>
<p>d METHOD DETAILS<br/>B IgG treatment of young mice<br/>B Tissue collection<br/>B Stereo-seq tissue processing<br/>B Library preparation and sequencing<br/>B Senescence-associated b-Galactosidase (SA-b-Gal) staining<br/>B SPiDER-bGal staining<br/>B Immunofluorescence staining<br/>B Immunohistochemistry staining<br/>B Masson staining<br/>B Hematoxylin and eosin (HE) staining<br/>B Aggresome staining<br/>B RNA isolation and analyses<br/>B Western blot analysis<br/>B Enzyme linked immunosorbent assay<br/>B Nitrite measurement<br/>B Mouse macrophage isolation and culture<br/>B Mouse microglia isolation and culture<br/>B Human macrophage isolation and culture<br/>B Human microglia isolation and culture<br/>B SiRNA and ASO transfection and analyses<br/>B Stereo-seq raw data processing<br/>B Establishment of spatial transcriptomic atlas of multiple mouse<br/></p>
<p>tissues<br/>B Quality control, integration, clustering, and spot type identification<br/></p>
<p>of spatial transcriptomic data for each mouse tissue</p>

</div><div><p>ure 7. Accumulation of immunoglobulin G drives multi-tissue aging in<br/>A schematic illustrating the treatment of 6-month-old mice with Veh or IgG.<br/>Immunostaining of IgG in multiple tissues from young mice treated with Veh or Ig<br/>Statistical analysis of the fold change of IgG-positive cells in multiple tissues (n =<br/>Immunohistochemical analysis of P21 expression in multiple tissues treated with V<br/>20 mm (zoomed in).<br/></p>
<p>Statistical analysis of the fold change of P21-positive cells in multiple tissues (n =<br/>mmunohistochemical staining of S100A8 in the spleen and lymph node from youn<br/>mm (zoomed in).<br/>Immunohistochemical staining of IL-1b in the lymph node treated with Veh or IgG<br/>A schematic illustrating the treatment of 19-month-old mice with Veh or IgG.<br/>munostaining of FcRn in multiple tissues from old mice treated with ASO-NC or<br/>tatistical analysis of the fold change of FcRn-positive cells in multiple tissues (n<br/>Immunostaining of IgG in multiple tissues from old mice treated with ASO-NC or<br/>Statistical analysis of the fold change of IgG-positive cells in multiple tissues (n =<br/>Immunohistochemical analysis of P21 expression in multiple tissues from old mic<br/>med in); for others, 50 and 20 mm (zoomed in).<br/>Statistical analysis of the fold change of P21-positive cells in multiple tissues (n =<br/>Immunohistochemical analysis of S100A8 expression in the spleen from oldmice tr<br/>50 mm (zoomed in).<br/></p>
<p>Immunohistochemical staining of IL-1b in the spleen and lymph node from old mi<br/>en, 80 and 40 mm (zoomed in); for lymph node, 50 and 20 mm (zoomed in).<br/>Mann-Whitney test was utilized in (C), (E)&#8211;(G), (J), (L), and (N)&#8211;(P). Data are prese<br/>also Figure S10.<br/></p>
<p>0 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>B Adjacency score analysis<br/>B Spot type proportion and proximity analysis<br/>B Analysis and calculation of organizational structure entropy<br/>B Adjustment of entropy<br/>B OSE range definition<br/>B Cell identity analysis<br/>B Ighigh and Ighhigh spot definition<br/>B Spatially connected domain identification analysis<br/>B Spatial location aging-related differentially expression gene<br/></p>
<p>analysis<br/>B Aging score and senescence-sensitive spot (SSS) analysis<br/>B SSS niche analysis and gradient definition<br/>B SSS niche gradient entropy definition<br/>B Senescence-associated secretory phenotype (SASP) score<br/></p>
<p>analysis<br/>B Pathway enrichment analysis and spatial mapping<br/>B Distance gradient gene analysis<br/>B Bulk RNA-seq analysis<br/>B Ingenuity pathway analysis of upstream regulators<br/>B Repeat element expression level analysis<br/></p>
<p>d QUANTIFICATION AND STATISTICAL ANALYSIS OF THE EXPERI-<br/>MENTAL DATA</p>

</div><div>
</div><div><p>SUPPLEMENTAL INFORMATION<br/></p>
<p>Supplemental information can be found online at https://doi.org/10.1016/j.cell.<br/>2024.10.019.<br/></p>
<p>Received: October 19, 2023<br/>Revised: July 29, 2024<br/>Accepted: October 12, 2024<br/>Published: November 4, 2024</p>

</div><div>
</div><div><p>REFERENCES<br/></p>
<p>1. Zhang, W., Che, Y., Tang, X., Chen, S., Song, M., Wang, L., Sun, A.-J.,<br/>Chen, H.-Z., Xu, M., Wang, M., et al.; Aging Biomarker Consortium<br/>(2023). A biomarker framework for cardiac aging: the Aging Biomarker<br/>Consortium consensus statement. Life Med. 2, lnad035. https://doi.<br/>org/10.1093/lifemedi/lnad035.</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>male mice<br/></p>
<p>G. Scale bars, 50 and 10 mm (zoomed in).<br/>5 mice per group).<br/>eh or IgG. Scale bars for spleen, 80 and 40 mm (zoomed in); for others, 50<br/></p>
<p>5&#8211;6 mice per group).<br/>g mice treated with Veh or IgG. n = 6&#8211;9 mice per group. Scale bars, 80 and<br/></p>
<p>. n = 5 mice per group. Scale bars, 50 and 20 mm (zoomed in).<br/></p>
<p>ASO-Fcgrt. Scale bars, 40 and 15 mm (zoomed in).<br/>= 5&#8211;6 mice per group).<br/>ASO-Fcgrt. Scale bars, 40 and 15 mm (zoomed in).<br/>5&#8211;6 mice per group).<br/>e treated with ASO-NC or ASO-Fcgrt. Scale bars for spleen, 80 and 40 mm<br/></p>
<p>5&#8211;10 mice per group).<br/>eatedwith ASO-NC or ASO-Fcgrt. n = 8&#8211;10mice per group Scale bars, 100<br/></p>
<p>ce treated with ASO-NC or ASO-Fcgrt. n = 8&#8211;10 per group. Scale bars, for<br/></p>
<p>nted as the mean &#177; SEM. The arrows indicate positive cells.</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>2. Jiang, M., Zheng, Z., Wang, X., Chen, Y., Qu, J., Ding, Q., Zhang, W., Liu,<br/>Y.-S., Yang, J., Tang, W., et al.; Aging Biomarker Consortium (2024). A<br/>biomarker framework for liver aging: the Aging Biomarker Consortium<br/>consensus statement. Life Med. 3, lnae004. https://doi.org/10.1093/life-<br/>medi/lnae004.<br/></p>
<p>3. Zhang, L., Guo, J., Liu, Y., Sun, S., Liu, B., Yang, Q., Tao, J., Tian, X.-L.,<br/>Pu, J., et al.; Aging Biomarker Consortium (2023). A framework of bio-<br/>markers for vascular aging: a consensus statement by the Aging<br/>Biomarker Consortium. Life Med. 2, lnad033. https://doi.org/10.1093/<br/>lifemedi/lnad033.<br/></p>
<p>4. Lo&#769;pez-Ot&#305;&#769;n, C., Blasco, M.A., Partridge, L., Serrano, M., and Kroemer, G.<br/>(2023). Hallmarks of aging: An expanding universe. Cell 186, 243&#8211;278.<br/>https://doi.org/10.1016/j.cell.2022.11.001.<br/></p>
<p>5. Yousefzadeh, M.J., Flores, R.R., Zhu, Y., Schmiechen, Z.C., Brooks,<br/>R.W., Trussoni, C.E., Cui, Y., Angelini, L., Lee, K.A., McGowan, S.J.,<br/>et al. (2021). An aged immune system drives senescence and ageing of<br/>solid organs. Nature 594, 100&#8211;105. https://doi.org/10.1038/s41586-<br/>021-03547-7.<br/></p>
<p>6. Rando, T.A., and Wyss-Coray, T. (2021). Asynchronous, contagious and<br/>digital aging. Nat Aging 1, 29&#8211;35. https://doi.org/10.1038/s43587-020-<br/>00015-1.<br/></p>
<p>7. Zhang, W., Wan, H., Feng, G., Qu, J., Wang, J., Jing, Y., Ren, R., Liu, Z.,<br/>Zhang, L., Chen, Z., et al. (2018). SIRT6 deficiency results in develop-<br/>mental retardation in cynomolgus monkeys. Nature 560, 661&#8211;665.<br/>https://doi.org/10.1038/s41586-018-0437-z.<br/></p>
<p>8. Bao, H., Cao, J., Chen,M., Chen,M., Chen,W., Chen, X., Chen, Y., Chen,<br/>Y., Chen, Y., Chen, Z., et al. (2023). Biomarkers of aging. Sci. China Life<br/>Sci. 66, 893&#8211;1066. https://doi.org/10.1007/s11427-023-2305-0.<br/></p>
<p>9. Sun, S., Jiang,M., Ma, S., Ren, J., and Liu, G.H. (2024). Exploring the het-<br/>erogeneous targets of metabolic aging at single-cell resolution. Trends<br/>Endocrinol. Metab. https://doi.org/10.1016/j.tem.2024.07.009.<br/></p>
<p>10. Cai, Y., Song, W., Li, J., Jing, Y., Liang, C., Zhang, L., Zhang, X., Zhang,<br/>W., Liu, B., An, Y., et al. (2022). The landscape of aging. Sci. China Life<br/>Sci. 65, 2354&#8211;2454. https://doi.org/10.1007/s11427-022-2161-3.<br/></p>
<p>11. Tabula; Muris Consortium (2020). A single-cell transcriptomic atlas char-<br/>acterizes ageing tissues in the mouse. Nature 583, 590&#8211;595. https://doi.<br/>org/10.1038/s41586-020-2496-1.<br/></p>
<p>12. Davie, K., Janssens, J., Koldere, D., De Waegeneer, M., Pech, U., Kreft,<br/>q., Aibar, S., Makhzami, S., Christiaens, V., Bravo Gonza&#769;lez-Blas, C.,<br/>et al. (2018). A Single-Cell Transcriptome Atlas of the Aging Drosophila<br/>Brain. Cell 174, 982&#8211;998.e20. https://doi.org/10.1016/j.cell.2018.05.057.<br/></p>
<p>13. Ma, S., Sun, S., Geng, L., Song, M., Wang, W., Ye, Y., Ji, Q., Zou, Z.,<br/>Wang, S., He, X., et al. (2020). Caloric Restriction Reprograms the<br/>Single-Cell Transcriptional Landscape of Rattus Norvegicus Aging. Cell<br/>180, 984&#8211;1001.e22. https://doi.org/10.1016/j.cell.2020.02.008.<br/></p>
<p>14. Wang, R., Zhang, P., Wang, J., Ma, L., E, W., Suo, S., Jiang, M., Li, J.,<br/>Chen, H., Sun, H., et al. (2023). Construction of a cross-species cell land-<br/>scape at single-cell level. Nucleic Acids Res. 51, 501&#8211;516. https://doi.<br/>org/10.1093/nar/gkac633.<br/></p>
<p>15. Lu, T.C., Brbi&#1;c, M., Park, Y.J., Jackson, T., Chen, J., Kolluru, S.S., Qi, Y.,<br/>Katheder, N.S., Cai, X.T., Lee, S., et al. (2023). Aging Fly Cell Atlas iden-<br/>tifies exhaustive aging features at cellular resolution. Science 380,<br/>eadg0934. https://doi.org/10.1126/science.adg0934.<br/></p>
<p>16. Zhang, B., Lee, D.E., Trapp, A., Tyshkovskiy, A., Lu, A.T., Bareja, A., Ker-<br/>epesi, C., McKay, L.K., Shindyapina, A.V., Dmitriev, S.E., et al. (2023).<br/>Multi-omic rejuvenation and life span extension on exposure to youthful<br/>circulation. Nat Aging 3, 948&#8211;964. https://doi.org/10.1038/s43587-023-<br/>00451-9.<br/></p>
<p>17. Ma, S., Wang, S., Ye, Y., Ren, J., Chen, R., Li, W., Li, J., Zhao, L., Zhao,<br/>Q., Sun, G., et al. (2022). Heterochronic parabiosis induces stem cell revi-<br/>talization and systemic rejuvenation across aged tissues. Cell Stem Cell<br/>29, 990&#8211;1005.e10. https://doi.org/10.1016/j.stem.2022.04.017.</p>

</div><div>
</div><div><p>18. Yang, Y., Lu, X., Liu, N., Ma, S., Zhang, H., Zhang, Z., Yang, K., Jiang, M.,<br/>Zheng, Z., Qiao, Y., et al. (2024). Metformin decelerates aging clock in<br/>male monkeys. Cell. https://doi.org/10.1016/j.cell.2024.08.021.<br/></p>
<p>19. Moses, L., and Pachter, L. (2022). Museum of spatial transcriptomics. Nat.<br/>Methods 19, 534&#8211;546. https://doi.org/10.1038/s41592-022-01409-2.<br/></p>
<p>20. Rao, A., Barkley, D., Franc&#807;a, G.S., and Yanai, I. (2021). Exploring tissue<br/>architecture using spatial transcriptomics. Nature 596, 211&#8211;220.<br/>https://doi.org/10.1038/s41586-021-03634-9.<br/></p>
<p>21. Chen, A., Liao, S., Cheng, M., Ma, K., Wu, L., Lai, Y., Qiu, X., Yang, J., Xu,<br/>J., Hao, S., et al. (2022). Spatiotemporal transcriptomic atlas of mouse<br/>organogenesis using DNA nanoball-patterned arrays. Cell 185, 1777&#8211;<br/>1792.e21. https://doi.org/10.1016/j.cell.2022.04.003.<br/></p>
<p>22. Chen, W.T., Lu, A., Craessaerts, K., Pavie, B., Sala Frigerio, C., Corthout,<br/>N., Qian, X., Lala&#769;kova&#769;, J., Ku&#776;hnemund, M., Voytyuk, I., et al. (2020).<br/>Spatial Transcriptomics and In Situ Sequencing to Study Alzheimer&#8217;s<br/>Disease. Cell 182, 976&#8211;991.e19. https://doi.org/10.1016/j.cell.2020.<br/>06.038.<br/></p>
<p>23. Kuppe, C., Ramirez Flores, R.O., Li, Z., Hayat, S., Levinson, R.T., Liao, X.,<br/>Hannani, M.T., Tanevski, J., Wu&#776;nnemann, F., Nagai, J.S., et al. (2022).<br/>Spatial multi-omic map of human myocardial infarction. Nature 608,<br/>766&#8211;777. https://doi.org/10.1038/s41586-022-05060-x.<br/></p>
<p>24. Calcinotto, A., Kohli, J., Zagato, E., Pellegrini, L., Demaria, M., and Ali-<br/>monti, A. (2019). Cellular Senescence: Aging, Cancer, and Injury. Physiol.<br/>Rev. 99, 1047&#8211;1078. https://doi.org/10.1152/physrev.00020.2018.<br/></p>
<p>25. He, S., and Sharpless, N.E. (2017). Senescence in Health and Disease.<br/>Cell 169, 1000&#8211;1011. https://doi.org/10.1016/j.cell.2017.05.015.<br/></p>
<p>26. Moiseeva, V., Cisneros, A., Sica, V., Deryagin, O., Lai, Y., Jung, S., An-<br/>dre&#769;s, E., An, J., Segale&#769;s, J., Ortet, L., et al. (2023). Senescence atlas re-<br/>veals an aged-like inflamed niche that blunts muscle regeneration. Na-<br/>ture 613, 169&#8211;178. https://doi.org/10.1038/s41586-022-05535-x.<br/></p>
<p>27. van Deursen, J.M. (2014). The role of senescent cells in ageing. Nature<br/>509, 439&#8211;446. https://doi.org/10.1038/nature13193.<br/></p>
<p>28. Zhang, W., Li, J., Suzuki, K., Qu, J., Wang, P., Zhou, J., Liu, X., Ren, R.,<br/>Xu, X., Ocampo, A., et al. (2015). Aging stem cells. A Werner syndrome<br/>stem cell model unveils heterochromatin alterations as a driver of<br/>human aging. Science 348, 1160&#8211;1163. https://doi.org/10.1126/sci-<br/>ence.aaa1356.<br/></p>
<p>29. Gorgoulis, V., Adams, P.D., Alimonti, A., Bennett, D.C., Bischof, O.,<br/>Bishop, C., Campisi, J., Collado, M., Evangelou, K., Ferbeyre, G., et al.<br/>(2019). Cellular Senescence: Defining a Path Forward. Cell 179,<br/>813&#8211;827. https://doi.org/10.1016/j.cell.2019.10.005.<br/></p>
<p>30. Gurkar, A.U., Gerencser, A.A., Mora, A.L., Nelson, A.C., Zhang, A.R.,<br/>Lagnado, A.B., Enninful, A., Benz, C., Furman, D., Beaulieu, D., et al.<br/>(2023). Spatial mapping of cellular senescence: emerging challenges<br/>and opportunities. Nat Aging 3, 776&#8211;790. https://doi.org/10.1038/<br/>s43587-023-00446-6.<br/></p>
<p>31. Di Micco, R., Krizhanovsky, V., Baker, D., and d&#8217;Adda di Fagagna, F.<br/>(2021). Cellular senescence in ageing: from mechanisms to therapeutic<br/>opportunities. Nat. Rev. Mol. Cell Biol. 22, 75&#8211;95. https://doi.org/10.<br/>1038/s41580-020-00314-w.<br/></p>
<p>32. Hernandez-Segura, A., Nehme, J., and Demaria, M. (2018). Hallmarks of<br/>Cellular Senescence. Trends Cell Biol. 28, 436&#8211;453. https://doi.org/10.<br/>1016/j.tcb.2018.02.001.<br/></p>
<p>33. Lo&#769;pez-Ot&#305;&#769;n, C., Pietrocola, F., Roiz-Valle, D., Galluzzi, L., and Kroemer,<br/>G. (2023). Meta-hallmarks of aging and cancer. Cell Metab. 35, 12&#8211;35.<br/>https://doi.org/10.1016/j.cmet.2022.11.001.<br/></p>
<p>34. Jia, Y.-J., Wang, J., Ren, J.-R., Chan, P., Chen, S., Chen, X.-C., Chhetri,<br/>J.K., Guo, J., Guo, Q., et al.; Aging Biomarker Consortium (2023). A<br/>framework of biomarkers for brain aging: a consensus statement by<br/>the Aging Biomarker Consortium. Life Med. 2, lnad017. https://doi.org/<br/>10.1093/lifemedi/lnad017.<br/></p>
<p>35. Zhang, H., Li, J., Ren, J., Sun, S., Ma, S., Zhang, W., Yu, Y., Cai, Y., Yan,<br/>K., Li, W., et al. (2021). Single-nucleus transcriptomic landscape of</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Cell 187, 7025&#8211;7044, November 27, 2024 7041</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>primate hippocampal aging. Protein Cell 12, 695&#8211;716. https://doi.org/10.<br/>1007/s13238-021-00852-9.<br/></p>
<p>36. Zhang, Y., Zheng, Y., Wang, S., Fan, Y., Ye, Y., Jing, Y., Liu, Z., Yang, S.,<br/>Xiong, M., Yang, K., et al. (2023). Single-nucleus transcriptomics reveals<br/>a gatekeeper role for FOXP1 in primate cardiac aging. Protein Cell 14,<br/>279&#8211;293. https://doi.org/10.1093/procel/pwac038.<br/></p>
<p>37. Wei, X., Fu, S., Li, H., Liu, Y., Wang, S., Feng, W., Yang, Y., Liu, X., Zeng,<br/>Y.Y., Cheng, M., et al. (2022). Single-cell Stereo-seq reveals induced<br/>progenitor cells involved in axolotl brain regeneration. Science 377,<br/>eabp9444. https://doi.org/10.1126/science.abp9444.<br/></p>
<p>38. Chen, A., Sun, Y., Lei, Y., Li, C., Liao, S., Meng, J., Bai, Y., Liu, Z., Liang,<br/>Z., Zhu, Z., et al. (2023). Single-cell spatial transcriptome reveals cell-<br/>type organization in the macaque cortex. Cell 186, 3726&#8211;3743.e24.<br/>https://doi.org/10.1016/j.cell.2023.06.009.<br/></p>
<p>39. Avelar, R.A., Ortega, J.G., Tacutu, R., Tyler, E.J., Bennett, D., Binetti, P.,<br/>Budovsky, A., Chatsirisupachai, K., Johnson, E., Murray, A., et al. (2020).<br/>A multidimensional systems biology analysis of cellular senescence in<br/>aging and disease. Genome Biol. 21, 91. https://doi.org/10.1186/<br/>s13059-020-01990-9.<br/></p>
<p>40. Liu, G.-H., Bao, Y., Qu, J., Zhang, W., Zhang, T., Kang, W., Yang, F., Ji,<br/>Q., Jiang, X., Ma, Y., et al. (2021). Aging Atlas: a multi-omics database for<br/>aging biology. Nucleic Acids Res. 49, D825&#8211;D830. https://doi.org/10.<br/>1093/nar/gkaa894.<br/></p>
<p>41. Saul, D., Kosinsky, R.L., Atkinson, E.J., Doolittle, M.L., Zhang, X.,<br/>LeBrasseur, N.K., Pignolo, R.J., Robbins, P.D., Niedernhofer, L.J., Ikeno,<br/>Y., et al. (2022). A new gene set identifies senescent cells and predicts<br/>senescence-associated pathways across tissues. Nat. Commun. 13,<br/>4827. https://doi.org/10.1038/s41467-022-32552-1.<br/></p>
<p>42. Fridman, A.L., and Tainsky, M.A. (2008). Critical pathways in cellular<br/>senescence and immortalization revealed by gene expression profiling.<br/>Oncogene 27, 5975&#8211;5987. https://doi.org/10.1038/onc.2008.213.<br/></p>
<p>43. Liu, X., Liu, Z., Wu, Z., Ren, J., Fan, Y., Sun, L., Cao, G., Niu, Y., Zhang, B.,<br/>Ji, Q., et al. (2023). Resurrection of endogenous retroviruses during aging<br/>reinforces senescence. Cell 186, 287&#8211;304.e26. https://doi.org/10.1016/j.<br/>cell.2022.12.017.<br/></p>
<p>44. Zhang, H., Li, J., Yu, Y., Ren, J., Liu, Q., Bao, Z., Sun, S., Liu, X., Ma, S.,<br/>Liu, Z., et al. (2023). Nuclear lamina erosion-induced resurrection<br/>of endogenous retroviruses underlies neuronal aging. Cell Rep. 42,<br/>112593. https://doi.org/10.1016/j.celrep.2023.112593.<br/></p>
<p>45. Zhou, S., Liu, L., and Lu, X. (2023). Endogenous retroviruses make aging<br/>go viral. Life Med. 2, lnad001. https://doi.org/10.1093/lifemedi/lnad001.<br/></p>
<p>46. Wu, Z., Qu, J., and Liu, G.H. (2024). Roles of chromatin and genome<br/>instability in cellular senescence and their relevance to ageing and<br/>related diseases. Nat. Rev. Mol. Cell Biol. https://doi.org/10.1038/<br/>s41580-024-00775-3.<br/></p>
<p>47. Sun, S., Ma, S., Cai, Y., Wang, S., Ren, J., Yang, Y., Ping, J., Wang, X.,<br/>Zhang, Y., Yan, H., et al. (2023). A single-cell transcriptomic atlas of<br/>exercise-induced anti-inflammatory and geroprotective effects across<br/>the body. Innovation (Camb) 4, 100380. https://doi.org/10.1016/j.xinn.<br/>2023.100380.<br/></p>
<p>48. Pa&#769;lovics, R., Keller, A., Schaum, N., Tan, W., Fehlmann, T., Borja, M.,<br/>Kern, F., Bonanno, L., Calcuttawala, K., Webber, J., et al. (2022). Molec-<br/>ular hallmarks of heterochronic parabiosis at single-cell resolution. Na-<br/>ture 603, 309&#8211;314. https://doi.org/10.1038/s41586-022-04461-2.<br/></p>
<p>49. Liu, L., Kim, S., Buckley, M.T., Reyes, J.M., Kang, J., Tian, L., Wang, M.,<br/>Lieu, A., Mao, M., Rodriguez-Mateo, C., et al. (2023). Exercise repro-<br/>grams the inflammatory landscape of multiple stem cell compartments<br/>during mammalian aging. Cell Stem Cell 30, 689&#8211;705.e4. https://doi.<br/>org/10.1016/j.stem.2023.03.016.<br/></p>
<p>50. Liu, X., Jiao, H., Zhang, B., Zhang, S., Yan, K., Qu, J., Zhang, W., Yu, L.,<br/>and Liu, G.-H. (2023). Migrasomes trigger innate immune activation and<br/>mediate transmission of senescence signals across human cells. Life<br/>Med. 2, lnad050. https://doi.org/10.1093/lifemedi/lnad050.</p>

</div><div><p>7042 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>51. Li, M.H., Jiang, X., Jing, Y., Yan, K., Bi, S.J., Wang, S., Ma, S., Liu, G.H.,<br/>Zhang, W., Sun, S., and Qu, J. (2024). CRISPR-based screening pin-<br/>points H2AZ1 as a driver of senescence in human mesenchymal stem<br/>cells. Protein Cell, pwae035. https://doi.org/10.1093/procel/pwae035.<br/></p>
<p>52. He, Y., Ji, Q., Wu, Z., Cai, Y., Yin, J., Zhang, Y., Zhang, S., Liu, X., Zhang,<br/>W., Liu, G.-H., et al. (2023). 4E-BP1 counteracts human mesenchymal<br/>stem cell senescence via maintaining mitochondrial homeostasis. Pro-<br/>tein Cell 14, 202&#8211;216. https://doi.org/10.1093/procel/pwac037.<br/></p>
<p>53. Zhang, B., Yan, H., Liu, X., Sun, L., Ma, S., Wang, S., Qu, J., Liu, G.-H.,<br/>and Zhang, W. (2023). SenoIndex: S100A8/S100A9 as a novel aging<br/>biomarker. Life Med. 2, lnad022. https://doi.org/10.1093/lifemedi/<br/>lnad022.<br/></p>
<p>54. Delidakis, G., Kim, J.E., George, K., and Georgiou, G. (2022). Improving<br/>Antibody Therapeutics by Manipulating the Fc Domain: Immunological<br/>and Structural Considerations. Annu. Rev. Biomed. Eng. 24, 249&#8211;274.<br/>https://doi.org/10.1146/annurev-bioeng-082721-024500.<br/></p>
<p>55. Yu, L., Wan, Q., Liu, Q., Fan, Y., Zhou, Q., Skowronski, A.A., Wang, S.,<br/>Shao, Z., Liao, C.Y., Ding, L., et al. (2024). IgG is an aging factor that<br/>drives adipose tissue fibrosis and metabolic decline. Cell Metab. 36,<br/>793&#8211;807.e5. https://doi.org/10.1016/j.cmet.2024.01.015.<br/></p>
<p>56. Schmauck-Medina, T., Molie&#768;re, A., Lautrup, S., Zhang, J., Chlopicki, S.,<br/>Madsen, H.B., Cao, S., Soendenbroe, C., Mansell, E., Vestergaard, M.B.,<br/>et al. (2022). New hallmarks of ageing: a 2022 Copenhagen ageing<br/>meeting summary. Aging (Albany, NY) 14, 6829&#8211;6839. https://doi.org/<br/>10.18632/aging.204248.<br/></p>
<p>57. Lo&#769;pez-Ot&#305;&#769;n, C., and Kroemer, G. (2021). Hallmarks of Health. Cell 184,<br/>33&#8211;63. https://doi.org/10.1016/j.cell.2020.11.034.<br/></p>
<p>58. Zhang, J., Yin, J., Heng, Y., Xie, K., Chen, A., Amit, I., Bian, X.-w., and Xu,<br/>X. (2022). Spatiotemporal Omics-Refining the landscape of precision<br/>medicine. Life Med. 1, 84&#8211;102. https://doi.org/10.1093/lifemedi/lnac053.<br/></p>
<p>59. Hahn, O., Foltz, A.G., Atkins, M., Kedir, B., Moran-Losada, P., Guldner,<br/>I.H., Munson, C., Kern, F., Pa&#769;lovics, R., Lu, N., et al. (2023). Atlas of the<br/>aging mouse brain reveals white matter as vulnerable foci. Cell 186,<br/>4117&#8211;4133.e22. https://doi.org/10.1016/j.cell.2023.07.027.<br/></p>
<p>60. Allen,W.E., Blosser, T.R., Sullivan, Z.A., Dulac, C., and Zhuang, X. (2023).<br/>Molecular and spatial signatures of mouse brain aging at single-cell res-<br/>olution. Cell 186, 194&#8211;208.e18. https://doi.org/10.1016/j.cell.2022.<br/>12.010.<br/></p>
<p>61. Lu, H., Jing, Y., Zhang, C., Ma, S., Zhang, W., Huang, D., Zhang, B., Zuo,<br/>Y., Qin, Y., Liu, G.-H., et al. (2024). Aging hallmarks of the primate ovary<br/>revealed by spatiotemporal transcriptomics. Protein Cell 15, 364&#8211;384.<br/>https://doi.org/10.1093/procel/pwad063.<br/></p>
<p>62. Uyar, B., Palmer, D., Kowald, A., Murua Escobar, H., Barrantes, I., Mo&#776;ller,<br/>S., Akalin, A., and Fuellen, G. (2020). Single-cell analyses of aging, inflam-<br/>mation and senescence. Ageing Res. Rev. 64, 101156. https://doi.org/<br/>10.1016/j.arr.2020.101156.<br/></p>
<p>63. Ma, S., Sun, S., Li, J., Fan, Y., Qu, J., Sun, L., Wang, S., Zhang, Y., Yang,<br/>S., Liu, Z., et al. (2021). Single-cell transcriptomic atlas of primate cardio-<br/>pulmonary aging. Cell Res. 31, 415&#8211;432. https://doi.org/10.1038/<br/>s41422-020-00412-6.<br/></p>
<p>64. Kubben, N., Zhang, W., Wang, L., Voss, T.C., Yang, J., Qu, J., Liu, G.H.,<br/>and Misteli, T. (2016). Repression of the Antioxidant NRF2 Pathway in<br/>Premature Aging. Cell 165, 1361&#8211;1374. https://doi.org/10.1016/j.cell.<br/>2016.05.017.<br/></p>
<p>65. Yousefzadeh, M.J., Zhao, J., Bukata, C., Wade, E.A., McGowan, S.J.,<br/>Angelini, L.A., Bank, M.P., Gurkar, A.U., McGuckian, C.A., Calubag,<br/>M.F., et al. (2020). Tissue specificity of senescent cell accumulation dur-<br/>ing physiologic and accelerated aging of mice. Aging Cell 19, e13094.<br/>https://doi.org/10.1111/acel.13094.<br/></p>
<p>66. Tchkonia, T., Zhu, Y., van Deursen, J., Campisi, J., and Kirkland, J.L.<br/>(2013). Cellular senescence and the senescent secretory phenotype:<br/>therapeutic opportunities. J. Clin. Invest. 123, 966&#8211;972. https://doi.org/<br/>10.1172/JCI64098.</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>67. Baker, D.J., Wijshake, T., Tchkonia, T., LeBrasseur, N.K., Childs, B.G.,<br/>van de Sluis, B., Kirkland, J.L., and van Deursen, J.M. (2011). Clearance<br/>of p16Ink4a-positive senescent cells delays ageing-associated disor-<br/>ders. Nature 479, 232&#8211;236. https://doi.org/10.1038/nature10600.<br/></p>
<p>68. Xu, M., Pirtskhalava, T., Farr, J.N., Weigand, B.M., Palmer, A.K., Wei-<br/>voda, M.M., Inman, C.L., Ogrodnik, M.B., Hachfeld, C.M., Fraser, D.G.,<br/>et al. (2018). Senolytics improve physical function and increase lifespan<br/>in old age. Nat. Med. 24, 1246&#8211;1256. https://doi.org/10.1038/s41591-<br/>018-0092-9.<br/></p>
<p>69. Campisi, J. (2013). Aging, cellular senescence, and cancer. Annu. Rev.<br/>Physiol. 75, 685&#8211;705. https://doi.org/10.1146/annurev-physiol-030212-<br/>183653.<br/></p>
<p>70. Kennedy, B.K., Berger, S.L., Brunet, A., Campisi, J., Cuervo, A.M., Epel,<br/>E.S., Franceschi, C., Lithgow, G.J., Morimoto, R.I., Pessin, J.E., et al.<br/>(2014). Geroscience: linking aging to chronic disease. Cell 159,<br/>709&#8211;713. https://doi.org/10.1016/j.cell.2014.10.039.<br/></p>
<p>71. Cai, Y., Ji, Z., Wang, S., Zhang, W., Qu, J., Belmonte, J.C.I., and Liu,<br/>G.-H. (2022). Genetic enhancement: an avenue to combat aging-related<br/>diseases. Life Med. 1, 307&#8211;318. https://doi.org/10.1093/lifemedi/<br/>lnac054.<br/></p>
<p>72. Gonza&#769;lez-Gualda, E., Baker, A.G., Fruk, L., and Mun&#771;oz-Esp&#305;&#769;n, D. (2021).<br/>A guide to assessing cellular senescence in vitro and in vivo. FEBS Jour-<br/>nal 288, 56&#8211;80. https://doi.org/10.1111/febs.15570.<br/></p>
<p>73. Keren, Z., Naor, S., Nussbaum, S., Golan, K., Itkin, T., Sasaki, Y.,<br/>Schmidt-Supprian, M., Lapidot, T., and Melamed, D. (2011). B-cell<br/>depletion reactivates B lymphopoiesis in the BM and rejuvenates the B<br/>lineage in aging. Blood 117, 3104&#8211;3112. https://doi.org/10.1182/blood-<br/>2010-09-307983.<br/></p>
<p>74. Liu, Y.-H., Giunta, B., Zhou, H.-D., Tan, J., andWang, Y.-J. (2012). Immu-<br/>notherapy for Alzheimer disease&#8212;the challenge of adverse effects. Nat.<br/>Rev. Neurol. 8, 465&#8211;469. https://doi.org/10.1038/nrneurol.2012.118.<br/></p>
<p>75. Sims, J.R., Zimmer, J.A., Evans, C.D., Lu, M., Ardayfio, P., Sparks, J.,<br/>Wessels, A.M., Shcherbinin, S., Wang, H., Monkul Nery, E.S., et al.<br/>(2023). Donanemab in Early Symptomatic Alzheimer Disease: The<br/>TRAILBLAZER-ALZ 2 Randomized Clinical Triall. JAMA 330, 512&#8211;527.<br/>https://doi.org/10.1001/jama.2023.13239.<br/></p>
<p>76. Fuller, J.P., Stavenhagen, J.B., and Teeling, J.L. (2014). New roles for Fc<br/>receptors in neurodegeneration-the impact on Immunotherapy for<br/>Alzheimer&#8217;s Disease. Front. Neurosci. 8, 235. https://doi.org/10.3389/<br/>fnins.2014.00235.<br/></p>
<p>77. Peng, Y., Ding, L., Xiao, Z., Song, M., Lv, J., and Liu, G.H. (2024). Ethical<br/>concerns in aging research: perspectives of global frontline researchers.<br/>Sci. China Life Sci. https://doi.org/10.1007/s11427-024-2650-y.<br/></p>
<p>78. Schneider, C.A., Rasband, W.S., and Eliceiri, K.W. (2012). NIH Image to<br/>ImageJ: 25 years of image analysis. Nat. Methods 9, 671&#8211;675. https://<br/>doi.org/10.1038/nmeth.2089.<br/></p>
<p>79. Wolf, F.A., Angerer, P., and Theis, F.J. (2018). SCANPY: large-scale sin-<br/>gle-cell gene expression data analysis. Genome Biol. 19, 15. https://doi.<br/>org/10.1186/s13059-017-1382-0.<br/></p>
<p>80. Waskom, M. (2021). seaborn: statistical data visualization. J. Open<br/>Source Software 6, 6. https://doi.org/10.21105/joss.03021.<br/></p>
<p>81. Hao, Y., Hao, S., Andersen-Nissen, E., Mauck,W.M., 3rd, Zheng, S., But-<br/>ler, A., Lee, M.J., Wilk, A.J., Darby, C., Zager, M., et al. (2021). Integrated<br/>analysis of multimodal single-cell data. Cell 184, 3573&#8211;3587.e29. https://<br/>doi.org/10.1016/j.cell.2021.04.048.<br/></p>
<p>82. Wu, T., Hu, E., Xu, S., Chen, M., Guo, P., Dai, Z., Feng, T., Zhou, L., Tang,<br/>W., Zhan, L., et al. (2021). clusterProfiler 4.0: A universal enrichment tool<br/>for interpreting omics data. Innovation (Camb) 2, 100141. https://doi.org/<br/>10.1016/j.xinn.2021.100141.<br/></p>
<p>83. Trapnell, C., Cacchiarelli, D., Grimsby, J., Pokharel, P., Li, S., Morse, M.,<br/>Lennon, N.J., Livak, K.J., Mikkelsen, T.S., and Rinn, J.L. (2014). The dy-<br/>namics and regulators of cell fate decisions are revealed by pseudotem-</p>

</div><div>
</div><div><p>poral ordering of single cells. Nat. Biotechnol. 32, 381&#8211;386. https://doi.<br/>org/10.1038/nbt.2859.<br/></p>
<p>84. Chen, S., Zhou, Y., Chen, Y., and Gu, J. (2018). fastp: an ultra-fast all-in-<br/>one FASTQ preprocessor. Bioinformatics 34, i884&#8211;i890. https://doi.org/<br/>10.1093/bioinformatics/bty560.<br/></p>
<p>85. Kim, D., Paggi, J.M., Park, C., Bennett, C., and Salzberg, S.L. (2019).<br/>Graph-based genome alignment and genotyping with HISAT2 and<br/>HISAT-genotype. Nat. Biotechnol. 37, 907&#8211;915. https://doi.org/10.<br/>1038/s41587-019-0201-4.<br/></p>
<p>86. Liao, Y., Smyth, G.K., and Shi, W. (2014). featureCounts: an efficient gen-<br/>eral purpose program for assigning sequence reads to genomic features.<br/>Bioinformatics 30, 923&#8211;930. https://doi.org/10.1093/bioinformatics/<br/>btt656.<br/></p>
<p>87. Love, M.I., Huber, W., and Anders, S. (2014). Moderated estimation of<br/>fold change and dispersion for RNA-seq data with DESeq2. Genome<br/>Biol. 15, 550. https://doi.org/10.1186/s13059-014-0550-8.<br/></p>
<p>88. Zhou, Y., Zhou, B., Pache, L., Chang, M., Khodabakhshi, A.H., Tanasei-<br/>chuk, O., Benner, C., and Chanda, S.K. (2019). Metascape provides a<br/>biologist-oriented resource for the analysis of systems-level datasets.<br/>Nat. Commun. 10, 1523. https://doi.org/10.1038/s41467-019-09234-6.<br/></p>
<p>89. Shannon, P., Markiel, A., Ozier, O., Baliga, N.S., Wang, J.T., Ramage, D.,<br/>Amin, N., Schwikowski, B., and Ideker, T. (2003). Cytoscape: a software<br/>environment for integrated models of biomolecular interaction networks.<br/>Genome Res. 13, 2498&#8211;2504. https://doi.org/10.1101/gr.1239303.<br/></p>
<p>90. Dobin, A., Davis, C.A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S.,<br/>Batut, P., Chaisson, M., and Gingeras, T.R. (2013). STAR: ultrafast uni-<br/>versal RNA-seq aligner. Bioinformatics 29, 15&#8211;21. https://doi.org/10.<br/>1093/bioinformatics/bts635.<br/></p>
<p>91. Jin, Y., Tam, O.H., Paniagua, E., andHammell, M. (2015). TEtranscripts: a<br/>package for including transposable elements in differential expression<br/>analysis of RNA-seq datasets. Bioinformatics 31, 3593&#8211;3599. https://<br/>doi.org/10.1093/bioinformatics/btv422.<br/></p>
<p>92. Debacq-Chainiaux, F., Erusalimsky, J.D., Campisi, J., and Toussaint, O.<br/>(2009). Protocols to detect senescence-associated beta-galactosidase<br/>(SA-betagal) activity, a biomarker of senescent cells in culture and in vivo.<br/>Nat. Protoc. 4, 1798&#8211;1806. https://doi.org/10.1038/nprot.2009.191.<br/></p>
<p>93. Wu, Z., Lu, M., Liu, D., Shi, Y., Ren, J., Wang, S., Jing, Y., Zhang, S.,<br/>Zhao, Q., Li, H., et al. (2023). m(6)A epitranscriptomic regulation of tissue<br/>homeostasis during primate aging. Nat Aging 3, 705&#8211;721. https://doi.org/<br/>10.1038/s43587-023-00393-2.<br/></p>
<p>94. Sun, S., Li, J., Wang, S., Li, J., Ren, J., Bao, Z., Sun, L., Ma, X., Zheng, F.,<br/>Ma, S., et al. (2023). CHIT1-positive microglia drive motor neuron ageing<br/>in the primate spinal cord. Nature 624, 611&#8211;620. https://doi.org/10.1038/<br/>s41586-023-06783-1.<br/></p>
<p>95. Kharitonenkov, A., Shiyanova, T.L., Koester, A., Ford, A.M., Micanovic,<br/>R., Galbreath, E.J., Sandusky, G.E., Hammond, L.J., Moyers, J.S.,<br/>Owens, R.A., et al. (2005). FGF-21 as a novel metabolic regulator.<br/>J. Clin. Invest. 115, 1627&#8211;1635. https://doi.org/10.1172/JCI23606.<br/></p>
<p>96. Wang, S., Zheng, Y., Li, J., Yu, Y., Zhang, W., Song, M., Liu, Z., Min, Z.,<br/>Hu, H., Jing, Y., et al. (2020). Single-Cell Transcriptomic Atlas of Primate<br/>Ovarian Aging. Cell 180, 585&#8211;600.e19. https://doi.org/10.1016/j.cell.<br/>2020.01.009.<br/></p>
<p>97. Zhang, X., Goncalves, R., andMosser, D.M. (2008). The isolation and char-<br/>acterization of murine macrophages. Curr Protoc Immunol Chapter 14,<br/>14.11.11&#8211;14.11.14. https://doi.org/10.1002/0471142735.im1401s83.<br/></p>
<p>98. Ba&#776;ckdahl, J., Franze&#769;n, L., Massier, L., Li, Q., Jalkanen, J., Gao, H., Ander-<br/>sson, A., Bhalla, N., Thorell, A., Ryde&#769;n, M., et al. (2021). Spatial mapping<br/>reveals human adipocyte subpopulations with distinct sensitivities to<br/>insulin. Cell Metab. 33, 1869&#8211;1882.e6. https://doi.org/10.1016/j.cmet.<br/>2021.07.018.<br/></p>
<p>99. Hounkpe, B.W., Chenou, F., de Lima, F., and De Paula, E.V. (2021). HRT<br/>Atlas v1.0 database: redefining human and mouse housekeeping genes<br/>and candidate reference transcripts by mining massive RNA-seq</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Cell 187, 7025&#8211;7044, November 27, 2024 7043</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>datasets. Nucleic Acids Res. 49, D947&#8211;D955. https://doi.org/10.1093/<br/>nar/gkaa609.<br/></p>
<p>100. de Boer, A.C., van Riel, L.A., and den Ottolander, G.J. (1979). Measure-<br/>ment of antithrombin III, alpha 2-macroglobulin and alpha 1-antitrypsin<br/>in patients with deep venous thrombosis and pulmonary embolism.<br/>Thromb. Res. 15, 17&#8211;25. https://doi.org/10.1016/0049-3848(79)<br/>90048-3.<br/></p>
<p>101. De Cecco, M., Ito, T., Petrashen, A.P., Elias, A.E., Skvir, N.J., Criscione,<br/>S.W., Caligiana, A., Brocculi, G., Adney, E.M., Boeke, J.D., et al. (2019).<br/>L1 drives IFN in senescent cells and promotes age-associated inflamma-<br/>tion. Nature 566, 73&#8211;78. https://doi.org/10.1038/s41586-018-0784-9.</p>

</div><div><p>7044 Cell 187, 7025&#8211;7044, November 27, 2024</p>

</div><div><p>102. Zhang, B., Fu, D., Xu, Q., Cong, X., Wu, C., Zhong, X., Ma, Y., Lv, Z.,<br/>Chen, F., Han, L., et al. (2018). The senescence-associated secretory<br/>phenotype is potentiated by feedforward regulatory mechanisms<br/>involving Zscan4 and TAK1. Nat. Commun. 9, 1723. https://doi.org/10.<br/>1038/s41467-018-04010-4.<br/></p>
<p>103. Watanabe, S., Kawamoto, S., Ohtani, N., and Hara, E. (2017). Impact of<br/>senescence-associated secretory phenotype and its potential as a ther-<br/>apeutic target for senescence-associated diseases. Cancer Sci. 108,<br/>563&#8211;569. https://doi.org/10.1111/cas.13184.<br/></p>
<p>104. Kim, D., Langmead, B., and Salzberg, S.L. (2015). HISAT: a fast spliced<br/>aligner with low memory requirements. Nat. Methods 12, 357&#8211;360.<br/>https://doi.org/10.1038/nmeth.3317.</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>STAR+METHODS</p>

</div><div>
</div><div><p>KEY RESOURCES TABLE</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>REAGENT or RESOURCE SOURCE IDENTIFIER<br/>Antibodies<br/></p>
<p>Rabbit anti-mouse CD45 Abcam Cat# 10558; RRID:AB_442810<br/>Goat anti-mouse Kappa-BIOT SouthernBiotech Cat# 1050-08; RRID:AB_2794383<br/>Rabbit anti-mouse Glutamine Synthetase Abcam Cat# 49873; RRID:AB_880241<br/>Rabbit anti-mouse NeuN Abcam Cat# 177487; RRID:AB_2532109<br/>Rabbit anti-mouse IgG Abcam Cat# 46540; RRID:AB_2614925<br/>Goat anti-IBA1 Abcam Cat# 5076; RRID:AB_2224402<br/>Rabbit anti-mouse F4/80 ServiceBio Cat# 113373; RRID:AB_2938980<br/>Goat anti-mouse E-Cadherin R&amp;D Systems Cat# AF748; RRID:AB_355568<br/>Rabbit anti-mouse CYP11A1 ABclonal Cat# A1713; RRID:AB_2763764<br/>Rat monoclonal anti-F4/80 Abcam Cat# 90247; RRID:AB_10712189<br/>Rabbit polyclonal anti-CD3 Abcam Cat# 5690;RRID:AB_305055<br/>Rabbit anti-CD79B (D7V2F) Cell signaling technology Cat# 96024S;RRID:AB_2800254<br/>Rat monoclonal anti-F4/80 Abcam Cat# 6640;RRID:AB_1140040<br/>Mouse monoclonal anti-CD138 Abcam Cat# 34164;RRID:AB_778207<br/>Goat anti-mouse IgG Huabio RRID: AB_1013<br/>Mouse monoclonal anti-IL1b Santa Cruz Cat# 52012;RRID:AB_629741<br/>Mouse monoclonal anti-TNF-a Abcam Cat# 1793; RRID:AB_302615<br/>Rabbit anti-human IgG Proteintech Cat# CL488-10284; RRID:AB_2918970<br/>Rabbit anti-mouse P21 Abcam Cat# 188224; RRID:AB_2734729<br/>Rabbit anti-mouse 4-HNE Abcam Cat# 46545; RRID:AB_722490<br/>Rabbit anti-mouse FcgRIV Cell signaling technology RRID: AB_73741<br/>Mouse anti-beta-actin Santa Cruz Biotechnolog Cat# sc-69879; RRID:AB_1119529<br/>Rat anti-F4/80 antibody [BM8] Abcam Cat# 16911; RRID:AB_443548<br/>Goat anti-human IgG Thermo Fisher Scientific Cat# 62-8420; RRID:AB_2533962;<br/>Rabbit anti-mouse iNOS Abcam Cat# 283655; RRID:AB_3083470<br/>Rabbit anti-mouse phospho-NF-kB p65 Cell Signaling Technology Cat# 3033S; RRID:AB_331284<br/>HRP-conjugated goat anti-mouse IgG (H+L) ZSGB-Bio Cat# ZB2305; RRID: AB_2747415<br/>HRP-conjugated goat anti-rabbit IgG (H+L) ZSGB-Bio Cat# ZB2301; RRID: AB_2747412<br/>Rabbit anti-S100A8 Abcam RRID: AB_180735<br/>Rabbit anti-phospho-Histone H2A.X Cell signaling technology Cat# 9718; RRID:AB_2118009<br/>Rabbit anti-MMTV Novus Biologicals RRID: AB_NBP2-44179<br/>Rabbit anti-H3K9me3 Abcam Cat# 8898; RRID:AB_306848<br/>Rabbit anti-Lamin B1 Abcam Cat# 16048; RRID:AB_443298<br/>Mouse anti-CD68 Abcam Cat# 955; RRID: AB_307338<br/>Rabbit anti-CD68 Abcam Cat# 213363; RRID:AB_2801637<br/>Anti-human endogenous retrovirus type K (HERVK)<br/>envelope protein<br/></p>
<p>Austral Biologicals Cat# HERM-1811-5; RRID:AB_10891119<br/></p>
<p>Rabbit anti-STAT1 Cell signaling technology Cat# 9172S; RRID:AB_2198300<br/>Alexa Fluor 488 Donkey Anti-Rabbit IgG (H+L) Invitrogen Cat# A-21206; RRID: AB_2535792<br/>Alexa Fluor 568 Donkey Anti-Rabbit IgG (H+L) Invitrogen Cat# A-10042; RRID: AB_2534017<br/>Alexa Fluor 647 Donkey Anti-Rabbit IgG (H+L) Invitrogen Cat# A-31573; RRID:AB_2536183<br/>Alexa Fluor 488 Donkey Anti-Goat IgG (H+L) Invitrogen Cat# A-11055; RRID: AB_2534102<br/>Alexa Fluor 568 Donkey Anti-Goat IgG (H+L) Invitrogen Cat# A-11057; RRID: AB_2534104<br/></p>
<p>(Continued on next page)<br/></p>
<p>Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024 e1</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Continued<br/>REAGENT or RESOURCE SOURCE IDENTIFIER<br/></p>
<p>Alexa Fluor 488 Donkey Anti-Mouse IgG (H+L) Invitrogen Cat# A-21202; RRID: AB_141607<br/>Alexa Fluor 568 Donkey Anti-Mouse IgG (H+L) Invitrogen Cat# A-10037; RRID: AB_2534013<br/>Alexa Fluor(R) 488 anti-mouse MAdCAM-1 BioLegend Cat# A-120708; RRID: AB_493398<br/>Alexa Fluor 488 Donkey Anti-Rat IgG (H+L) Invitrogen Cat# A-21208; RRID: AB_2535794<br/></p>
<p>Biological samples<br/>Human brain sample for microglial collection Peking University Third<br/></p>
<p>Hospital<br/>N/A<br/></p>
<p>Human peripheral blood for macrophage isolation Chinese PLA General<br/>Hospital<br/></p>
<p>N/A<br/></p>
<p>Human liver, lymph node, brain, and spleen samples West China Hospital of<br/>Sichuan University<br/></p>
<p>N/A<br/></p>
<p>Chemicals, peptides, and recombinant proteins<br/>0.25% Trypsin-EDTA Gibco Cat# 25200-072<br/>Collagenase, type I Gibco Cat# 17100017<br/>Hoechst 33342 Thermo Fisher Scientific Cat# H3570<br/>SUPERase In RNase inhibitor Invitrogen Cat#AM2694<br/>RNasin Plus RNase inhibitor Promega Cat#N2615<br/>Penicillin/streptomycin (P/S) Gibco Cat# 15140-163<br/>4% paraformaldehyde, PFA Dingguo Cat# AR-0211<br/>pH 6.0 sodium citrate buffer ZSGB-BIO Cat# ZLI-9064<br/>Normal Donkey Serum Jackson ImmunoResearch Cat#017-000-121, RRID: AB_2337258<br/>Triton X-100 Sigma-Aldrich Cat# T9284<br/>M-CSF Human StemImmune Cat# HHM-MC-0100<br/>Mouse IgG protein Bioss Cat# bs-0296P<br/>Tissue-Tek OCT Sakura Cat# 4583<br/>Nucleic Acid Dye Thermo Cat# Q10212<br/>20XSSC Thermo Cat# AM9770<br/>Pepsin Sigma Cat# P7000<br/>RNase inhibitor NEB Cat# M0314L<br/>SuperScript II Invitrogen Cat# 18064-014<br/>Exonuclease I NEB Cat# M0293L<br/>KAPA HiFi Hotstart Ready Mix Roche Cat# KK2602<br/>Qubit dsDNA HS Assay Kit Thermo Cat# Q32854<br/>Modified Masson&#8217;s Trichrome Stain Kit Solarbio Cat# G1346<br/>AMPure XP Beads Vazyme Cat# N411-03<br/>SuperScript II Invitrogen Cat# 18064-014<br/>cDNA HIFI PCR mix MGI Cat# 1000028514<br/>SDS buffer BBI Cat# A600485-0500<br/>Aggresome dye (Proteostat&#2; Protein aggregation<br/>assay<br/></p>
<p>Enzo Cat# ENZ-51023-KP002<br/></p>
<p>Lipofectamine&#3; RNAiMAX Invitrogen Cat# 13778100<br/>Protease inhibitor cocktail Roche Cat# 4693159001<br/>X-gal Amresca Cat# 0428-25G<br/>PVDF membranes Millipore Cat# IPVH00010<br/>RIPA lysis buffer Beyotime Biotechnology Cat# P0013B<br/>M-CSF Mouse BioLegend Cat# 576404<br/>GM-CSF Novoprotein Cat# CK02<br/>Human IgG protein ROCKLAND Cat# 009-0102<br/>DMEM/F-12 Medium Thermo Fisher Scientific Cat# 11330057<br/>Gelatin Sigma-Aldrich Cat# V900863<br/></p>
<p>(Continued on next page)<br/></p>
<p><b>ll<br/></b></p>
<p>e2 Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024<br/></p>
<p>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Continued<br/>REAGENT or RESOURCE SOURCE IDENTIFIER<br/>Fetal bovine serum (FBS) Gibco Cat# 42Q7980K<br/>Thioglycollate broth Merck Cat# 1081900500<br/></p>
<p>Critical commercial assays<br/>STOmics Gene Expression kit S1 BGI Cat#1000028493<br/>Cellular Senescence Detection Kit-SPiDER-bGal Dojindo Cat# SG03<br/>Mouse IgG ELISA Kit Bethyl Laboratories Cat# E99-131, RRID: AB_2892028<br/>Mouse Igkc ELISA Kit Cloud-Clone Corp. Cat# SED038Mu<br/>Mouse IL6 ELISA Kit Biolegend Cat# 431307<br/>Mouse IFN-b ELISA Kit Abcam Cat# 252363<br/>Mouse CCL2 ELISA Kit Abcam Cat# 208979<br/>Mouse CCL5 ELISA Kit Abcam Cat# 100739<br/>Mouse IL6 ELISA Kit Invitrogen Cat# 88-7064, RRID:AB_2574986<br/>Griess Reagent System Promega Cat# G2930<br/></p>
<p>Deposited data<br/></p>
<p>The raw spatial transcriptomic data This study STOmicsDB: STT0000039<br/>The raw bulk RNA-seq data This study GSA: CRA012965<br/></p>
<p>Experimental models: Organisms/strains<br/>C57B1/6J male mouse SPF Biotechnology<br/></p>
<p>Co., Ltd., Beijing, China<br/>N/A<br/></p>
<p>C57B1/6J female mouse Qizhen laboratory animal<br/>technology Co., Ltd.,<br/>Hangzhou, China; Biocytogen<br/>Pharmaceuticals Co., Ltd.,<br/>Beijing, China<br/></p>
<p>N/A<br/></p>
<p>Oligonucleotides<br/></p>
<p>Primers for qRT-PCR, siRNA, and oligonucleotides,<br/>see Table S4<br/></p>
<p>This paper N/A<br/></p>
<p>Software and algorithms<br/></p>
<p>ImageJ Schneider et al.78 https://imagej.nih.gov/ij/<br/>GraphPad Prism 9 GraphPad Software Inc. https://www.graphpad.com/scientific%20software/<br/></p>
<p>prism/<br/>SAW (version v5.4.0) BGI https://github.com/STOmics/SAW<br/>Scanpy (version 1.9.3) Wolf et al.79 https://github.com/scverse/scanpy<br/>scikit-learn (version 1.2.2) N/A https://github.com/scikit-learn/scikit-learn<br/>Seaborn (version 0.12.2) Waskom et al.80 https://github.com/mwaskom/seaborn<br/>Seurat (version 4.1.0) Hao et al.81 https://github.com/satijalab/seurat<br/>ClusterProfiler (version 4.0.0) Wu et al.82 https://github.com/YuLab-SMU/clusterProfiler<br/>enrichplot (version 1.12.0) N/A https://github.com/GuangchuangYu/enrichplot<br/>Monocle3 (version 1.0.0) Trapnell et al.83 https://cole-trapnell-lab.github.io/monocle3/<br/>pheatmap (version 1.0.12) N/A https://github.com/raivokolde/pheatmap<br/>fastp (version 0.23.2) Chen et al.84 https://github.com/OpenGene/fastp<br/>HISAT2 (version 2.1.0) Kim et al.85 http://daehwankimlab.github.io/hisat2/download/<br/>featureCounts (version 2.0.3) Liao et al.86 https://github.com/ShiLab-Bioinformatics/subread<br/>DESeq2 (version 1.2.4) Love et al.87 https://github.com/mikelove/DESeq2<br/>Metascape (version 3.5) Zhou et al.88 http://metascape.org<br/>Cytoscape (version 3.10.1) Shannon et al.89 https://cytoscape.org<br/>STAR (version 2.7.2b) Dobin et al.90 https://github.com/alexdobin/STAR<br/>STAR (version 2.7.10a) Dobin et al.90 https://github.com/alexdobin/STAR<br/>Tetranscripts (version 2.2.3) Jin et al.91 https://github.com/mhammell-laboratory/<br/></p>
<p>Tetranscripts<br/></p>
<p><b>ll<br/></b></p>
<p>Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024 e3<br/></p>
<p>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS<br/></p>
<p>Ethical statement<br/>All experimental processes in this study comply with the Animal Care and Use Institutional Committee of the Institute of Zoology<br/>(IOZ-IACUC-2021-144), the Research and Ethical Committee of Tiantan Hospital, Capital Medical University (KYSQ-2021-373-<br/>01), the Research and Ethical Committee ofWest China Hospital, Sichuan University (2023-801), the Research and Ethics Committee<br/>of the Chinese People&#8217;s Liberation Army General Hospital (S2024-466-01), and the Research and Ethical Committee of Peking Uni-<br/>versity Third Hospital (M2023866).<br/></p>
<p>ASO treatment of old mice<br/>Nineteen-month-old male wild-type C57BL/6J mice were injected intraperitoneally with ASO-Fcgrt or non-targeting control ASO<br/>(ASO-NC). ASO of 10 nmol per injection (Ribobio, siBDM1999A) was administrated every 4 days for 10 times in total.<br/></p>
<p>Experimental mouse models<br/>In this study, all male and female wild-type C57BL/6J mice were sourced from SPF Biotechnology Co., Ltd., Beijing, China, Bio-<br/>cytogen Pharmaceuticals Co., Ltd., Beijing, China, and Qizhen laboratory animal technology Co., Ltd., Hangzhou, China. The<br/>mice were raised in a certified SPF grade facility with controlled environmental conditions, maintaining a temperature range of<br/>20-25&#1;C, humidity between 30% to 70%, and a 12-hour light-dark cycle. The mice were divided into five age groups for spatial<br/>transcriptional analysis and experimental validation: a 2-month-old group (referred to as &quot;Young&quot; or &quot;2M&quot; in figures), 4-month-old<br/>group (referred to as &quot;4M&quot; in figures), 13-month-old group (referred to as &quot;13M&quot; in figures), 19-month-old group (referred to as<br/>&quot;19M&quot; in figures) and 25-month-old group (referred to as &quot;Old&quot; or &quot;25M&quot; in figures). The mice were randomly assigned to the exper-<br/>imental groups. All experimental procedures were approved by the Chinese Academy of Sciences Institutional Animal Care and Use<br/>Committee.<br/></p>
<p>Human samples<br/>In this study, all human samples were sourced from individuals of Chinese ethnicity. The written informed consents were obtained<br/>from all participants or their relatives before tissue and blood sample collection for research according to regular principles. Human<br/>frontal lobe samples were collected from Tiantan Hospital, Capital Medical University during surgeries on patients with brain lesions<br/>or glioma. Liver, spleen, and lymph node samples were sourced from deceased individuals in West China Hospital, Sichuan Univer-<br/>sity, and their next of kin provided written informed consent for the collection and storage of tissue samples for research purposes.<br/>Among them, human liver samples were obtained from 9 males aged 15, 35, 36, 43, 45, 48, 61, 65, and 67 years old. Human lymph<br/>node samples were obtained from 18 individuals aged 14 (female), 14 (female), 15 (male), 15 (male), 21 (male), 21 (female), 36 (male),<br/>37 (male), 45 (female), 45 (female), 50 (male), 55 (male), 56 (male), 57 (male), 58 (male), 60 (male), 65 (male), and 65 years old (male). For<br/>frontal lobe samples, the age and gender of 10 individuals are 16 (female), 27 (female), 28 (male), 31 (male), 32 (female), 64 (female),<br/>66 (male), 67 (male), 68 (female), and 69 (male) years old. For human spleen samples, human ages and gender are 14 (female),<br/>15 (male), 21 (female), 23 (female), 35 (male), 52 (female), 52 (female), 61 (male), 65 (male), and 66 (female) years old.<br/></p>
<p>Human macrophage cells were derived from blood samples provided by healthy male young volunteers (about 18-30 years old)<br/>with written informed consent, as approved by the ethics committee of the Chinese People&#8217;s Liberation Army General Hospital. Hu-<br/>man microglial cells were derived from fetal brain sample collected by Peking University Third Hospital. Cell cultivation is conducted<br/>under controlled in vitro conditions at a temperature of 37 &#1;C and with an atmosphere containing 5% CO2, ensuring that a sterile<br/>environment is meticulously maintained throughout the entire procedural workflow.<br/></p>
<p>METHOD DETAILS<br/></p>
<p>IgG treatment of young mice<br/>Six-month-oldmale wild-type C57BL/6Jmicewere administered IgG or vehicle. IgG (Bioss, bs-0296P) wasmixed in PBS and admin-<br/>istrated tomice by intraperitoneal (i.p.) injection. IgG (0.33 g/kg bodyweight) or vehicle was given tomice every ten days for ten times.<br/>Briefly, 10 mg IgG dissolved in 100 mL PBS was administrated to the young mice of 30 g body weight.<br/></p>
<p>Tissue collection<br/>After anesthesia and dissection, tumor-bearing individuals, specifically the 25-month-old and the 19-month-old male were initially<br/>excluded. Subsequently, a range of tissues (heart, lung, liver, small intestine, testis, spleen, mesenteric lymph node, spinal cord,<br/>and hippocampus) from randomly selected mice across both young and old cohorts were collected. Subsequently, the tissues<br/>were extracted and immediately rinsed with PBS solution containing 0.4 U/mL SUPERase In RNase inhibitor (Invitrogen,<br/>AM2694). Excess liquid was gently blotted from the tissue surface using absorbent paper. Tissue samples from the same regional<br/>location were then directly immersed in OCT compound (Sakura, 4583) and placed in Cryo-molds, with pre-chilled OCT at the bot-<br/>tom, and kept on dry ice. Throughout the entire procedure, utmost care was taken to maintain RNase-free conditions. Additionally,<br/>consistent placement and polarity of samples were ensured within each tissue. Once the samples were completely frozen on dry ice,</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>e4 Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>they were promptly transferred into sealed bags and stored at -80&#1;C freezer for subsequent sectioning and spatial transcriptomic<br/>sequencing.<br/>Furthermore, to ensure consistency in tissue sectioning for spatial transcriptomics, we have standardized the process of tissue<br/></p>
<p>sampling by consistently using the largest cross-sectional area of each tissue. This approach was meticulously followed for all<br/>mice, ensuring uniformity and comparability in the depth and orientation of the tissue sections analyzed.<br/>In mice that shared the same age and feeding conditions, the above-mentioned organs were obtained for histological staining and<br/></p>
<p>subsequent experimental validation from each group. After removal, the organs were rinsed with a PBS solution containing RNase<br/>inhibitor. Any remaining liquid was promptly removed, and the tissues were then placed in 4%PFA or liquid nitrogen for preservation.<br/>Subsequently, tissues maintained in PFA were paraffin embedded for staining to facilitate further validation. Tissues kept in liquid<br/>nitrogen were used for western blot and other required experiments.<br/></p>
<p>Stereo-seq tissue processing<br/>To ensure the reproducibility and rigor of spatial transcriptomics sequencing, we conducted both biological and technical replicates<br/>in male mice before sequencing. Following the generation of raw sequencing data, the quality control process was implemented for<br/>data processing. Details are provided in Table S1; Figure S1, which outline all mouse IDs and specifics of the technical and biological<br/>replicates used in our experimental and sequencing protocols. Specifically, for the 2-month-old &#8216;&#8216;Young&#8217;&#8217; or &#8216;&#8216;2M&#8217;&#8217; group and the<br/>25-month-old &#8216;&#8216;Old&#8217;&#8217; or &#8216;&#8216;25M&#8217;&#8217; group, spatial transcriptomics was applied to samples from three distinct young and three distinct<br/>aged mice for most tissues, including the hippocampus, spinal cord, heart, liver, lymph node, and spleen (each representing a bio-<br/>logical replicate). Young testis were sampled with three biological replicates, while the aged group utilized four. The small intestine<br/>featured four young and five aged biological replicates. Additionally, two technical replicates were conducted for most biological rep-<br/>licates. However, one technical replicate was missing due to experimental errors for the young heart, young spleen, aged intestine,<br/>and aged testis, resulting in a single technical replicate for these tissues. The lung was unique, receiving samples from two young and<br/>two aged mice without technical replication.<br/>Additionally, spatial transcriptomics was applied to analyze key tissues from three distinct male mice at ages 4, 13, and 19months,<br/></p>
<p>such as the hippocampus, spleen, and lymph node, ensuring biological replication. A random experimental error resulted in an<br/>incomplete biological replicate for the 13-month-old liver. Details are provided in Table S1; Figure S4, which outline all mouse IDs<br/>and specifics of the technical and biological replicates used in our experimental and sequencing protocols. Technical replicates<br/>were executed for the spleen and lymph node, while the hippocampus and liver were not replicated due to the unavailability of Ste-<br/>reo-seq chips. Further details on raw data processing and quality control are outlined in the raw data processing section.<br/>The STOmics Gene Expression kit S1 (BGI, 1000028493) was utilized according to the standard protocol V1.21 At first, tissue sec-<br/></p>
<p>tionswere placed on 131 cm2 Stereo-seq chips. Then, the chipswere dried on a slide dryer at 37&#1;C for 3minutes immediately. There-<br/>after, the chips were moved to pre-cold methanol at -20&#1;C for 40 minutes, followed by staining with nucleic acid dye (Thermo Fisher,<br/>Q10212) for 5 minutes and nucleus imaging with Ti-7 Nikon Eclipse microscope at the channel of FITC. The chips were washed with<br/>0.13SSC buffer (Thermo, AM9770) and permeabilized using 0.1% pepsin (Sigma, P7000) in 0.01 M HCl buffer for 6-24 minutes at<br/>37&#1;C. Given different tissue properties, obtaining high-quality spatial transcriptomes of each tissue requires different incubating<br/>times by pepsin: 6 minutes for testis and spinal cord, 8 minutes for small intestine, 10 minutes for liver, 12 minutes for heart, lung,<br/>and brain, 15 minutes for lymph node, and 24 minutes for spleen. Samples with low RNA quality will not be proceeded to the next<br/>steps. After washing chips with 0.13SSC, we conducted reverse transcription at 42&#1;C for 3 hours using SuperScript II (Invitrogen),<br/>followed by washing with 0.13SSC twice. Then, the tissue sections on the chips were digested with a tissue removal buffer at 55&#1;C<br/>for 10 minutes, after which the chips were subjected to Exonuclease I (NEB, M0293L) at 55&#1;C overnight to release cDNA. The cDNA<br/>was purifiedwith AMPure XPBeads (Vazyme, N411-03) and amplified using a cDNAHIFI PCRmix (MGI) for 15 cycles. The PCRprod-<br/>ucts were purified using the AMPure XP Beads and were quantified by Qubit dsDNA HS Assay Kit (Invitrogen, Q32854).<br/></p>
<p>Library preparation and sequencing<br/>A total of 80 ng of cDNAwere fragmented to 400&#8211;600 bp using Tn5 transposases (Vazyme) at 55&#1;C for 10 minutes. Then, the reaction<br/>was stopped by adding 0.02% sodium dodecyl sulfate (SDS). The fragmented products were amplified using KAPA HiFi Hotstart<br/>Ready Mix. The PCR reaction consisted of 1 cycle at 95&#1;C for 5 minutes, followed by 13 cycles at 98&#1;C for 20 seconds, 58&#1;C for<br/>20 seconds, and 72&#1;C for 30 seconds. Finally, one cycle was performed at 72 &#1;C for 5 minutes. The amplified products were purified<br/>using AMPure XP Beads to generate DNA nanoball (DNB) libraries, which were sequenced using the 50 + 100 bp strategy on the<br/>DNBSEQTM T1 sequencing platform (MGI, Shenzhen, China).<br/></p>
<p>Senescence-associated b-Galactosidase (SA-b-Gal) staining<br/>SA-b-Gal staining of multiple mouse tissues was conducted following previously published protocols.92,93 Briefly, mouse tissues,<br/>including the hippocampus, spinal cord, lung, liver, small intestine, spleen, lymph node, and testis, were embedded in OCT com-<br/>pound (Sakura, 4583), after which tissues were cryosectioned at a thickness of 10 mm using a Leica CM3050S cryomicrotome<br/>and mounted on superfrost&#3; plus microscope slides (VWR) and stored at -80&#1;C freezer. For cell staining, as previously described,94<br/>cells were pre-seeded on plates. Before SA-b-Gal staining, sections were thawed at room temperature for 10 minutes. Then tissue<br/>sections or cells were fixed in a solution containing 2% formaldehyde and 0.2% glutaraldehyde at room temperature for 5 or</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024 e5</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>15 minutes, respectively. Subsequently, sections or cells were rinsed in PBS for three times and then stained with freshly prepared<br/>SA-b-Gal staining buffer (consisting of 150 mM NaCl, 2 mMMgCl2, 40 mM citric acid/Na phosphate buffer, 5 mM K4[Fe(CN)6], 5 mM<br/>K3[Fe(CN)6], and 1 mg/mL X-Gal (Amresco, 0428) at 37&#1;C overnight. Microscopic images were captured using a digital camera<br/>attached to a microscope (Nikon), and the SA-b-Gal-positive areas or cells were quantified using ImageJ software (version 1.48v).<br/></p>
<p>SPiDER-bGal staining<br/>OCT-embedded tissue sections were thawed at room temperature and then incubated at 37&#1;C for 30 minutes to promote tissue<br/>adhesion to the glass slide. The sections were then fixed with 4% PFA for 20 minutes at room temperature, washed with 1xPBS,<br/>and incubated with SPiDER-bGal staining solution (SG04, Dojindo) at 4&#1;C overnight. Images of stained tissues were obtained<br/>with a ZEISS LSM900 confocal microscope and quantified by ImageJ software (version 1.48v). When performing co-staining, the<br/>immunofluorescent antibody staining protocols were conducted after the SPiDER-bGal staining, as previously described in pub-<br/>lished protocols.43<br/></p>
<p>Immunofluorescence staining<br/>As previously described,44 tissue sections collected from frozen OCT and cells cultured on coverslips (Thermo Fisher Scientific) were<br/>fixed in 4% PFA for 20 minutes. For paraffin-embedded samples, they were deparaffinized, rehydrated and antigen retrieval as pre-<br/>viously described.44 Then, samples including tissue sections and cells were permeabilized with 0.4% TritonX-100 (Sigma) for 30 mi-<br/>nutes, and blocked with 5% donkey serum (Jackson Immuno Research) diluted in PBS for 1 hour at room temperature. Subse-<br/>quently, samples were incubated with primary antibodies overnight at 4&#1;C, washed with PBS, and incubated with secondary<br/>antibodies for 1 hour at room temperature, after which nuclei were stained with Hoechst 33342 (Thermo Fisher Scientific). Images<br/>of staining samples were obtained with a ZEISS LSM900 confocal microscope and quantified by ImageJ software (version 1.48v).<br/>For human frontal lobe sample used for immunostaining, human age and gender are 16 (female), 27 (female), 28 (male), 31 (male),<br/>32 (female), 64 (female), 66 (male), 67 (male), 68 (female), and 69 (male) years old. For human lymph nodes used for immunostaining,<br/>human ages and gender are 14 (female), 15 (male), 15(male), 21 (male), 21 (female), 45 (female), 56 (male), 58 (male), 60 (male), and 65<br/>(male) years old. For human spleen samples used for immunostaining, human ages and gender are 14 (female), 15 (male), 21 (female),<br/>23 (female), 35 (male), 52 (female), 52 (female), 61 (male), 65 (male), and 66 (female) years old. The antibody information can be found<br/>in the key resources table.<br/></p>
<p>Immunohistochemistry staining<br/>Initially, sections embedded in paraffin were treated with xylene to remove the paraffin, followed by a sequential rehydration process<br/>through a series of ethanol solutions (100%, 90%, 80%, 70%, and 50%). The sections were then briefly washed in distilled water. For<br/>antigen retrieval, the sectionswere processed in a pressure cooker at 115&#1;C for 10minutes using sodium citrate buffer (pH 6.0). Post-<br/>cooling to room temperature, the sections were treated with 0.4% Triton X-100 in PBS for an hour to allow for permeabilization, and<br/>then with 3% hydrogen peroxide for 20minutes to suppress the activity of endogenous peroxidase enzymes. Blocking was achieved<br/>by incubating the sections in 5% donkey serum at room temperature for an hour. The primary antibody incubation was conducted<br/>overnight at 4&#1;C. The following day, the sections were exposed to an HRP-conjugated secondary antibody for one hour at RT. The<br/>staining process was finalized by applying DAB for color development and hematoxylin for counterstaining. The sections were then<br/>dehydrated in an ascending ethanol series (50%, 70%, 80%, 90%, 100%) and cleared in xylene before being mounted with a neutral<br/>resin-based medium. Imaging was performed using either a PerkinElmer Vectro Polaris or a Leica Aperio VESA8 system, with the<br/>percentage of positively stained cells quantified using Image J software. The antibody information can be found in the key resources<br/>table.<br/></p>
<p>Masson staining<br/>Masson staining was performed as previously described using the Solarbiomasson triple staining kit.13 Briefly, paraffin sectionswere<br/>dewaxed, rehydrated and then stained sequentially with iron hematoxylin working solution, ponceau-acid fuchsin solution, phospho-<br/>molybdic-phosphotungstic acid solution, and aniline blue. The slides were rinsed in distilled water and differentiated with 1% acetic<br/>acid solution. Sections were then dehydrated with alcohol, made transparent with xylene and sealed with neutral resin. Images were<br/>scanned using PE panorama and counted with ImageJ software (version 1.48v).<br/></p>
<p>Hematoxylin and eosin (HE) staining<br/>Hematoxylin and eosin (HE) staining was performed as previously reported.95 The paraffin-embedded tissues were sectioned at a<br/>thickness of 5 mm with a rotary microtome. Sections were deparaffinized in xylene and rehydrated in gradient alcohols, incubated<br/>in hematoxylin solution, rinsed in running water to remove excess hematoxylin, differentiated with 1% acid alcohol for 5-10 seconds<br/>and then rinsed in running water for 10 minutes. Lastly, sections were stained with eosin, dehydrated in gradient ethanol and xylene,<br/>and mounted with cytoseal-60 (Stephens Scientific).</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>e6 Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>Aggresome staining<br/>For aggresome staining, cells cultured on cover slips within a 24-well plate or tissue sections that had been dewaxed and rehydrated<br/>were initially rinsed with PBS three times. Subsequently, the cells or tissue sections were treated with a 0.4% solution of Triton X-100<br/>in PBS for 10 minutes, followed by three further rinses with PBS. The samples were then stained with a 1:2500 dilution of the aggre-<br/>some staining solution in the dark for 3 minutes, after which they were rinsed three more times with PBS. A 15-minute destaining<br/>process with 1% acetic acid ensued, and the samples were again washed with PBS three times. The tissue samples were then pre-<br/>pared for microscopy by sealing them to prevent drying. For the cell samples, a blocking step with 5%donkey serumwas performed,<br/>followed by an overnight incubation at 4&#1;Cwith the primary antibodies F4/80 or IBA-1. On the subsequent day, a fluorescently labeled<br/>secondary antibody was applied, and the cover slips were mounted for microscopic imaging.<br/></p>
<p>RNA isolation and analyses<br/>Total cellular RNA was extracted using TRIzol (Thermo Fisher Scientific) and treated with DNase I to eliminate genomic DNA contam-<br/>ination. The RNA was then reverse transcribed into cDNA using the GoScript Reverse Transcription System (Promega). qRT-PCR<br/>was performed using the THUNDERBIRD qPCR Mix (Toyobo) on a CFX384 Real-Time System (Bio-Rad Laboratories, Inc.). For<br/>RNA analysis, GAPDH was used as the internal control. All primers used for qRT-PCR are listed in Table S4. For RNA-seq, a<br/>sequencing library was prepared using the Next Ultra RNA Library Prep Kit for Illumina (NEB) following the manufacturer&#8217;s protocol,<br/>with 1.5 mg of total RNA being utilized. The libraries were then subjected to paired-end 150-bp sequencing on Illumina HiSeq X-Ten<br/>platforms. Quality control and sequencing were carried out by NoVo gene Bioinformatics Technology.<br/></p>
<p>Western blot analysis<br/>The Western blot procedure was executed generally based on the methodology outlined by Wang et al.96 Tissues and cells were<br/>lysed in RIPA lysis buffer (P0013B, Beyotime Biotechnology) and subsequently heated to 105&#1;C for 10 minutes. We then quantified<br/>the total protein concentration using the BCA kit, as per the manufacturer&#8217;s directives. Proteins were then separated using SDS-<br/>PAGE and transferred to PVDF (polyvinylidene fluoride) membranes from Millipore. Membranes were then blocked using 5% skim<br/>milk powder (sourced from BBI Life Sciences) dissolved in 13TBST and incubated overnight at 4&#1;C with the designated primary an-<br/>tibodies. Subsequently, membranes were exposed to HRP-linked secondary antibodies at room temperature for 1 hour, and bands<br/>were visualized using the Bio-RadChemiDoc XRS system. The antibody information can be found in key resources table. Human liver<br/>samples used for the western blot experiment were obtained frommales aged 15, 35, 36, 43, 45, 48, 61, 65, and 67 years old. Human<br/>lymph node samples used for the western blot experiment were obtained from individuals aged 14 (female), 36 (male), 37 (male),<br/>45 (female), 50 (male), 55 (male), 57 (male), and 65 years old (male).<br/></p>
<p>Enzyme linked immunosorbent assay<br/>Following the manufacturer&#8217;s instruction, commercial ELISA kits were used to measure the levels of IgG, Igkc and IL6 in plasma from<br/>young and old mice or IL6, IFN-b, CCL2 and CCL5 in Veh or IgG treated macrophage and microglia cell culture medium. ELISA kit<br/>information can be found in key resources table.<br/></p>
<p>Nitrite measurement<br/>Nitrite detection was conducted following the instructions provided in the product manual of the Griess reagent system (Promega,<br/>G2930). Firstly, 50 mL of macrophage or microglia cell culture medium was added to wells in triplicate. Subsequently, 50 mL of the<br/>sulfanilamide solution was dispensed to all wells and then incubated for 5-10 minutes at room temperature, protected from light. Af-<br/>terward, 50 mL of the NED solution was dispensed to all wells and incubated for 5-10 minutes at room temperature, protected from<br/>light. Finally, absorbance within 30 minutes was measured in a plate reader (BioTek) with a filter between 520 nm and 550 nm.<br/></p>
<p>Mouse macrophage isolation and culture<br/>C57BL/6J mice, sourced from SPF Biotechnology Co., Ltd., Beijing, were acclimated to a 12-hour light-dark cycle for at least 5 days<br/>before the experiments. Subsequently, peritoneal cavity macrophages (PCDMs) and splenic macrophages (SPDMs) were isolated.<br/>To stimulate a substantial macrophage yield, 1 mL of 3% sterile thioglycolic acid broth was administered intraperitoneally every day<br/>for a period of 3 days. Mice were then humanely euthanized via cervical dislocation. A 1 mL aliquot of pre-chilled PBS was injected<br/>into the peritoneal cavity, and a gentle abdominal massage was performed for 3-5 minutes. The abdominal cavity was then carefully<br/>opened to retrieve the peritoneal fluid, which was centrifuged at 500 g at 4&#1;C for 5 minutes. The supernatant was discarded. For<br/>SPDMs, mouse spleens were surgically removed aseptically and placed in plates containing pre-cooled PBS. Connective tissue<br/>and fat were snipped. The spleen was transferred to the cold DMEM/F12 (Gibco, 11330057) with 10% fetal bovine serum (FBS),<br/>1% penicillin-streptomycin (PS, Gibco, 15140-163), and then extruded with a sterile syringe core through a 70 mm cell sieve to obtain<br/>single cells. After centrifuging at 1000 rpm for 5 minutes at 4&#1;C, the supernatant was removed. For lysis of red blood cells, cells were<br/>incubated with 3 mL of lysing buffer (BD) for 5 minutes at room temperature, and then washed once with the washing buffer (PBS<br/>containing 2% fetal bovine serum, Gibco). Following another round of centrifugation, the cells were harvested and cultured in a</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024 e7</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>medium supplemented with macrophage colony-stimulating factor (M-CSF).97 These cells were subsequently cultured for 5 days<br/>and exposed to either PBS or IgG (0.6 mg/mL). After 24 hours of PBS or IgG treatment, the culture medium and cell pellets were<br/>harvested.<br/></p>
<p>For the preparation of bone marrow-derived macrophages (BMDMs), mice were euthanized via cervical dislocation and then dis-<br/>infected with a 2-minute immersion in 70%ethanol. After wiping off the ethanol, fix themouse limbs and carefully separate the femurs<br/>and tibias, ensuring minimal contamination and avoiding premature exposure of the bone marrow. Transfer the bones to pre-chilled<br/>PBS in a laminar flow hood andwash twicewith PBS containing 2%penicillin-streptomycin (Gibco, 15140-163). Remove any remain-<br/>ing muscle tissue from the bones without cutting them open. Place the cleaned bones in fresh PBS with 2% penicillin-streptomycin.<br/>Trim the bone ends to reveal the marrow, then meticulously flush the marrow contents into a 50 mL centrifuge tube using a 2 mL<br/>syringe until the effluent runs clear. Subsequently, strain the marrow through a 70 mm filter that has been pre-moistened with<br/>PBS. After that, centrifuge the preparation at 4&#1;C with a force of 500 g for a duration of 10 minutes. After centrifugation, carefully<br/>decant the supernatant, leaving behind the red blood cell pellet. Proceed to lyse the red blood cells by adding 2 mL of lysis buffer,<br/>followed by a 5-minute incubation at room temperature, and then neutralize with an equal volume of PBS. Repeat the centrifugation<br/>step under identical conditions to isolate the macrophage pellet. Seed the harvested macrophages at an optimal density into either<br/>24-well or 6-well culture plates. Cultivate the cells in DMEM/F12 complete growth medium, which is a mixture of DMEM/F12 (Gibco,<br/>11330057), 10% fetal bovine serum, 1% penicillin-streptomycin, further enhanced with 10 ng/mL of M-CSF. Refresh the medium<br/>every 48 hours, and commence treatments with either PBS or IgG (0.6 mg/mL) on the fourth day of culture. After 24 hours of PBS<br/>or IgG treatment, the culture medium and cell pellets were harvested for further analysis.<br/></p>
<p>Mouse microglia isolation and culture<br/>For microglia isolation, mice were sprayed with 75% ethanol until they became unconscious. The mouse brains were then excised<br/>and placed in a dish containing PBS with 2% penicillin-streptomycin. Using fine forceps, the brain dura and skull were sequentially<br/>perforated and meticulously removed. The whole brain was then transferred to a new dish containing DMEM/F12 (Gibco, 11330057)<br/>complete medium. Under a microscope, the olfactory bulbs were removed, and the left and right hemispheres and cerebellum were<br/>separated. The cerebral cortex meninges of the mice were then carefully peeled away with fine forceps, preferably on ice. The<br/>cleaned tissue was immediately transferred to a centrifuge tube containing 5 mL of DMEM/F12 (Gibco, 11330057) with 10% fetal<br/>bovine serum, 1% penicillin-streptomycin, and triturated approximately 10 times until the tissue was mostly dispersed. The tissue<br/>was then filtered through a 70 mm cell strainer, and the filtrate was transferred to a 25 cm2 culture flask pre-coated with Poly-L-lysine<br/>hydrobromide (PLL) (Sigma-Aldrich, P1274), and the medium was supplemented to a total volume of 8 mL. On the second day of<br/>culture, the mediumwas replaced with a complete medium containing 25 ng/mL GM-CSF (Novoprotein, CK02). Full medium change<br/>was performed. By days 5-6, microglia could be observed floating in the medium. At this point, the supernatant was collected for<br/>centrifugation to obtain the microglia pellet, which was then seeded at a density of 1x105 cells. These cells were subsequently<br/>cultured for 2 days and exposed to either PBS or IgG (1.2 mg/mL). After 24 hours of PBS or IgG treatment, the culture medium<br/>and cell pellets were harvested.<br/></p>
<p>Human macrophage isolation and culture<br/>Peripheral blood mononuclear cells (PBMCs) were extracted from human blood samples at the Chinese People&#8217;s Liberation Army<br/>General Hospital using magnetic bead sorting with the STEMCELL kit (catalog number 19359) to isolate CD14+CD16- classical<br/>monocytes. The sorted cells were seeded and cultured in plates with a medium consisting of 1640 medium supplemented with<br/>10% fetal bovine serum, 5% penicillin-streptomycin, and 50 ng/mL human-M-CSF (stemImmune, HHM-GC-0100) for differentiation<br/>into macrophages. On the third day, a partial medium change was performed, followed by a complete medium change on the sixth<br/>day. Once the cells reached a stable state, they were treated with PBS or IgG (1.2 mg/mL). After 24 hours, the supernatant of the cell<br/>culture medium was collected for analysis of ELISA, and cells were collected for CD68 staining to assess macrophage differentiation<br/>efficiency. In the heat inactivation experiment with human macrophages, cells were cultured for 6 days, then treated with vehicle,<br/>1.2 mg/mL IgG, or heat-inactivated (HI, 60&#1;C for 30 minutes) IgG for 24 hours, followed by SA-b-Gal staining analysis.<br/></p>
<p>Human microglia isolation and culture<br/>After mechanically homogenizing the human fetal brain tissue obtained from Peking University Third Hospital, the sample was di-<br/>gested using 0.25% trypsin and 10% collagenase I. The tissue mixture was strained through a 70 mm filter, resuspended in microglia<br/>culture medium (DMEM/F12+10% FBS+1% PS+1% GlutaMax), and seeded into a cell culture bottle precoated with polylysine for<br/>5 minutes to collect themixture of glia cells. Next, non-adherent cells were collected and reseeded into polylysine-coated cell culture<br/>bottles and cultured for 9 days. On the 9th day, the cell culture bottles were placed on a 120 rpm rotary shaker for 2 hours to remove<br/>loosely attached microglia. The collected cell suspension was centrifuged, resuspended in microglia culture medium, and microglia<br/>were counted and seeded at a density of 13105 cells/mL on cell culture plates. After 2 days, the cells adhered to the surface, and the<br/>medium was changed to microglia culture medium containing PBS or IgG (1.2 mg/mL) for 24 hours.</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>e8 Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>SiRNA and ASO transfection and analyses<br/>For siRNA transfection in macrophages, cells were pre-seeded on plates and treated with si-NC (50 nM) and si-fcgr4 (50 nM),<br/>following the instructions provided in the product manual of the Lipofectamine&#3; RNAiMAX (Invitrogen, 13778100). After 48 hours<br/>siRNA transfection, cells were treated with PBS or 0.6 mg/mL IgG for 24 hours, and then samples were collected and tested. For<br/>ASO transfection in macrophages, cells were pre-seeded on plates and treated with ASO-NC (50 nM) and ASO-Fcgrt (50 nM)<br/>following the instructions provided in the product manual of the Lipofectamine&#3; RNAiMAX (Invitrogen, 13778100). After 48 hours<br/>ASO transfection, cells were collected and tested.<br/></p>
<p>Stereo-seq raw data processing<br/>The raw fastq data for Stereo-seq analysis contained CID and MID in read 1 (CID: 1-25 bp, MID: 26-35 bp), while read 2 consisted of<br/>cDNA sequences. To process the data, the raw reads were mapped to the designed coordinates matrix of the in situ captured chip<br/>with a tolerance of one basemismatch. UMIs containing N bases or with a quality score below 10 were filtered out, and the remaining<br/>reads were aligned to the mm10 genome reference using STAR (version 2.7.10a).90 Only reads with a MAPQ score of 10 or higher<br/>were retained for further analysis. The resulting mapped reads were counted and annotated to their corresponding genes using han-<br/>dleBam, generating gene expression profiles. This procedure was implemented in the publicly available SAWpipeline (version v5.4.0)<br/>(https://github.com/STOmics/SAW). The gene expression profile matrices were then proceeded within non-overlapping bins of 50 x<br/>50 DNB. The transcripts of the same gene were aggregated within each bin for further analysis.<br/></p>
<p>Establishment of spatial transcriptomic atlas of multiple mouse tissues<br/>To construct a spatial transcriptomic atlas of multiple mouse tissues, gene count matrices were analyzed using Scanpy (version<br/>1.9.3).79 The following steps were performed for filtering, data normalization, and dimensionality reduction analysis:<br/></p>
<p>1. Spots with fewer than 200 detected genes were excluded from the gene count matrices (spots with less than 50 genes were<br/>filtered in lung).<br/></p>
<p>2. All sample normalization and integration were performed using the standard Scanpy procedure, and batch effects were<br/>removed using the &#8216;&#8216;sc.external.pp.bbknn&#8217;&#8217; function with the first 30 principal components (PCs).<br/></p>
<p>3. After data integration, dimensionality reduction was performed using the &#8216;&#8216;sc.tl.umap&#8217;&#8217; function of Scanpy.<br/></p>
<p>To construct a spatial transcriptomic sample atlas of multiple mouse tissues, the integrated and normalized expression data was<br/>first averaged by sample. Then, dimensionality reduction and visualization were performed using the scikit-learn package (version<br/>1.2.2), followed by these steps:<br/></p>
<p>1. After data integration, the raw counts of the same gene were aggregated within each sample, converting the integrated data<br/>into a sample-level pseudo-bulk gene count matrix.<br/></p>
<p>2. Scaling and dimensionality reduction were performed using the &#8216;&#8216;StandardScaler&#8217;&#8217; and &#8216;&#8216;TSNE&#8217;&#8217; tools from scikit-learn. The re-<br/>sults of the dimensionality reduction were visualized using the Seaborn package (version 0.12.2).<br/></p>
<p>Following the above pipeline, we analyzed all 103 samples across nine tissues based on 1,535,191 high-quality spots to create a<br/>spatial transcriptomic atlas of multiple mouse tissues.<br/></p>
<p>Quality control, integration, clustering, and spot type identification of spatial transcriptomic data for each mouse<br/>tissue<br/>To create a detailed atlas for each mouse tissue, we performed independent analyses on the spatial transcriptomic data. For each<br/>tissue, gene count matrices were analyzed using the Seurat package (version 4.1.0)81 in R (version 4.0.5). The following steps were<br/>performed for filtering, data normalization, dimensionality reduction, clustering, and gene differential expression analysis:<br/></p>
<p>1. Spots with less than 200 genes were filtered from the gene count matrices (spots with less than 50 genes were filtered in lung).<br/>2. The gene count matrix of each sample was normalized using the &#8216;&#8216;SCTransform&#8217;&#8217; function with the default parameters. For 2M<br/></p>
<p>and 25M data, the matrices of each tissue were intergraded using the Seurat SCTransform-based CCA pipeline. Briefly, the<br/>&#8216;&#8216;SelectIntegrationFeatures&#8217;&#8217;, &#8216;&#8216;PrepSCTIntegration&#8217;&#8217; and &#8216;&#8216;FindIntegrationAnchors&#8217;&#8217; functions were used to select the features<br/>and anchors for integration. With the integration anchors, all datasets of the same tissue were subjected to the &#8216;&#8216;IntegrateData&#8217;&#8217;<br/>function. For 4M, 13M, and 19M data, with the limitation of data size, the matrices of each tissue were intergraded using the<br/>Seurat SCTransform-based RPCA pipeline. Briefly, the &#8216;&#8216;SelectIntegrationFeatures&#8217;&#8217;, &#8216;&#8216;PrepSCTIntegration&#8217;&#8217;, &#8216;&#8216;RunPCA&#8217;&#8217; and<br/>&#8216;&#8216;FindIntegrationAnchors&#8217;&#8217; functions were used to select the features and anchors for integration. With the integration anchors,<br/>all datasets of the same tissue were subjected to the &#8216;&#8216;IntegrateData&#8217;&#8217; function.<br/></p>
<p>3. After data integration, the linear dimensional reduction was performed using the &#8216;&#8216;RunPCA&#8217;&#8217; function. The &#8216;&#8216;RunUMAP&#8217;&#8217; function<br/>was used to reduce the first 10&#3;20 principal components into a low-dimensional space in which spots with similar gene<br/>expression would be placed together.</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024 e9</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>4. After data dimensional reduction, the unsupervised clusteringwas performed using &#8216;&#8216;FindNeighbors&#8217;&#8217; and &#8216;&#8216;FindClusters&#8217;&#8217; func-<br/>tions with a resolution of 1.<br/></p>
<p>5. Marker genes for each cluster were identified using the &quot;FindAllMarkers&quot; function, which employed theWilcoxon rank-sum test<br/>as the statistical method to compare between clusters. Spot types were identified based on canonical marker genes and their<br/>spatial locations. Additionally, we utilized marker genes calculated by the &quot;FindAllMarkers&quot; function, along with additional<br/>genes known for their specificity to different cell types within the tissues, as analyzed from published literature, to form robust<br/>marker gene sets. Spot types were identified by the scores of thesemarker gene sets, calculated using the &quot;AddModuleScore&quot;<br/>function, and their spatial locations.<br/></p>
<p>6. We excluded low-quality spots from the integrated data, which included all erythroid spots except the spleen, and the spots<br/>that highly expressed low-quality associated genes, such as Camk1d, Cdk8, Cmss1 and AY036118.<br/></p>
<p>The spot types identified in each tissue and its representative marker genes are shown in the Table S1.<br/></p>
<p>Adjacency score analysis<br/>To describe the proximity between two spot types (assumed as A and B), we introduce the Adjacency Score, based on insights from<br/>previously published research.98 This metric quantifies the average number of B spots close to A spots, effectively measuring the<br/>adjacency of B to A spots. The observed adjacency score Aobs is calculated as:<br/></p>
<p>Aobs =<br/>Nad<br/></p>
<p>B<br/>NA</p>

</div><div>
</div><div><p>where Nad represents the total number of B spots adjacent to A</p>

</div><div>
</div><div>B spots within a given sample, NA is the number of A spots.<br/>Acknowledging potential biases due to variations in sample geometry and the inherent differences in the numbers of the two spot<br/><p></p>
<p>types, we refine the observed score Aobs by accounting for a hypothetical random distribution. This adjustment is achieved through a<br/>shuffling algorithm that simulates a random adjacency of B spots to A spots across 50 iterations. From these simulations, we calcu-<br/>lated the average Am<br/></p>
<p>exp and standard deviation As<br/>exp of the expected adjacency score. Ultimately, a z-score was calculated as:<br/></p>
<p>Az =<br/>Aobs &#4; Am<br/></p>
<p>exp<br/></p>
<p>As<br/>exp<br/></p>
<p>This z-score represents how many standard deviations the observed adjacency score is from the expected mean, providing a<br/>normalized measure of proximity between A and B spots that accounts for random distributions.<br/></p>
<p>Spot type proportion and proximity analysis<br/>Before analyzing spot type proportions, we excluded &#8216;&#8216;Others&#8217;&#8217; spot types that were hard to identify as major spot types. The number<br/>of spots for each cell type was tallied in each sample and then divided by the total number of spots within the same sample. This<br/>calculation enabled the determination of the proportion of each spot type within each sample. To assess the proximity between<br/>spot types, pairwise adjacency scores were calculated as shown above, and subsequently, the &#8216;&#8216;hclust&#8217;&#8217; function was employed<br/>to conduct hierarchical clustering of the data using the &#8216;&#8216;ward.D2&#8217;&#8217; method.<br/></p>
<p>Analysis and calculation of organizational structure entropy<br/>Aging brings about changes in tissue structures that, while subtle, can profoundly impact their function. Therefore, we design orga-<br/>nizational structure entropy (OSE), which is used to describe the disorder of organizational structure quantitatively. Here is a step-by-<br/>step explanation of our calculation framework:<br/>Tissue segmentation using a Simple Linear Iterative Clustering (SLIC)-based method<br/>Given the complexity of tissue, comprising numerous structural units, the difference in aging might be obscured when examining<br/>tissues as a whole. To address the challenges presented by tissue complexity, our approach first segments tissue into distinct struc-<br/>tural units. Within small structural units, cells are similar in composition and close to each other, which makes it easier to detect the<br/>spot subtle structural variance.<br/></p>
<p>To achieve precise segmentation of the tissue&#8217;s structural and functional units, we adopt amethod based on SLIC. SLIC segments<br/>images into superpixels, which are conglomerates of pixels exhibiting visual similarity and conforming more accurately to object<br/>boundaries than traditional pixel grids.<br/></p>
<p>Referring to the definition of distance based on pixel color similarity and proximity in SLIC, we defined the distanceD based on spot<br/>gene expression similarity dexp and proximity dxy in our spatial transcriptomic data. Firstly, principal component analysis (PCA) was<br/>performed on the integrated data of each tissue for dimension reduction. For a given spot i, the first 15 principal components were<br/>selected as the eigenvalue ei of gene expression, and the spatial coordinate &#240;xi; yi&#222; is represented as si. Therefore, the distance D<br/>between two spots k and i in a sample is:<br/></p>
<p>dexp = kek &#4; eik2</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>e10 Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div>
</div><div>
</div><div><p>dxy = ksk &#4; sik2</p>

</div><div>
</div><div>
</div><div>
</div><div><p>D0 =<br/></p>
<p>ffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffi&#3;<br/>dexp<br/></p>
<p>Nexp<br/></p>
<p>&#4;2<br/></p>
<p>+<br/>&#3;<br/>dxy<br/></p>
<p>Nxy<br/></p>
<p>&#4;2<br/>s</p>

</div><div>
</div><div><br/><p>Where D0 is the sum of the gene expression distance normalized</p>

</div><div>
</div><div> by a constant Nexp and the spatial distance normalized by the grid<br/>interval Nxy. To simplify the distance measure, we defined D as follows:<br/><p></p>
<p>D =<br/>ffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffi<br/>d2<br/>exp+m &#5; d2<br/></p>
<p>xy<br/></p>
<p>q<br/>By defining D in this manner,m allows us to weigh the relative importance of gene expression similarity and spatial proximity. This<br/></p>
<p>tailored approach not only respects the inherent complexity of tissues but also enhances our ability to detect and analyze subtle<br/>structural variations indicative of aging.<br/>Identification of niches and initial entropy calculation<br/>In order to detect the subtle structural disorder of tissue arrangement, we defined a spot and its immediate neighborhood spots as a<br/>niche. By assessing the Shannon entropy based on the occurrence frequencies of these niches within a given structural unit (super-<br/>pixel), we can quantitatively evaluate the unit&#8217;s disorder.<br/>For a specific niche type nichei;j, i represents the intra spot, j represents a unique combination of the neighboring spots. Within a<br/></p>
<p>superpixel, the frequency Pij of nichei;j is calculated by:<br/></p>
<p>Pij =<br/>Ni;j<br/></p>
<p>Nsuperpixel</p>

</div><div>
</div><div><p>whereN is the number of niche ,N is the number of spot</p>

</div><div>
</div><div>i;j i;j superpixel s (niches) within the superpixel. Subsequently, referring to the defi-<br/>nition of Shannon index, the initial entropy E is calculated as:<br/><p></p>
<p>E =<br/>XNniche<br/></p>
<p>i;j<br/>&#4;Pi;j log&#240;Pi;j&#222;</p>

</div><div>
</div><div><p>whereNniche is the total number of niche types observed within the</p>

</div><div>
</div><div> superpixel. To ensure that the spot types in a tissue have the same<br/>granularity, we use the clustering results from Seurat as spot types.<br/><p></p>
<p>Adjustment of entropy<br/>To ensure the comparability of entropy between different superpixels, we need to consider the inconsistent size of superpixels and<br/>the spots missing after quality control and adjust them.<br/>First, use the convex hull to estimate the number of spots inside the superpixel. Due to the setup of our coordinate system, the<br/></p>
<p>coordinates of each spot are integers and the area is 1, the convex hull area Schull equates to the total number of spots Nchull it<br/>contains.<br/>Then, estimate the entropy of the most chaotic case within each superpixel. Considering the most chaotic situation of the super-<br/></p>
<p>pixel, that is, the niche of each spot in the superpixel is different, then the entropy of the superpixel is:<br/></p>
<p>Echull =<br/>X<br/>Nchull<br/></p>
<p>&#4; 1<br/>Nchull<br/></p>
<p>log<br/>&#3;<br/></p>
<p>1<br/>Nchull<br/></p>
<p>&#4;<br/>= log&#240;Schull&#222;<br/></p>
<p>The final step involves normalizing the initial entropy E of each superpixel using the entropy calculated for the most chaotic sce-<br/>nario Echull. The adjusted entropy Eadj of the superpixel is:<br/></p>
<p>Eadj =<br/>E<br/></p>
<p>Echull<br/></p>
<p>Adjustment of entropy ensures that the entropy measurement reflects the disorder relative to the maximum possible disorder for<br/>that superpixel size, providing a more accurate and comparable measure of structural disorder across different superpixels.<br/></p>
<p>OSE range definition<br/>We defined the superpixels with top 15%OSE score for each old sample as high OSE score ranges (R-OSEhigh), and the superpixels<br/>with bottom 15 % OSE score for each old sample as low OSE score ranges (R-OSElow).</p>

</div><div><p>Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024 e11</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>Cell identity analysis<br/>The cell identity gene set for each spot type were calculated using the &#8216;&#8216;FindAllMarkers&#8217;&#8217; function in young tissue data with a cutoff of<br/>adjusted P-values &lt; 0.05, Log2FC &gt; 0.25 and the minimum proportion of gene expression exceeding 10%. We selected the top<br/>10&#3;30 marker genes based on the rank of adjusted P values and Log2FC to score the corresponding spots using the &#8216;&#8216;AddModule-<br/>Score&#8217;&#8217; function.<br/></p>
<p>Ighigh and Ighhigh spot definition<br/>We defined spots with high expression of Igkc (normalized expression hippocampus and testis &gt; 1, spinal cord &gt; 1.5, heart and<br/>liver &gt; 2, lung and small intestine &gt; 4, lymph node and spleen &gt; 5) and any heavy chain gene (normalized expression &gt; 0) as spots<br/>with high Ig expression spot (Ighigh spot). We defined the max normalized expression of heavy chain genes as the expression of Igh,<br/>and we defined spots with high expression of Igh (hippocampus &gt; 2, lymph &gt; 4.5, liver &gt; 2, and spleen &gt; 5.5) as Ighhigh spots.<br/></p>
<p>Spatially connected domain identification analysis<br/>To identify the connectivity degree of MZmacro spot, we identified connected domains of MZmacro spot in eight-connected neigh-<br/>borhoods by drawing on digital image processing techniques. We used the Seed-Filling algorithm, as follows:<br/></p>
<p>1. Filter MZ macro spot for each sample.<br/>2. Search for a start spot (seed) of a new region.<br/>3. Search all spots in the slice that are connected with the seed spot and mark them to indicate their association.<br/>4. Repeat steps 2 and 3 until all spots are marked.<br/></p>
<p>Using this algorithm, we could identify the connectivity of MZmacro spot and calculate the connectivity degree of each connected<br/>area (the number of spots in the connected area). We set 5 as the threshold for the connectivity degree. If greater than the threshold,<br/>then it is adjacent; if less than the threshold, then it is non-adjacent.<br/></p>
<p>Spatial location aging-related differentially expression gene analysis<br/>Differential expression analysis for each spot type between 2M and 25M was performed using the Wilcoxon rank-sum test as imple-<br/>mented in the &#8216;&#8216;FindMarkers&#8217;&#8217; function of Seurat. Prior to the analysis, the count matrix was normalized using the &#8216;&#8216;NormalizeData&#8217;&#8217;<br/>function of Seurat and those of hippocampus and spinal cord were normalized with the corresponding tissue housekeeping genes<br/>obtained from HRT Atlas.99 Differentially expressed genes (Aging-DEGs) between the 25M and 2M groups were computed and<br/>filtered using the following criteria: |Log2FC| &gt; 0.25, adjusted P-value &lt; 0.05, and the minimum proportion of gene expression<br/>exceeding 10%.<br/></p>
<p>Differential expression analysis for the 4M, 13M, and 19M data were calculated using the &#8216;&#8216;fit_models&#8217;&#8217; function of monocle3<br/>(version 1.0.0). Before the analysis, the count matrix was normalized using the &#8216;&#8216;NormalizeData&#8217;&#8217; function of Seurat. Aging-related<br/>genes from 4M to 19M were identified and filtered with a q-value less than 0.05 for each spot type. Subsequently, we merged the<br/>aging-related genes by tissue, and used the &#8216;&#8216;pheatmap&#8217;&#8217; package (version 1.0.12) to cluster the genes into 8 distinct clusters. Genes<br/>that are upregulated with age increase are considered positively correlated with age, while the downregulated are considered nega-<br/>tively correlated. The positive and negative aging-related genes were designated as aging-related DEGs for each tissue.<br/></p>
<p>All the DEGsmatched the following regular expression: &#8217;&#770;Gm|&#770;mt-|&#770;Rps|&#770;Rpl|&#770;Hb[ab]&#8217; andwere excluded low-quality associated genes,<br/>such as Camk1d, Cdk8, Cmss1 and AY036118 (See the detailed list of DEGs in Table S2).<br/></p>
<p>Aging score and senescence-sensitive spot (SSS) analysis<br/>To accurately assess the senescence level of individual spots and pinpoint senescence-sensitive spots (SSSs), our methodology<br/>employs a calculation framework centered on aging-related differential genes in each tissue.<br/></p>
<p>For 2M and 25M data:<br/></p>
<p>1) The well-established and extensively utilized senescence marker lists were integrated to construct the aging-related gene set,<br/>which include CellAge,39 Aging Atlas,40 SenMayo,41 the GO category for cellular senescence100 (GO:0090398), and the Frid-<br/>man &amp; Tainsky gene set 42.<br/></p>
<p>2) The differentially expressed genes during aging (Aging-DEGs) from each tissue were overlapped with the aging-related gene<br/>set to obtain aging-sensitive genes.<br/></p>
<p>3) Utilizing the &#8216;&#8216;AddModuleScore&#8217;&#8217; function of Seurat, the aging score was calculated based on these aging-sensitive genes for<br/>each tissue. Specifically, the aging score for each spot was determined by calculating the difference between the scores of<br/>upregulated and downregulated aging-sensitive genes.<br/></p>
<p>4) To highlight the sensitive spots, the spots within the top 5% of aging scores in each old sample were designated as an SSS.</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>e12 Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>For 4M, 13M and 19M data:<br/></p>
<p>1) The aging scorewas calculated based on the aging-related DEGs for each tissue by the &#8216;&#8216;AddModuleScore&#8217;&#8217; function of Seurat,<br/>then determined by calculating the difference between the scores of upregulated and downregulated genes.<br/></p>
<p>2) To highlight the SSS, we calculated the aging score fold by dividing each spot&#8217;s aging score by the corresponding average<br/>mean aging score of its spot type in 4M, then designated the spots within the top 5% aging score fold as SSSs in each tissue.</p>

</div><div>
</div><div><p>SSS niche analysis and gradient definition<br/>In order to study the molecular and cellular characteristics extending from SSSs to outward surrounding spots, we used the nearest<br/>neighbor distance to describe the relative spatial relationship between spots and SSSs.<br/>For any given spot i, the distance Di to the SSS is determined as the minimum distance from i to the SSS. Specifically, the nearest<br/></p>
<p>neighbor distance Di form i to the SSS is calculated as:<br/></p>
<p>Di = minj&#731;SSS kSi &#4; Sjk2</p>

</div><div>
</div><div><br/>where S represents the coordinates &#240;x ; y &#222; of i, and S represent<p></p>

</div><div>
</div><div>i i i j s the coordinates of the SSS.<br/>To facilitate downstream computation, we introduce a niche gradient, denoted as G. This gradient is calculated by applying the<br/><p></p>
<p>floor function to the distance D, resulting in:<br/></p>
<p>G = PDR<br/></p>
<p>Therefore, for a given SSS, G1 spots are its immediate neighborhood non-SSS spots. Similarly, G2 spots are the further non-SSSs.<br/>Each gradient level, such as G1, G2, and so forth, encapsulates a ring of spots characterized by increasing distances from the SSS,<br/>thus allowing for a layered analysis.<br/></p>
<p>SSS niche gradient entropy definition<br/>To quantify the entropy across various gradients surrounding the SSS, we have refined the OSEmethod. Initially, we divide the tissue<br/>into different regions based on gradients, such asG1-G5. Subsequently, we employ random sampling to ensure an equitable number<br/>of sample spots across these regions. The entropy of each region is calculated according to the niche of the sampled spots, the same<br/>as step 2 of OSE calculation. Ultimately, we derive the region&#8217;s entropy by averaging the results from 100 times sampling.<br/></p>
<p>Senescence-associated secretory phenotype (SASP) score analysis<br/>In order to adequately describe cellular senescence, we initially constructed a new gene list associated with the Senescence-<br/>Associated Secretory Phenotype (SASP) by integrating previous studies101&#8211;103 and Reactome database (R-HSA-2559582:<br/>Senescence-Associated Secretory Phenotype).<br/>Subsequently, we selected genes that exhibit an average upregulation in their expression levels across different spot types during<br/></p>
<p>aging. These genes were then aggregated by tissue and intersected with our SASP gene list to generate tissue-specific SASP<br/>gene lists.<br/>Utilizing the &#8216;&#8216;AddModuleScore&#8217;&#8217; function of Seurat, the SASP score was calculated based on these tissue-specific SASP gene lists<br/></p>
<p>for each tissue.<br/></p>
<p>Pathway enrichment analysis and spatial mapping<br/>ClusterProfiler (version 4.0.0)82 was used to perform the enrichment analysis of GO biological process. The upregulated and down-<br/>regulated aging-DEGs of different spot types were enriched by the &#8216;&#8216;compareCluster&#8217;&#8217; function with a cutoff of q value &lt; 0.05. The<br/>results were further visualized with the &#8216;&#8216;enrichplot&#8217;&#8217; package (version 1.12.0). We scored spots using the &#8217;AddModuleScore&#8217; function<br/>from Seurat, with scores based on DEGs from GO terms, and then mapped the gene set scores spatially.<br/></p>
<p>Distance gradient gene analysis<br/>The R package scITDG (https://github.com/YandongZheng/scITDG) was utilized to identify genes that exhibit changes with respect<br/>to the niche gradient. Metascape (version 3.5)88 (http://metascape.org) was used to perform pathway enrichment analyses for the<br/>genes in each module. The results were further visualized with the ggplot2 R package.<br/></p>
<p>Bulk RNA-seq analysis<br/>RNA-seq raw data was trimmed, and low-quality reads and adapters were removed using fastp (version 0.23.2).84 After quality con-<br/>trol and adaptor trimming, sequencing reads were mapped to the mouse mm10 genome with HISAT2 software (version 2.1.0).104<br/>Then, the mapped data was counted using featureCounts (version 2.0.3). DEGs were identified using DESeq2 (version 1.2.4) with<br/>a cutoff of Benjamini-Hochberg adjusted P-value of less than 0.05 and an absolute |Log2FC| of greater than 1 (see the detailed<br/>list of DEGs in Table S3).</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024 e13</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div><p><b>ll<br/></b>Resource</p>

</div><div><p>Ingenuity pathway analysis of upstream regulators<br/>The regulators factors of DEGs from bulk RNA-seq data were identified using ingenuity pathway analysis (Qiagen; https://www.<br/>qiagenbioinformatics.com/products/ingenuitypathway-analysis). Only regulators with P values less than 0.05 were retained for<br/>downstream analyses. The results were further visualized with Cytoscape (version 3.10.1).89<br/></p>
<p>Repeat element expression level analysis<br/>The clean reads from bulk RNA-seq data were aligned to the mouse mm10 genome using STAR software (version 2.7.2b)90 with the<br/>following parameters: &#8216;&#8216;&#8211;outFilterType BySJout &#8211;winAnchorMultimapNmax 100 &#8211;outFilterMultimapNmax 100 &#8211;outFilterMismatch-<br/>NoverLmax 0.04&#8217;&#8217;. TEtranscripts software91 (version 2.2.3) was employed for quantification and differential analysis, utilizing a cutoff<br/>of &#8216;&#8216;|log2FC| &gt; 1 and adjusted P values &lt; 0.05&#8217;&#8217; (see the detailed list of DREs in Table S3).<br/></p>
<p>QUANTIFICATION AND STATISTICAL ANALYSIS OF THE EXPERIMENTAL DATA<br/></p>
<p>Statistical analyses and data handling of all experimental datasets were rigorously performed utilizing GraphPad Prism software<br/>(version 9.0.0). Data are represented as mean &#177; standard error of the mean (SEM). Statistical comparisons were made using the<br/>nonparametric Mann-Whitney test for biological replicates. In all figures, asterisks denote significant differences between different<br/>groups: *P &lt; 0.05, **P &lt; 0.01, ***P &lt; 0.001, ****P &lt; 0.0001. Specific P-values are listed for those exceeding the 0.05 threshold.</p>

</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>e14 Cell 187, 7025&#8211;7044.e1&#8211;e14, November 27, 2024</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Supplemental figures<br/></p>
<p>(legend on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S1. Morphological and histological profiling of male mouse tissues to identify aging phenotypes, related to Figure 1<br/>(A) SPiDER-bGal staining of multiple tissues in young (2-month-old) and old (25-month-old) male mice, including the hippocampus, liver, small intestine, spleen,<br/>lymph node, and testis. Top, representative images; and bottom, the percentage of SPiDER-bGal-positive cells in the tissues (n = 5 mice per group). Scale bars,<br/>20 mm.<br/>(B) Immunofluorescence of the immune cell marker CD45 in heart, lung, small intestine, and testis of young and old male mice. Top, representative images; and<br/>bottom, the percentage of CD45-positive cells in the tissues (n = 5 mice per group). Scale bars, 15 mm.<br/>(C) Immunofluorescence of the microglia maker IBA1 in hippocampus and spinal cord of young and old male mice. The percentages of IBA1-positive microglia<br/>were quantified. n = 6 mice in each group. Scale bars, 50 and 20 mm (zoom-in).<br/>(D) H&amp;E staining of liver tissues in young and old male mice. The dashed circles indicate inflammatory area, and the fold change of the inflammatory area was<br/>quantified. n = 5 mice per group. Scale bars, 100 and 50 mm (zoom-in).<br/>(E) Masson&#8217;s trichrome staining of spleen, small intestine, and heart in young and old male mice. Left, representative images. Right, the percentages of fibrotic<br/>area were quantified (n = 5 in small intestine, heart, and old spleen, n = 4 in young spleen). Scale bars, 50 mm.<br/>(F) Immunofluorescence of CD31 in hippocampus, spinal cord, and spleen of young and old male mice. The percentages of CD31-positive area were quantified<br/>(n = 6 mice per group). Scale bars, 500 and 50 mm (zoom-in).<br/>(G) Immunofluorescence of DCX in the hippocampus of young and old male mice. The percentages of DCX-positive cells were quantified. n = 7 in young group,<br/>n = 6 in old group. Scale bars, 50 and 20 mm (zoom-in).<br/>(H) H&amp;E staining of male mouse cardiac tissues in indicated groups. n = 4 mice per group. Scale bars, 1 mm and 50 mm (zoom-in).<br/>(I) H&amp;E staining of male mouse lung tissues in indicated groups. n = 5 mice per group. Scale bars, 100 and 50 mm (zoomed in).<br/>(J) Spatial mappings showing the structures and number of genes per spot of each biological repetition and technical repetition for mouse hippocampus, spinal<br/>cord, heart, lung, liver, small intestine, spleen, lymph node, and testis in young (2-month-old) and old (25-month-old) male mice. Colors ranging from blue to red<br/>indicate the number of genes from low to high. Scale bars, 500 mm.<br/>The Mann-Whitney test was utilized in (A)&#8211;(I). Data are presented as the mean &#177; SEM.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>(legend on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S2. A summary of spatial transcriptomic data of nine tissues from young and old male mice, related to Figure 1<br/>(A) Bar plots showing the quality control-related information of spatial transcriptomic datasets for mouse tissues.<br/>(B) t-SNE visualization of all samples (103 samples) of spatial transcriptomic data among nine tissues of young (2-month-old) and old (25-month-old) male mice,<br/>colored by tissue (top) and age (bottom).<br/>(C) UMAP visualization of all spots (1,535,191 spots) of spatial transcriptomic data among nine tissues of young (2-month-old) and old (25-month-old) male mice,<br/>colored by tissue (top) and age (bottom).<br/>(D) UMAP plots showing the distributions of spots from different batches, samples, or age groups before and after batch correction.<br/>(E) Spatial mappings showing the representative distribution of spot types and marker gene scores for the indicated spot types among nine mouse tissues. The<br/>full names and marker gene lists for each spot type can be found in Table S1. Colors from blue to red indicate the scores from low to high. Scale bars, 200 and<br/>100 mm (zoom-in) for lymph node; 500 and 200 mm (zoom-in) for hippocampus and spinal cord; 1 mm and 200 mm (zoom-in) for other tissues.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>(legend on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S3. Organizational structure analysis in diverse tissues of male mice, related to Figure 2<br/>(A) The boxplots showing the ratio of niche types to the number of spots within each superpixel between young (2-month-old) and old (25-month-old) mouse<br/>tissues. Wilcoxon rank-sum test.<br/>(B) The spatial mappings showing the OSE score and the corresponding spot types across liver, lung, and spinal cord during aging. The color of the point ranging<br/>from blue to red represents OSE score from low to high. Scale bars, 500 and 100 mm (zoom-in) for spinal cord, 1 mm and 200 mm (zoom-in) for the others.<br/>(C) The spot compositions of high and low OSE score regions across tissues of old mouse. Scale bars, 100 mm for hippocampus, spinal cord, lymph node, and<br/>testis; 200 mm for the others.<br/>(D) Bar plots showing the ratios of proportion for spots in high OSE score and low OSE score regions across different tissues of old mouse. &#8216;&#8216;*&#8217;&#8217; means that the<br/>number of spots in low OSE score regions is zero.<br/>(E) Left, spatial mappings of white pulp scores andmarginal zone (MZ) macrophages of spatial transcriptomic data from representative samples of young and old<br/>mouse spleens with the zoomed-in views. Scale bars, 1 mm and 200 mm (zoom-in). Right, plots showing the cell identity loss of white pulp (top) and decreased<br/>spot composition of the MZ macrophage (bottom) in spleen during aging. Wilcoxon rank-sum test.<br/>(F) Bar plot showing the enriched cell types of top 30 marker genes of white pulp by using Metascape tools (version 3.5).<br/>(G) Left, spatial mappings showing the distribution of adjacent MZ macro (defined as a collection of spots, numbering five or more, located in close proximity).<br/>Right, boxplot showing the ratio of spot number within adjacent MZ macro to the overall count of MZ macro spots. Wilcoxon rank-sum test.<br/>(H) H&amp;E staining of spleen tissues from young and old male mice. White pulp with a nucleus density of less than 450 nuclei/10,000 mm2 was defined as broken<br/>white pulp. n = 5 mice per group. Scale bars, 100 mm.<br/>(I) Spatial mapping of spot type showing the changes in spot proportion of young and old mouse lymph nodes. Top, the whole lymph nodes with colors of the<br/>spots indicate the spot type. Bottom, spot proportion and odds ratio of the indicated spot type. Wilcoxon rank-sum test. Scale bars, 500 mm.<br/>(J) Left, spatial mappings of zone 1 and zone 3 hepatocyte spot of spatial transcriptomic data from representative samples of young and old mouse livers. Right,<br/>boxplots showing the proportion of zone 1, zone 3, and the ratio of zone 3/zone 1 hepatocyte spots between young and old groups. White and red colors indicate<br/>zone 1 and zone 3 hepatocyte spots, respectively. Wilcoxon rank-sum test. Scale bars, 1 mm and 200 mm (zoom-in).<br/>(K) Immunofluorescence staining showing structural changes of livers in young and old mice, represented by the varied proportion of liver zonation (E-cadherin is<br/>themarker of zone 1, and glutamine synthetase (GS) serves as zone 3marker) (n= 5mice per group). The fold changes for the proportion of zone 1 and zone 3 from<br/>young and old mouse livers are quantified and displayed. Scale bars, 200 and 40 mm (zoomed in).<br/>(L) Representative spatial mappings of the gene set score for adipocyte in liver. Spots in the first and second columns are colored by spot types. Spots in the third<br/>to fourth columns are colored by cell identity score, with colors ranging from blue to red indicating scores from low to high. Right panel showing the quantification<br/>for adipocyte identity scores of young and old groups. Wilcoxon rank-sum test. Scale bars, 1 mm and 200 mm (zoom-in).<br/>The Mann-Whitney test was utilized in (H) and (K). Data are presented as the mean &#177; SEM.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>(legend on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S4. Spatial scale gene expression changes among different young and old mouse tissues, related to Figures 2 and 3<br/>(A) Spatial mappings showing the structures and number of genes per spot of spatial transcriptomic data for mouse hippocampus, liver, spleen, and lymph node<br/>from male mice aged 4-month-old (4M), 13-month-old (13M), and 19-month-old (19M). Colors ranging from blue to red indicate the number of genes from low to<br/>high. Scale bars, 500 mm.<br/>(B) Left, spatial mappings showing the representative distribution of spot types in 4-month-old mouse tissues with zoomed-in views. Right, spatial mappings<br/>illustrating the marker gene scores for the indicated spot types across different mouse tissues. The full names and marker gene lists for each spot type can be<br/>found in Table S1. Colors from blue to red indicate the scores from low to high. Scale bars, 500 and 200 mm (zoom-in) for hippocampus; 1mm and 200 mm (zoom-<br/>in) for liver and spleen; 200 and 100 mm (zoom-in) for lymph node.<br/>(C) Bar plot displaying the number of aging-related differentially expressed genes (aging DEGs) (25-month-old vs. 2-month-old) across various spot types from<br/>different tissues in spatial transcriptomic data.<br/>(D) Spatial mappings of the aging-DEG numbers across tissues, with colors ranging from blue to red, indicating the DEG numbers from low to high. The spot types<br/>with the most aging DEGs within each tissue are annotated in the plot.<br/>(E) Network plots showing the enriched pathways and terms for the aging DEGs. The size of the nodes represents the number of genes in the corresponding<br/>pathways. The color of each node indicates the proportion of DEGs from different spot types.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>(legend on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S5. Spatial evaluation of biological pathways among different young and old mouse tissues, related to Figure 3<br/>(A) Spatial mappings of the scores of gene set &#8216;&#8216;immune and inflammatory response&#8217;&#8217; in the indicated tissues. Spots in the first column are colored by spot types.<br/>Spots in the second to fourth columns are colored by scores, and colors fromblue to red indicate the gene set scores from low to high. Scale bars, 200 and 100 mm<br/>(zoom-in) for lymph node; 1 mm and 200 mm (zoom-in) for spleen.<br/>(B) Spatial mappings of the scores of gene set &#8216;&#8216;mitochondrial function and cellular respiration&#8217;&#8217; in the indicated tissues. Spots in the first column are colored by<br/>spot types. Spots in the second to fourth columns are colored by scores, and colors from blue to red indicate the gene set scores from low to high. Scale bars, 500<br/>and 200 mm (zoom-in) for hippocampus; 1 mm and 200 mm (zoom-in) for liver.<br/>(C) Spatial mappings of the scores of gene set &#8216;&#8216;apoptosis and neuron death&#8217;&#8217; in the indicated tissues. Spots in the first column are colored by spot types. Spots in<br/>the second to fourth columns are colored by scores, and colors from blue to red indicate the gene set scores from low to high. Scale bars, 500 and 200 mm<br/>(zoom-in).<br/>(D) Spatial mappings of the scores of gene set &#8216;&#8216;oxidative stress&#8217;&#8217; in the indicated tissues. Spots in the first column are colored by spot types. Spots in the second<br/>to fourth columns are colored by scores, and colors from blue to red indicate the gene set scores from low to high. Scale bars, 500 and 200 mm (zoom-in).<br/>(E) Spatial mappings of the scores of gene set &#8216;&#8216;T/B cell proliferation and activation&#8217;&#8217; in the indicated tissues. Spots in the first column are colored by spot types.<br/>Spots in the second to fourth columns are colored by scores, and colors fromblue to red indicate the gene set scores from low to high. Scale bars, 200 and 100 mm<br/>(zoom-in) for lymph node, 1 mm and 200 mm (zoom-in) for spleen.<br/>(F) Spatial mappings of the scores of gene set &#8216;&#8216;fatty acid and lipid metabolic process&#8217;&#8217; in the indicated tissues. Spots in the first column are colored by spot types.<br/>Spots in the second to fourth columns are colored by scores, and colors fromblue to red indicate the gene set scores from low to high. Scale bars, 500 and 200 mm<br/>(zoom-in) for small intestine; 1 mm and 200 mm (zoom-in) for liver.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>(legend on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S6. Spatial mapping of senescence-sensitive spots and their microenvironmental contexts, related to Figure 3<br/>(A) Boxplots and spatial mappings showing the increased OSE score around the SSS across tissues in old (25-month-old) male mice. Kruskal-Wallis test. Scale<br/>bars, 1 mm and 200 mm (zoom-in) for liver and testis; 200 and 100 mm (zoom-in) for lymph node.<br/>(B) Heatmap displaying the gradual changes in gene expression surrounding the SSS, categorized into six gene expression modules.<br/>(C) Boxplots showing the increased SASP around the SSS across tissues in old male mice. Kruskal-Wallis test.<br/>(D) Co-staining for SPiDER-bGal and TNF-a in liver, spleen, and testis from young (2-month-old) and old (25-month-old) male mice. The left panel shows the<br/>representative images of SPiDER-bGal and TNF-a co-staining and the zoomed-in views. The middle panel shows the fold change of SPiDER-bGal and TNF-<br/>a-positive cells in multiple tissues. The right panel shows the quantifications of TNF-a-positive cells along with the distance from SPiDER-bGal-positive cells.<br/>Using a randomly selected SPiDER-bGal-positive cell as the center point, the number of TNF-a-positive cells within equal intervals of distance and per unit area is<br/>counted. n = 4&#8211;6 for each group. Scale bars, 20 mm.<br/>(E) Spatial mappings showing the spot types of the corresponding sample in Figure 3I. The colors indicated different spot types as in Figure S2E. Scale bars, 500<br/>and 100 mm (zoom-in) for hippocampus, spinal cord, and lymph node, 1 mm and 200 mm (zoom-in) for the others.<br/>(F) Bar plot showing the proportion of Ptprc+ spot in SSSs across tissues in old mouse.<br/>(G) Pie charts showing the proportion of different spot types in SSSs in old mouse. The colors indicated different spot types as in Figure S2E.<br/>(H) Boxplots showing the proportion of different spot types in SSSs in each old mouse, with each point indicating a sample.<br/>(I) SPiDER-bGal and NeuN (neuron&#8217;s marker) co-staining of hippocampus from young and old male mice. The left panel shows the representative images of<br/>SPiDER-bGal and NeuN co-staining. The middle panel displays the percentage of NeuN-positive cells in SPiDER-bGal-positive cells. The right panel displays the<br/>fold change of SPiDER-bGal-positive cells in hippocampus from young and old group, and the proportion of NeuN-positive and NeuN-negative cells within<br/>SPiDER-bGal-positive cells in each group (n = 5 mice per group). Scale bars, 50 and 10 mm (zoom-in).<br/>(J) SPiDER-bGal and CYP11A1 (Leydig cell&#8217;s marker) co-staining of testis from young and old male mice. The left panel shows the representative images of<br/>SPiDER-bGal and CYP11A1 staining in testis. The middle panel displays the fold changes of SPiDER-bGal-positive cells in young and old groups. The right panel<br/>displays the fold change of CYP11A1-positive and CYP11A1-negative cells within SPiDER-bGal-positive cells in each group (n = 5 mice per group). Scale bars,<br/>200 and 20 mm (zoom-in).<br/>(K) Co-staining for SPiDER-bGal andCD138 (plasmocyte&#8217;smarker) in the spleen from young and oldmalemice. The left panel shows the representative images of<br/>CD138 and SPiDER-bGal co-stainings and the zoomed-in views. The right panel displays the statistical analysis for the percentage of CD138-positive cells in<br/>SPiDER-bGal-positive cells (n = 5 mice per group). Scale bars, 100 and 15 mm (zoom-in).<br/>(L) Co-staining for SPiDER-bGal and CD138 in lymph nodes from young and old male mice. The left panel shows the representative images of CD138 and<br/>SPiDER-bGal co-stainings and the zoomed-in views. The right panel displays the statistical analysis for the percentage of CD138-positive cells in SPiDER-bGal-<br/>positive cells (n = 5 mice per group). Scale bars, 200 and 25 mm (zoom-in).<br/>(M) Left, Spatial mappings showing the distribution of plasmo spots in spleen, lymph node, and small intestine from young and old mice. Right, boxplots showing<br/>the proportion of plasma spots across tissues during aging. Wilcoxon rank-sum test. Scale bars, 1 mm and 200 mm (zoom-in) for spleen and small intestine,<br/>200 and 100 mm (zoom-in) for lymph node.<br/>The Mann-Whitney test was utilized in (D) and (I)&#8211;(L). Data are presented as the mean &#177; SEM.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S7. Deciphering cellular and transcriptional changes in male mouse tissues, related to Figures 3 and 4<br/>(A) Heatmap showing the age-related genes, categorized into eight gene expression modules. The color of heatmap represents the overall trend of the scaled<br/>gene expression.<br/>(B) Boxplot showing the increased OSE score around the SSS in 19-month-oldmouse liver and spleen. Kruskal-Wallis test. Scale bars, 500 and 200 mm (zoom-in).<br/>(C) Heatmap displaying the gradual changes in gene expression surrounding the SSS in 19-month-old mouse, categorized into six gene expression modules.<br/>(D) Enriched pathways of different gene expression modules that are down- or upregulated with the increased distance from SSSs in 19-month-old mouse.<br/>(E) Fitting curves of expression levels of gene involved in the indicated pathways in the module 2 and the module 4 + 5.<br/></p>
<p>(legend continued on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>(F) Immunofluorescence staining of complement 4B (C4B) in spleen and lymph node of young and old mice. The left panel shows the representative images<br/>of C4B staining and the zoomed-in views. The right panel displays the fold change of C4B-positive cells from young (2-month-old) and old (25-month-old) mice<br/>(n = 4&#8211;5 mice per group). Scale bars, 40 and 15 mm (zoomed in). Mann-Whitney test. Data are presented as the mean &#177; SEM.<br/>(G) Heatmap showing the expression of upregulated immunoglobulin-related genes across tissues during aging. Colors ranging from white to red indicate the<br/>difference from low to high.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>(legend on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S8. Analysis of immunoglobulin-expressing cell accumulation and localization in multi-tissues of agedmale mice, related to Figure 4<br/>(A) Representative western blot images of IgG heavy chain (IgG-H), light chain (IgG-L), and Igkc proteins in multiple tissues from young (2-month-old) and old<br/>(25-month-old) male mice (n = 6 mice per group). b-actin was used as the loading control for all the tissues except heart. A band extracted from Ponceau red<br/>membrane staining was utilized as a loading control (L.C.) in heart.<br/>(B) Quantification of western blot data for IgG heavy chain (IgG-H), IgG light chain (IgG-L), and Igkc protein levels inmultiple tissues from young and oldmice (n = 6<br/>mice per group).<br/>(C) Co-staining of IgG and Igkc in hippocampus, small intestine, spleen, and lymph node from young and oldmice. The left panel shows the representative images<br/>of IgG and Igkc co-staining. The right panel displays the fold change of IgG and Igkc-positive cells from multiple tissues in young and old mice (n = 5 mice per<br/>group). Scale bars, 20 mm.<br/>(D) Spatial mappings revealing the distribution of Igkchigh spots (left), the overlap between Igkchigh spots, plasmo spots, and SSSs (middle), and the overlap<br/>between Igkchigh spots and SSSs (right). Scale bars, 1 mm and 200 mm (zoom-in) for spleen, 200 and 100 mm (zoom-in) for lymph node.<br/>(E) Co-staining of CD138 and Igkc in spleen and lymph node from young and old mice. The left panel shows the representative images of CD138 and Igkc<br/>co-stainings and the zoomed-in views. The right panel shows the quantifications of changes in CD138-positive and Igkc-positive cells, as well as the percentage<br/>of CD138-positive cells in Igkc-positive cells (n = 5 mice per group). Scale bars, 200 and 25 mm (zoom-in).<br/>(F) Co-staining of SPiDER-bGal and Igkc in spleen and lymph nodes from young and old mice. The left panel shows the representative images of SPiDER-bGal<br/>and Igkc co-stainings and the zoomed-in views (n = 5mice per group). The right panel shows the quantifications of Igkc-positive cells along with the distance from<br/>SPiDER-bGal-positive cells. Using a randomly selected SPiDER-bGAL-positive cell as the center point, the number of Igkc-positive cells within equal intervals of<br/>distance and per unit area is counted. Scale bars, 200 and 25 mm (zoom-in).<br/>(G) Pie plots showing the proportion of Ighigh spots in different cell types of old tissues.<br/>(H) Spatial mappings showing the Igh and Igkc expression across tissues during aging. Scale bars, 500 mm.<br/>(I) Boxplot showing the proportion of Ighhigh and Igkchigh spots across tissues during aging. Wilcoxon rank-sum test.<br/>The Mann-Whitney test was utilized in (B), (C), (E), and (F). Data are presented as the mean &#177; SEM.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>(legend on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S9. Multi-time point histological analysis of aging and inflammatory biomarkers in male mouse tissues, related to Figure 5<br/>(A) SPiDER-bGal staining of multiple tissues, including the male mouse hippocampus, liver, spleen, and lymph node, in 5 different age groups (2-month-old,<br/>4-month-old, 13-month-old, 19-month-old and 25-month-old). Left, representative images, and right, the percentage of SPiDER-bGal-positive cells in the tissues<br/>(n = 3&#8211;5 mice per group). Scale bars, 40 mm.<br/>(B) Immunohistochemical staining of P21 in the mouse hippocampus, liver, spleen, and lymph node from 5 different age groups. Top, representative images, and<br/>bottom panel, the percentage of P21-positive cells in the tissues (n = 5 mice per group). Scale bars, 50 and 10 mm (zoom-in).<br/>(C) Immunofluorescence staining of lamin B1 in the mouse liver and spleen from 5 different age groups. Top, representative images and zoomed-in view. Bottom,<br/>the fold change of lamin B1 fluorescent intensity in tissues (n = 5 mice per group). Scale bars, 10 mm.<br/>(D) Aggresome staining in the mouse hippocampus, spleen, and lymph node from 5 different age groups. Top, representative images and zoomed-in view.<br/>Bottom, the fold change of aggresome-positive areas in the tissues (n = 4&#8211;5 mice per group). Scale bars, 50 and 10 mm (zoom-in).<br/>(E) Masson staining in spleen and lymph node from 5 different age groups. Top, representative images and zoomed-in view. Bottom, the fold change of fibrosis<br/>areas in the tissues (n = 5&#8211;6 mice per group). Scale bars, 50 mm.<br/>(F) Immunohistochemistry staining of gH2AX in liver, spleen, and lymph node from 5 different age groups. Top, representative images and zoomed-in view.<br/>Bottom, the fold change of gH2AX-positive cells in the tissues (n = 4&#8211;5 mice per group). Scale bars, 50 and 20 mm (zoom-in).<br/>(G) Immunofluorescence staining of 4-HNE in the mouse hippocampus, spleen, and lymph node from 5 different age groups. Top, representative images and<br/>zoomed-in view. Bottom, the percentage of 4-HNE-positive cells in tissues (n = 4&#8211;5 mice per group). Scale bars, for hippocampus and spleen, 40 and 15 mm<br/>(zoom-in); for lymph node, 200 and 15 mm (zoom-in).<br/>(H) IL-1b staining in the mouse hippocampus, spleen, and lymph node from 5 different age groups. Top, representative images and zoomed-in view. Bottom, the<br/>percentage of IL-1b-positive cells in tissues (n = 4&#8211;5 mice per group). Scale bars, 40 and 15 mm (zoom-in).<br/>(I) Immunohistochemistry staining of TNFa in spleen and lymph node from 5 different age groups. Top, representative images and zoomed-in view. Bottom, the<br/>fold change of TNFa-positive cells in the tissues (n = 4&#8211;5 mice per group). Scale bars, 50 and 10 mm (zoom-in).<br/>(J) Immunofluorescence staining of MMTV in the mouse spleen and lymph node from 5 different age groups. Top, representative images and zoomed-in view.<br/>Bottom, the fold change of the number of MMTV-positive cells in tissues (n = 5 mice per group). Scale bars, 40 and 15 mm (zoom-in).<br/>(K) Co-staining of SPiDER-bGal and Igkc in hippocampus, spleen, and lymph node from 4-month-old and 19-month-old mice. The left panel shows the<br/>representative images of SPiDER-bGal and Igkc co-stainings and the zoomed-in views (n = 5 mice per group). The right panel shows the quantifications of Igkc-<br/>positive cells along with the distance from SPiDER-bGal-positive cells. Using a randomly selected SPiDER-bGAL-positive cell as the center point, the number of<br/>Igkc-positive cells within equal intervals of distance and per unit area is counted. Scale bars, 50 and 15 mm (zoom-in) for hippocampus; 200 and 35 mm (zoom-in)<br/>for spleen; 200 and 20 mm (zoom-in) for lymph node.<br/>(L) Schematic diagram of the parabiosis experiment in mice, in which two living mice are joined surgically and shared physiological systems.<br/>(M) Immunofluorescence staining of IgG in spleens from parabiosis mice. Left, representative images from four groups of mice, including young mice from<br/>isochronic parabiosis (Iso-Y), old mice from isochronic parabiosis (Iso-O), and young mice from heterochronic parabiosis (Het-Y), as well as old mice from<br/>heterochronic parabiosis (Het-O). Right, the fold change of IgG-positive cells in each group (n = 4&#8211;5 per group). Scale bars, 50 mm.<br/>(N) Immunohistochemical staining of P21 in spleens from parabiosis mice. Left, representative images from four groups of mice, including Iso-Y, Iso-O, and<br/>Het-Y, as well as Het-O. Right, the fold change of P21-positive cells in each group (n = 5 per group). Scale bars, 50 and 20 mm (zoom-in).<br/>(O) Schematic diagram of the exercise experiment in mice.<br/>(P) Immunofluorescence staining of IgG in spleens from exercise old mice or control old mice. Left, representative images and zoomed in view. Right, the fold<br/>change of IgG-positive cells in each group (n = 5&#8211;6 per group). Scale bars, 40 and 15 mm (zoom-in).<br/>(Q) Immunohistochemical staining of P21 in spleens from exercise old mice or control old mice. Left, representative images and zoomed in view. Right, the fold<br/>change of P21-positive cells in each group (n = 4&#8211;5 per group). Scale bars, 50 and 20 mm (zoom-in).<br/>The Mann-Whitney test was utilized in (A)&#8211;(K), (M), (N), (P), and (Q). Data are presented as the mean &#177; SEM.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>(legend on next page)<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
<div style="page-break-before:always; page-break-after:always"><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div>
</div><div><p>Figure S10. Immunoglobulin G induces senescence in mouse and human macrophages, related to Figures 6 and 7<br/>(A) Heatmap showing the upregulated FC receptor-related genes in macrophages across different tissues during aging.<br/>(B) Co-staining of IgG and F4/80 (macrophage&#8217;s marker) in spleen and lymph node from 2-month-old and 25-month-old male mice. The left panel shows the<br/>representative images of IgG and F4/80 co-staining, as well as the zoomed-in views (n = 5 mice per group). The right panel shows the quantifications of IgG-<br/>positive cells alongwith the distance from F4/80-positive cells. Using a randomly selected F4/80-positive cell as the center point, the number of IgG-positive cells<br/>within equal intervals of distance and per unit area was counted. Scale bars, 200 and 40 mm (zoom-in).<br/>(C) Network diagram showing the results of enrichment analysis of upregulated aging DEGs in macrophages across different mouse tissues.<br/>(D) Immunofluorescence staining of F4/80 in BMDM, SPDM, and PCDM. The left panel shows the representative images of F4/80. The right panel displays the<br/>proportion of F4/80 positive cells. n = 3 biological replicates. Scale bars, 50 mm for BMDM and SPDM and 20 mm for PCDM.<br/>(E) SA-b-Gal staining of culturedmacrophages (SPDM and PCDM) treated with vehicle (Veh) or IgG. The left panel displays the representative images of SA-b-Gal<br/>staining. The right panel shows the fold change of SA-b-Gal-positive cells in each group (Vehicle or IgG treated). n = 4 biological replicates. Scale bars, 50 mm.<br/>(F) Co-staining of aggresome and F4/80 in SPDMand PCDM treated with Veh or IgG. The left panel shows the representative images of aggresome and F4/80 co-<br/>staining and the zoomed-in views. The right panel displays the statistical data of aggresome-positive cells in Veh- and IgG-treated groups. n = 5 biological<br/>replicates. Scale bars, 50 and 20 mm (zoom-in).<br/>(G) Representative western blot images and quantifications of P21 and iNOS protein levels in SPDM treated with Veh or IgG. n = 4 biological replicates.<br/>(H) Representative western blot images and quantifications of p-P65 and STAT1 protein levels in SPDM treated with Veh or IgG. n = 4 biological replicates.<br/>(I) Left, ELISA test to detect the IL-6, IFN-b, CCL2, and CCL5 levels in the culture medium collected from cultured macrophages (PCDM) treated with Veh or IgG.<br/>n = 4 biological replicates. Right, ELISA test to detect the IL-6, IFN-b, CCL2, and CCL5 levels in the culture medium collected from culturedmacrophages (SPDM)<br/>treated with Veh or IgG. n = 4 biological replicates.<br/>(J) Measurement of nitrite production in the medium of cultured macrophages (SPDM and PCDM) treated with Veh or IgG. n = 4 biological replicates.<br/>(K) Left, principal-component analysis (PCA) displaying the differences in transcriptional profile in BMDM, SPDM, PCDM, andmicroglia after IgG treatment. Right,<br/>network diagram showing the results of enrichment analysis of upregulated DEGs in BMDM, SPDM, PCDM, and microglia after IgG treatment.<br/>(L) Heatmap showing the upregulated DEGs overlapped with SASP gene set in BMDM, SPDM, PCDM, and microglia after IgG treatment.<br/>(M) Network diagram showing the upstream regulatory factors of upregulated DEGs in BMDM, SPDM, PCDM, and microglia after IgG treatment.<br/>(N) Top, the ring diagram showing the proportion of differentially expressed repeat elements (DREs) across different cell types after IgG treatment, with colors<br/>representing different repeat element class. Bottom, heatmap showing the upregulated DREs in BMDM, SPDM, PCDM, and microglia after IgG treatment.<br/>(O) SA-b-Gal staining of siRNA-transfected culturedmacrophages (BMDM) treated with Veh or IgG. The left panel displays the representative images of SA-b-Gal<br/>staining. The right panel shows the proportion of SA-b-Gal-positive cells in each group (si-NC Vehicle, si-NC IgG or si-Fcgr4 IgG treated). n = 4 biological<br/>replicates. Scale bars, 50 mm.<br/>(P) Immunofluorescence staining of CD68 (macrophage&#8217;s marker) in human macrophage. The left panel shows the representative images of CD68 staining. The<br/>right panel displays the proportion of CD68-positive cells. n = 5 biological replicates. Scale bars, 50 mm.<br/>(Q) Co-staining of P21 and CD68 in human macrophage treated with Veh or IgG. The left panel shows the representative images of P21 and CD68 co-staining<br/>views. The right panel displays the fold change of P21 fluorescence intensity values in Veh- and IgG-treated groups. n = 5 biological replicates. Scale bars, 20 mm.<br/>(R) ELISA test to detect the IL-6 level in the culture medium collected from human macrophage treated with Veh or IgG. n = 4 biological replicates.<br/>(S) SA-b-Gal staining of cultured humanmacrophages treated with Veh, IgG, or heat-inactivated IgG (IgG-HI). The upper panel displays the representative images<br/>of SA-b-Gal staining. The lower panel shows the fold change of SA-b-Gal-positive cells in each group. n = 5 biological replicates. Scale bars, 50 mm.<br/>(T) Immunofluorescence staining of IBA-1 in mouse and human microglia. The left panel shows the representative images of IBA-1. The right panel displays the<br/>proportion of IBA-1 positive cells. Data are presented as the mean &#177; SEM. n = 3 biological replicates. Scale bars, 50 mm (mouse microglia), 20 mm (human<br/>microglia).<br/>(U) Co-staining of aggresome and IBA-1 in human microglia treated with Veh or IgG. The left panel shows the representative images of aggresome and IBA-1 co-<br/>staining and the zoomed-in views. The right panel displays the statistical data of aggresome-positive cells in Veh- and IgG-treated groups. n = 5 biological<br/>replicates. Scale bars, 20 mm.<br/>(V) ELISA test to detect the IL-6 level in the medium human microglia (n = 6 biological replicates) treated with Veh or IgG.<br/>(W) Representative western blot images of FcRn protein levels in BMDMand SPDM treated with 50 nMASO-NC or 50 nMASO-Fcgrt. Protein levels are illustrated<br/>as a heatmap on the right, utilizing a color spectrum transitioning fromwhite to blue to denote a progressive elevation in protein contents, ranging frommaximal to<br/>minimal.<br/>(X) Masson trichrome staining of the spleen and lymph node treated with Veh or IgG. Left, representative images. Right, the fold change of fibrosis area (n = 9&#8211;10<br/>mice per group). Scale bars, 100 and 50 mm (zoomed in).<br/>(Y) SA-b-Gal staining of the spleen, lymph node, hippocampus, and lung from old mice treated with ASO-NC or ASO-Fcgrt. The left panel displays the repre-<br/>sentative images of SA-b-Gal staining. The right panel shows the fold change of SA-b-Gal-positive areas in each group (n = 5&#8211;10 mice per group). Scale bars,<br/>100 mm.<br/>The Mann-Whitney test was utilized in (B), (E)&#8211;(J), (O), (Q)&#8211;(S), (U)&#8211;(V), (X), and (Y). Data are presented as the mean &#177; SEM. The arrows indicate positive cells.<br/></p>
<p><b>ll<br/></b>Resource</p>

</div></div>
</body></html>